sql/tests.TestRandomSyntaxFunctions: TestRandomSyntaxFunctions failed [unexpected type *tree.DLTree for key value]

sql/tests.TestRandomSyntaxFunctions.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20671833?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20671833?buildTab=artifacts#/) on master @ [7f098a12bbaf8c31787db28b36f06c6e133c5c00](https://github.com/cockroachdb/cockroach/commits/7f098a12bbaf8c31787db28b36f06c6e133c5c00):


```
    rsg_test.go:125: Last executed (4): SELECT sha384() 
    rsg_test.go:125: Last executed (5): SELECT sha512(e'W-/V\x1a\x0bh))':::STRING, e'u?\x134&\x0fXm':::STRING, '':::STRING) 
    rsg_test.go:125: Last executed (6): SELECT shobj_description(69608892:::OID, e'm\x11':::STRING) 
    rsg_test.go:125: Last executed (7): SELECT sha512('\x27':::BYTES, '\xa1b6fc9d46':::BYTES, '\x3e83f944':::BYTES) 
    rsg_test.go:125: Last executed (8): SELECT show_trgm(e'1*\t,<\t':::STRING) 
    rsg_test.go:125: Last executed (9): SELECT sign(0.25287605304302574:::FLOAT8) 
    rsg_test.go:125: Last executed (10): SELECT sign((-2.23707780643329769E+34):::DECIMAL) 
    rsg_test.go:125: Last executed (11): SELECT sign((-128):::INT8) 
    rsg_test.go:125: Last executed (12): SELECT similar_escape(e'\x07``koR\x01':::STRING) 
    rsg_test.go:125: Last executed (13): SELECT similar_escape(e'd\x1d\'b\x01{\x1d9':::STRING, e'\x1dMg-W':::STRING) 
    rsg_test.go:125: Last executed (14): SELECT similar_to_escape(e'\x13&\x13':::STRING) 
    rsg_test.go:125: Last executed (15): SELECT similar_to_escape(e'.R\\h>_':::STRING, NULL::string) 
    rsg_test.go:125: Last executed (16): SELECT similar_to_escape(e'\x14\x10\n':::STRING, 'mdK':::STRING, e'r\f\fg':::STRING) 
    rsg_test.go:125: Last executed (17): SELECT similarity(e'H,B\r\x0e ':::STRING, 'bVOU59':::STRING) 
    rsg_test.go:125: Last executed (18): SELECT sin((-0.19143187178980847):::FLOAT8) 
    rsg_test.go:125: Last executed (19): SELECT sind(1.4538055227070708:::FLOAT8) 
    rsg_test.go:125: Last executed (20): SELECT sinh(1.5127787276562334:::FLOAT8) 
    rsg_test.go:125: Last executed (21): SELECT soundex((SELECT NULL)::string) 
    rsg_test.go:125: Last executed (22): SELECT split_part((SELECT NULL)::string, NULL::string, (-3558427288983803259):::INT8) 
    rsg_test.go:125: Last executed (23): SELECT sqrdiff((-3535033619384751711):::INT8) 
    rsg_test.go:125: Last executed (24): SELECT sqrdiff(0.01366579158687756299:::DECIMAL) 
    rsg_test.go:125: Last executed (25): SELECT sqrdiff(1.7089089918444453:::FLOAT8) 
    rsg_test.go:125: Last executed (26): SELECT sqrt((-0.935643020272936):::FLOAT8) 
    rsg_test.go:125: Last executed (27): SELECT sqrt(2.101790188518715676E+25:::DECIMAL) 
    rsg_test.go:125: Last executed (28): SELECT st_3dlength('010400000000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (29): SELECT st_addmeasure(NULL, (-1.1858619033235875):::FLOAT8, 1.1201411387747906:::FLOAT8) 
    rsg_test.go:125: Last executed (30): SELECT st_addpoint('01060000C00700000001030000C0010000000B00000020FE8E32E42BD4C1C83D75FD83BBDA41428C8C2CC3E8F44198B9D6E56F53D8C17065DB9DD6E9CE4198A33460984AD741D653565B0EB7FDC180E5EEDE9E1DE341AC19FB72E44CEBC18B463126CEFAF4C1A8DC269DA16FFE41570AA99B680DF2C118D58EDEEE5EDC41A041F89E3B19CA41C0C080694239E241D86BF6AA5B3DF641CC4545D80011FC41852EFFF75C7EF4C162B488A20D3FE4C15070B192BCB6D3416019FDB6AEC0FE41967CD5DC6928F9411CC01CDEB0A2FCC1D625E95CC2FBE9C12A94DBA108D9F541E46BAB86ACC7FE4183F24D2F58DFF9C1B8F2990C8ECFE2418AEF9A287E59F5413690DC69CBB90142BC2D86FA9A45FE4160D47B48CAC2C5C168093AAC6C82F6C12498BC04FD0302425CDD7E90473EEDC102EC9EB95E79014210592CCD44A4D8419897ACDCAB96E841C85E47106CC0F341E9B29428998BFDC120FE8E32E42BD4C1C83D75FD83BBDA41428C8C2CC3E8F44198B9D6E56F53D8C101030000C00100000009000000A73948D48878FAC1267BF3BFA6BAFCC15008C7265551D341E8E7D20688B6E341990205D51A97F5C1049BB86FE59202C2F8226B0DF5890242263DE366BDD4F5417E37CB5D77BFF0C13F8389033FB701C29C156D14FAA7E8C18E4B31A719F5014250C46C7B3319C8410DDF7DAD855E02C209A504F9C67302C249C2B7729E0EFCC17C72958CCFE80042D0A2B85BBD50D4416478AA0DD012EDC1208B9B955213C0C1A0C0AF0B2795D341B82757AE94E7E2C174C5FB5FE409FB41308500300592DA41B8C24CC0C99DD541F81AFD0B42C3E1C14425996EA856F9416A2CCB0AE775014262E36D7A5E47E4C160ED55CAE4DCD74132D309AA19540242181B8E2A0765DEC1A73948D48878FAC1267BF3BFA6BAFCC15008C7265551D341E8E7D20688B6E34101030000C00100000004000000C4A8F5207F17EA41BDD9295998C5F2C1071AB4EAF8E9F1C1844E3EF35105DBC110CF1DA03D67F841946EC5FC9C300042168182D874BD00C200A7226433DAF14120955DF0E501F0410C1EAA7E3B1AE941EEE88A853C86F3C1DCCAD771C638F6C1C4A8F5207F17EA41BDD9295998C5F2C1071AB4EAF8E9F1C1844E3EF35105DBC101030000C0010000000C000000234AE57B6C06FCC1D95AE4C6D847FEC116916E2292CDECC1DE04993415DEEEC10826D996DAECF7C1CBA027DEE808FFC1903847C932BCE64146ADA5B4AA2CFA41F2ACF99C129DEFC1706690EECF5A02C2E595FDF4AF7202C20F900C60DDC300C2CA05D68644FDE9C169D252B351A300C28CB28EA7282FFB4178236F50D4AD004214D343452E3EDCC13AE80766605001C2A9DDF37E331AFFC18E7F81961DDAEDC11832083B9D000242F09327B0A55CD4C130A033C7D36BFF4108FFC2871BD4D3C11886CAE632C9FA41809EE9429946A9C1000D6F7A7116FE41DC84A1871F84FA4194B9737159A5EB41C0BC83DA3E47C1C12A6F2534D842F3C1147869912B82EC416805DDCB571A01421E4D594CEBF0F241A08A03D24D23D9C198F4CB062A2FE8414E3C4C9C90E6E8C1649BB136727BED41901A57F6E5BFD0C1BED8CCA989A6E3C1A46011AA92EAF6C1A0450A865E55EAC1D0BC33F7B77AF341E0C5EE37244FFB41234AE57B6C06FCC1D95AE4C6D847FEC116916E2292CDECC1DE04993415DEEEC101030000C0010000000C000000AE001EFD5400EAC1A853C2467BA0EBC1F884575674C8E6C15EC2A83B88EFFFC1BA97FC2A4F9BFEC14DB45090E191F6C196971B36AD100042E266F8E94AF4F94190C0A01AC0C9DAC17E0BB3D656B8FFC1E07A8A052807E7C1C48B688C1593FE410000B63F3A269BC15F0BD8F59A0201C250E0DD79716DFFC1F126D458F33CF7C1D49310A525E1FC4112CF2680AF2800C26848032B8F31D5C118DCDC281D93F241CAD2E7A8E120024229F2A735576101C20034141D1032B441C646F37812880242BABAF7F748E80042902C63BA72A8C941ECEC2683F764E9C1A2B383CEE100F7C1D4D42D271DD4FD41FCE7AC0DFF6CFC417EB9E1A2FD6702425068F35F1660E0C158BE507B0355E6419076D0F2AEF4FA41A928C526BD4CFFC1C88123576CB2FF41B0837FBA0173D14180E703051B36FD41902AC20C8669CD41B085487B52D0CB41FEA44F78C756E4C1B06DF5B3210ED1C1A0EBC3A10D2BDBC1D00C3E89ECB8D9C1AE001EFD5400EAC1A853C2467BA0EBC1F884575674C8E6C15EC2A83B88EFFFC101030000C0010000000C00000020203A6CC3A4FBC174A5335800F4E9C188EC61797AAFFB414064720E3A99CDC1408BB1D9CF49F7C1906C62215593F2C1ACE07EEFBE72E8C13A9260D94A4901420073A011A218DC414EC425F1D2AEF3C1460C3EDC277FE2C1E52B6D1339AE00C268AA8883B073ED41C367FEB26962F0C134F3C46CF5A3EF417A7C75322DE2EEC1A0436CCF2F9BE141003AD9D3AA0A7EC14DA0C5EC8A6EFFC19AC02A64F2450242F858BD7E301ED841400D9A20EDC8C541A81ECB10619FD3C189AE642E090DF9C1707504136E1BC0C1F45B398426E2E3415A0E2438513EE5C16ADA05614CAD0142F1173F5C2496F6C170EB12E1C154FE41A0866137FA2FB341A8A43B73DA3BDEC1F00CE301F88CFEC194ED52EBCA43FC4164860BF3D1D80142E87C942FAE7BE841B308A238049F01C2005666DC870FE941CCB9B8005984F7C1F467EED20902F741E9820CBED0CAFFC130E7286F44B1DC4144D3A2FE1F14EA4159591F54A8AF01C220203A6CC3A4FBC174A5335800F4E9C188EC61797AAFFB414064720E3A99CDC101030000C0010000000D000000E3B5710294BFF9C100FA6657E28F8F4118F80825F3C6D9C1B0E79B8F2D81C0C1F25CD06A0D14F6C152AAAE3DCB4AEEC170653C2BAAADFA418C610F52A27DFC41A40DD56D5FE9E341865303391F61EAC1B0F76992E2FD01C2A801FB4043BADF416C4C4E626CF2F6412E20C3CABDBAF3C116917CE8FDD8F841A8514B03CF53E74128EFA825064EE941B0A9EE7BB7B2E54110845D5396CDFB41ECE589755ED5E041E00B10CC990201426C7EC1E30460F5416CAA4345F764E341303D67B3465EDE41E47E6B604DBAFE413A08C76B43F5F6419ACFCCDD1C30FAC1D8D81D36A5B3E9C160963555C427EA41CA6786FF5ADDF441A610F20E34E1E7C120F1B6CEF27AEEC1FE8C5AC721A4F54118D2538F59B7FC41962351C0192EF24128F1D6B3F80CE2419E3F07BAB0B1EAC18C530FC39724F841B6579269A2F6EDC16054BAF86BC9E941DA98A327AF1801C25685EFC6443A0242022771C74258F941E07892E5F39BC041CF095034DB42FFC1B68259161E5FF1410A63FE4BCFD50042FC971D23E0BCFC41E3B5710294BFF9C100FA6657E28F8F4118F80825F3C6D9C1B0E79B8F2D81C0C1':::GEOMETRY, '01020000C0050000006C98659EB7D3F6410C93FD721243F5C16051EDE35186DF41638DB318002DF8C1767DD2BA10720142FE2E4B535CE1F6C174189818246FEF41AA09D1F7E305FAC1521FC40A05D701420096519549348F4172A251F844E6F541148439A4BE94EB4160C897620186F141780B30E53B6BFB41782055C878F8E941F0DC346B277FE1419313EA9F21C7F6C1847D57BD1153EB41B05E6E16639AE4C110B3308705FDD8C1':::GEOMETRY, 3069612451114767254:::INT8) 
    rsg_test.go:125: Last executed (31): SELECT st_addpoint('0106000040050000000103000040010000000C000000F361B6FE0B66F4C1C86017A375F5DB41C0997FCEFF8DC1C1E0EAD051C93BCDC1013AE008CBF2F1C1EE6DD73DEBEA0142B6164A42547FF041F3E0A9C84C2002C2981755AF841FF14100D929799B7BC1C14064B95D0715B041003A3159230984C1D2E4444EDA4EFA41D83C71F085C5E24170B2AB7D8F64F341D06EFE518BA7F3414C9EF25C423DFE4112C5FF602D39F7C1881401CA07F2DCC11EB57B5F4E75F941F28EBC2A100EF5C18643B647F4B6EBC1D09267DA00BC00425E1C5BE07F3602427C5E454AF2DFE8C13C764C350B35EC41ACFEE036D330D8C182355876AA1400C20C7DE3370197E041DC5F6E40B465D5C1B7AD83CD21E701C2B453DC573AB9E041F7F0683BBF7AFFC1F361B6FE0B66F4C1C86017A375F5DB41C0997FCEFF8DC1C101030000400100000005000000930CFCC63D04F9C10030C9E6374157416A4A35CCA8DFE9C1A90E869C2CC5F7C14CBF06406C07DBC1008300430EAC8E41582622C60870ED4118D40C1E9234D141905119067076EE41AA3D75625631F041083B38E545D5DC414C27F37AABF3F041930CFCC63D04F9C10030C9E6374157416A4A35CCA8DFE9C10103000040010000000900000070A7540D9C83FCC1ED8F2505C5B8F5C160CAC14A9FB6C3C12820AD50B139F7C12EA4032BFA87F8C175A90AA8D3A401C26C356010CE67F2C117735E54977BFFC19CE89C36B9C5EAC11013060234C60042D4F23CE5A7F6F8C1C3089F5B4B8DFDC1F682DB956FC3F0417416452F267BE8C1A0DBDA6F0BA8C541BC329DC30578E44100FFDA3A9BCAD3C11097F8A1B8140042A8C73DEB3CE3D9C114E8E65D243E024264EC06B05192F6C1B3CC76420E2DFFC158C5C6F05D0CEC4100C713F59776DCC170A7540D9C83FCC1ED8F2505C5B8F5C160CAC14A9FB6C3C10103000040010000000C000000EA86DA978F8BF4C1327DB243C559F1C184F2363564A1EA4162E928A3F17FEEC1367D2E77FABBEBC1702239D4FAF3C7C1D055C16B14ECDBC1421D7A49DA4EF5C15C96E91D9987F3417862DAD1D0DAD1416F1172C2CAE8FCC140255621E305BCC130B960F3E6AAD6C10AA5D46A9159EDC1B473524FBF60F641BCE394751E95F2412A0AFBCF3A45E5C14026026C9DDCD941CC087FC9D78EE4C120E0C764831CC7C10E4573536D76FCC1F3AEEE2A923EF4C142631E4AE461F641D4DFAB9E8464E3C158A3E8A729E1F6C1B4CD81E1D25EEA41342B3444B576F441DCB38E8C47B4FEC1A431C4EF2952F841180F88A1F37EF4C129F69B8A505301C2E009D3097C58C641A88DB83F1A72FE41EA86DA978F8BF4C1327DB243C559F1C184F2363564A1EA4101030000400100000007000000D8D76FF16BE4D4C142E0EE7D7D7FE8C13C60490AE238FE417EA83B4E3C56E0C1D0DC636310E3E9C16C91FF7C1F9FF141BCCFCD965982FD4174EF54ED98F000C2B034BDFB2850D641F836C8C3C93CFC4104FF9D1DBEFFFAC1AF600B352594F3C1DC21A0443EA1F9417012C4641A83E14158EF82F8881EE441A8D223707C8B0042A464FCAB4D2CF941D052B9DD273DE3C1D8D76FF16BE4D4C142E0EE7D7D7FE8C13C60490AE238FE41':::GEOMETRY, '010300008000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (32): SELECT st_affine('01060000C00300000001030000C00100000005000000A90278C7140700C2E8A534292A89FBC1B46F119554FEF741D8FEB89D109CF5417875502EBDF7F1C1FBE1B1B32D00F5C1D6002579503EE2C1FAAA9A6E6FC701421898BA2422A7EB41641A348D3DF90042A0075AB02AD5C641803F74A059510242A846F5F7E7FBDB413CE9CE41A1ACFF4148396D548248DD418C645B6B4C50E9C1A90278C7140700C2E8A534292A89FBC1B46F119554FEF741D8FEB89D109CF54101030000C0010000000A00000006202CEA0F8300C294AD62941789F9C199B09D5248F101C2EA1EA62EFCF5F5C140C54CEA32FBF8C1D2CB5C0640E4FDC19878A51346A1DC4114FC4BAD9BEAFB417879D6018B36EC4134D601695F9CF9C120988C07DB10BB4178CA72A052D2F5411644AAAF3F1E00425658BEF386AFE2C13E204A08CCE4E8C1FE4B9CBCE441E8C1B07ED37EC572FE4164BAE7CA0B63DDC1B2E4BBCF1209FDC14030735B4F7EEF41343EB89ED710FE41E030E1C92423BE4138D70F0023D1EDC1FC0E6CCF3ADDE6414C0BF8EB28C6E74160B60C061EA3C741C0322C2E60BAFB41343D7F568161DDC14C4AADFD2AE4EA41448640657BCEFE4140A2C2978FA300C270FC7CCF2F3EE9411AC94AD24E26FCC14207F0FE1C810142380E5650A9C6D841C8342CE1B831014206202CEA0F8300C294AD62941789F9C199B09D5248F101C2EA1EA62EFCF5F5C101030000C00100000007000000E0DEA799FEFEC7C1A6DB5DEE7AD4E6C1802C18F58CC1C4C1E43BC0C347DBE54120AEC65EAC34BF41CC00D63A12E6F1C1C0E74BEFAEE6BF4180D754387374BF419471C2DA03CAE4414AE7DE6923D400C2886B0A0E93E0F241044FCE6D7841D5C12A672F78723EF84180593F414FB3A8C16A00117DDFC7E7C14B1869517D6EF0C1385B5FB93F0ED5C1246A08A43A39E34151638FB10CE701C25ACA4860C6F5E1C1269CF10BB32F01C2604DBC2B08D7FE4146FA77FB3FF9F6C1B8E947E3D34EEE41E0DEA799FEFEC7C1A6DB5DEE7AD4E6C1802C18F58CC1C4C1E43BC0C347DBE541':::GEOMETRY, (-0.32638040697618825):::FLOAT8, (-0.054317128953548816):::FLOAT8, (-1.0):::FLOAT8, NULL::float, 0.35924892568085265:::FLOAT8, (-1.3069424877071532):::FLOAT8) 
    rsg_test.go:125: Last executed (33): SELECT st_affine('010300000000000000':::GEOMETRY, (-1.1348662853470866):::FLOAT8, '+Inf':::FLOAT8, 0.25383724333446434:::FLOAT8, 0.9180836692059491:::FLOAT8, (-0.3217688446483676):::FLOAT8, 0.4312420994796354:::FLOAT8, 0.6904082523092122:::FLOAT8, 0.2608080009062341:::FLOAT8, 0.23003832869400165:::FLOAT8, 1.401298464324817e-45:::FLOAT8, (-0.278168864038537):::FLOAT8, 0.4228772150467003:::FLOAT8) 
    rsg_test.go:125: Last executed (34): SELECT st_angle(NULL::geometry, '0101000080D074E879748BFCC1F461A44CEA3EE541D0C234D60B6DEC41':::GEOMETRY, '0103000000010000000400000020BB7E032666FAC12965E110ECCF00C260D6C5FC2847D9C180611A914B18D74132A88E7161F4ECC1A8E89C4EDC12E24120BB7E032666FAC12965E110ECCF00C2':::GEOMETRY, '01010000C08049B9DCAA3B01C2C047CDEDFD6EDEC12CDE89F1519ADCC194B4FC23F12D0242':::GEOMETRY) 
    rsg_test.go:125: Last executed (35): SELECT st_angle('01020000800700000082D28664B9FFF04182A31957205400C2E66ED093EA05FCC12E406B80893600428682276E3F45E8C1300B8D466A41C441B4958E99DF91FE412893DB321FE1E5C1382ED15A0E3DD84162DFA1BAB0170242A67A61142D4CE3C190E3B6C0333E01C20083DEDBF0FFB1411E812C5E5A25F641E5113A41357E00C2EE3D43078CF0F5C10E04746AAFC4F141080D093BDD66DCC12032995E8E04BF4148820BD78E3ED2C12289501A9931FBC1':::GEOMETRY, '01040000C00200000001010000C0A8FA145B191ED7C1BEC826FE9EFBF641D8FC03856D78DA417A5921651F7A02C201010000C0121351BB4079F6C12805409D77EADE41FA78EE8CEF58F541B864D32999C8E5C1':::GEOMETRY, '010400000002000000010100000008FC99BC2F6E01422086BFA7BF30B24101010000003C83093FF97EE54174A047ED891BD9C1':::GEOMETRY) 
    rsg_test.go:125: Last executed (36): SELECT st_angle('0102000040070000000868AF6E87E9FAC19F0E8F805AAC00C21A66569989DBF8C1144923ECB44AD9C1850A29C84857FFC1AFF613331E4F00C200A3D495C558CB418C34607E3DB2EBC1405F68480079C041E0788EAE48BEF4418E2BF90A24DD00C217932B88D7A6F2C16CD7B379A535E8410AD7873A15DBE9C1600FE49C1454E441240739000109F341E6293ACB478BE0C1CA12D8714C27F441EA7DA624B457F841B0618888A48BC94124311BC127F4F0C1':::GEOMETRY, '010500004000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (37): SELECT st_area('0105000020E610000002000000010200000003000000000000000000244000000000000024400000000000003440000000000000344000000000000024400000000000004440010200000004000000000000000000444000000000000044400000000000003E400000000000003E40000000000000444000000000000034400000000000003E400000000000002440':::GEOGRAPHY, true) 
    rsg_test.go:125: Last executed (38): SELECT st_area((SELECT NULL)::geography) 
    rsg_test.go:125: Last executed (39): SELECT st_area('010200000006000000A8D2854A958200C27D39E49DDBA8FFC15CCCD151FDFFE0417A4664E1B393FDC144E19D468B35F74130F06E66AB80F7C1F87F6D7AF375FF41F234B08FDACEFDC15A934BF7FF86F241505A3142EE1DC9C1B66943634EEF0042B89BE8570E8FDF41':::GEOMETRY) 
    rsg_test.go:125: Last executed (40): SELECT st_area(e'\x01':::STRING) 
    rsg_test.go:125: Last executed (41): SELECT st_area2d('01070000C000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (42): SELECT st_asbinary('0101000080A0A36A7113ACDAC1EEBF3049F81EF6C14502F42B69F3F2C1':::GEOMETRY) 
    rsg_test.go:125: Last executed (43): SELECT st_asbinary('0102000020E610000002000000000000000000F03F000000000000F03F00000000000000400000000000000040':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (44): SELECT st_asbinary('010400008002000000010100008078A6F0A71ECCE841B26DC9DC357EF2C1407ED9F8A63AA0410101000080FE1D3CFDDE6C0042B830BB3242CFFA41103BEB30CFE3FC41':::GEOMETRY, e'\x16c>}\x14!':::STRING) 
    rsg_test.go:125: Last executed (45): SELECT st_asencodedpolyline('01020000000200000058D1094450EBFB419CAF3EE91BE7F141B0063367A1A1E141B82974149D39F741':::GEOMETRY) 
    rsg_test.go:125: Last executed (46): SELECT st_asbinary(NULL, 'U)':::STRING) 
    rsg_test.go:125: Last executed (47): SELECT st_asencodedpolyline(NULL::geometry, 1286636602:::INT8) 
    rsg_test.go:125: Last executed (48): SELECT st_asewkb('01040000C00800000001010000C0885684A35B4EEF4104EFAD206EFBF7416EF486A8B33801C24A058860D41301C201010000C0C0C965DBDCFBBE4193854F6C871EF5C1FE107385EBC10042A0AA881525FBBE4101010000C0E05E0D5CE052FF417159BCF6A91B01C2005F219BBD47CA414C6EEF2DD634EA4101010000C04AAF30B81510E6C16E88DC1A311AF9411C6859EFD84B0142E0E0DB3306A5D0C101010000C0B8CA605DB548FCC190D66AD31761FB41F595DACD636F02C2E596B2635E4B01C201010000C03DFCF1184A1502C2A5B2E4545087FFC1385808DC9D37F6C14C323CD880A6E14101010000C021AFCCEF1AE8F1C1EC944F5159C6F34140A2978A388DD4C148E29459A90BFE4101010000C0D8B685A0FFE8FDC184B487CCCF5C0042247F6C254C87F0413A1B2BB1E62FF3C1':::GEOMETRY) 
    rsg_test.go:125: Last executed (49): SELECT st_asewkb('0102000020E610000007000000A0A4CF118F7B13C087F27B94C31E47C0C0B6C2648C6C3A408041BE402D3C50C0443A9C297D1C4040F70AFF1B87B841C07A9E52BE3F056040544AAC9AEF5648C0DAE60C2DFDCF6540531251741D5A48C0F0F80FD7286237406A3A08482CCD5240ABF3EC6430BE59C0085B1D6D4D634440':::GEOGRAPHY) 
    rsg_test.go:128: Currently active statements: map[]
    rsg_test.go:515: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2183496438
--- FAIL: TestRandomSyntaxFunctions (301.38s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-56041

sql/tests.TestRandomSyntaxFunctions.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20678264?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20678264?buildTab=artifacts#/) on master @ [9bcc3fdacfcaa0775d3b601c356518386d687fbf](https://github.com/cockroachdb/cockroach/commits/9bcc3fdacfcaa0775d3b601c356518386d687fbf):


```
    rsg_test.go:125: Last executed (4): SELECT similar_to_escape('aZ':::STRING) 
    rsg_test.go:125: Last executed (5): SELECT similar_to_escape(e'R\x0b':::STRING, 'D':::STRING) 
    rsg_test.go:125: Last executed (6): SELECT similar_to_escape(e'SN\x1a(y\x19e':::STRING, e'L\x05@\x05<\x1e':::STRING, e':\b&':::STRING) 
    rsg_test.go:125: Last executed (7): SELECT similarity(e'om\x1d':::STRING, e'j\x1bF\x11\x1f':::STRING) 
    rsg_test.go:125: Last executed (8): SELECT sin((-0.25495804311876924):::FLOAT8) 
    rsg_test.go:125: Last executed (9): SELECT sinh((-0.3758870898466853):::FLOAT8) 
    rsg_test.go:125: Last executed (10): SELECT sind(0.30922164230167243:::FLOAT8) 
    rsg_test.go:125: Last executed (11): SELECT soundex(e'B\x7f;G':::STRING) 
    rsg_test.go:125: Last executed (12): SELECT split_part(e'\x1a\x0fD\\':::STRING, e'\x10':::STRING, 5146113138153760251:::INT8) 
    rsg_test.go:125: Last executed (13): SELECT sqrdiff(1:::INT8) 
    rsg_test.go:125: Last executed (14): SELECT sqrdiff(3.685799972027867992E+33:::DECIMAL) 
    rsg_test.go:125: Last executed (15): SELECT sqrdiff(0.3385644850463486:::FLOAT8) 
    rsg_test.go:125: Last executed (16): SELECT sqrt(1.920540553720855416E+31:::DECIMAL) 
    rsg_test.go:125: Last executed (17): SELECT sqrt((SELECT NULL)::float) 
    rsg_test.go:125: Last executed (18): SELECT st_3dlength('0102000000070000003000ACA1884CD74194BA6A0C954FF1C1AE860B01BF2DFA41BD7C3FC6DB9300C2880EC4E73365F741B09F01AA8C58E9C122FC0773262D0142000F24CF93048CC1706D640B1F92D741E8D1CD2AEF9FE741428222A7336CEFC1C8655876FDDDD541B04F89CBFCE201C2D06E2A170BF2D141':::GEOMETRY) 
    rsg_test.go:125: Last executed (19): SELECT st_addmeasure('01040000C00200000001010000C0742D85D2F883D5C1C47AD61A9EAEF5C1BA51094CFBBFF9417CD3297641B8EFC101010000C0F2F927F327F5E0C184C27E834F5AF841FEFF2A3F7ADFFDC126C220EF2DA8F0C1':::GEOMETRY, 0.8717985929341634:::FLOAT8, NULL::float) 
    rsg_test.go:125: Last executed (20): SELECT st_addpoint('0101000000000000000000F03F000000000000F03F':::GEOMETRY, NULL::geometry, 3414801051306116533:::INT8) 
    rsg_test.go:125: Last executed (21): SELECT st_affine('01030000C0010000000D00000068206D238C83DD41249C60046FABF6C12AABA14080820242A83CA0332A4AD541548E04A7B524E541E664523E5747ECC11A21C8C5C426004213472BCFDBE500C2A0DDAE49E0A0F9412362AFC94C73FCC15496C30A223FF141801B0FA04506F9C1E448C4DEFFE8EE41E8A78CD82E85D541CF5864035DB9F0C1EC4A656592160242341BB9FB56FC01428CA384461E3EEC4118B038526F4FFBC17A1AF7CF2039EFC144A1508A3761EC41C8F1E26758EFE341288A302F38AEE4C152E089E3F864EAC12C1BC6113BBEF741D824D60D5A4FF941628A662FA3CCF841FC85359537C3F74186B3F5AEA605F4417C9496761936FB419099CA513A19D7416690859C822FF1410A9C39BF7AB8FCC1B6B2DFEA2EACF8419FD69864324E01C218DA67909D2CFE41A46A8EE3F393E5C1FC43FCD0C7F2ED415C75DCA32378FEC12AC349737B80F9C1BA11090FFC74F4C1F21D44D5802BF2416832A7D98104D84100A5C9295256F9415351892E92ACFDC166A0C1DF2298F441849D609AADE3F5C12CF94DC95408F14168206D238C83DD41249C60046FABF6C12AABA14080820242A83CA0332A4AD541':::GEOMETRY, (-0.9108154221936092):::FLOAT8, 0.6062337218201158:::FLOAT8, (-0.6014789768521341):::FLOAT8, (-0.16776238684572675):::FLOAT8, (-1.0772645165036518):::FLOAT8, 0.162194792568911:::FLOAT8) 
    rsg_test.go:125: Last executed (22): SELECT st_affine('010700000000000000':::GEOMETRY, (-0.14305738059077608):::FLOAT8, 0.5205944355184778:::FLOAT8, 0.8442806274955439:::FLOAT8, 0.4202864791181835:::FLOAT8, (-0.0):::FLOAT8, 0.650621900841502:::FLOAT8, 0.7694081686072571:::FLOAT8, (-0.03705444882912923):::FLOAT8, 0.09901933953677056:::FLOAT8, 1.8131157404629:::FLOAT8, 0.04042773672373601:::FLOAT8, 1.101114465691334:::FLOAT8) 
    rsg_test.go:125: Last executed (23): SELECT st_addpoint('010700008008000000010400008009000000010100008098CD3C7C5F02DA41B863C7962701FD4192EDF9FF6066FDC10101000080042DA7A081110042B4FAE1F3C1D4FD41629BDC4C8941E1C10101000080122FDFD73EECF5418C87A5D6D56F0142EAB1B6419BAF00420101000080E6CCD486F11DF6C108282A7F8CFBFFC12A249C6E5DD9E9C1010100008044A2A50EA3A9E841EAA77B5AD331E6C184962290C8B9ED41010100008010E6D6E1918FD44178A9FDF8BBDED2419ED4DB3E66A1F6410101000080BA25B9D5601001C2348D43B85227E8414417B13F6AFDF4410101000080A435572F4146D8C178FDE99E70C7DA4110FBF5E24649CAC101010000806E3BB2F7C7440142C9071F54FB8302C2F0C28C84965EEFC1010200008003000000CC35025C5455FD4170810E854955E6C184C7B3A3934F01C2380663EC9ACD0142705FCFCEE264E9C1F0328D96041CCDC150B22DA9537E0142B078B634E4E2C9C1C023B029B35CDD41010200008005000000B03A378E5ACDFA41EA76DA1C7A4BF541F80516EA3899D541886AC9C7624BDB41D039453F6E0DF541C54D55F86ED1F4C1F0FD0A3C9331D8C1CC9BE52EC3EBF241EEE16D13C665FBC1787BB2D48839DBC188D2088517FB0042A4D677E8A001DEC118A2F343B7DAEBC14226B30BAC90F6415A919E99FE7D00C20102000080060000006C5A719A3060024210EE1B5883B9FA41F0DF8C1E9544D0C1BC238229C748004268FF542E89D2F84100BA552DEC31B7C1F64C96FA19F6FAC1F6540D2ABBE5F741064ED78C1FA9F5410161B35FCBE401C2B43345FCC9EEFA419DC6BE3F6208F1C18AF29E899792FFC19CA202A267E9E84198230ACA588CD0419CE8E6BDC387E5C1008A131E488A8341E69C4DDF4DEFF1410104000080070000000101000080E0595D676111F8C1C647CC5F56CEE6C13D08864CC15802C20101000080BA59AD239253E3C1FA472F1A3703FBC1C8FAA3A7750DF8C1010100008060F5401937F5D14180B24B24B092ED41803DD34C96BFEA410101000080546CBCE6D18AF541109459253416FB41A83855A29569DB410101000080A87ECFD6C989F3C1D8FD7097A7AFF3416CC23E311D71E341010100008012DEEA18063B0142E0D65CB85C39DA41FDE9B453DE2EF5C1010100008010C4510526ABD0C190081D46F696FC41C0732510A124C34101010000801CE7B4294BB0F9C164342C685D55E041F841BB651C9BE2410103000080010000000400000072DB32BA14BD01C28084596BE188D8C1F87339B491A3DF411650C917675C0142B634B8AAA1B8EDC11844A3E9052CE9415E26A9F6BA1CF941200FD5B97CF8DB41884E9C9DB51AEB4172DB32BA14BD01C28084596BE188D8C1F87339B491A3DF410106000080010000000103000080010000000A00000065FA998672D9FBC1C0BA5B95A5ACD8415A9D994B8B1AF04145871F119C4BF8C190BED7871A70CFC138A656538A79E841B4F4FA424831FDC124C033E1FB97DDC134A93B260CFADDC1AA5E553E3E31F0C142AECA89A05BFBC12FDC5A65275C02C2A8E0F98E6B7FDC416931021B627DFEC1AC887BF9E1A5E1C1C2FDB4AE1D7301428CF757813E95FF41506E28BC1682D1C1D4EBB85CC94EE541F41B27A2ADA4F341800ACCE2D7B0FD412C25664054E2DFC1B4C89F69A17B00427E16668E693FF241847B13FA915AFDC11490D8A0659B0142C002A0F5C62CA64165FA998672D9FBC1C0BA5B95A5ACD8415A9D994B8B1AF041':::GEOMETRY, '01070000400900000001040000400300000001010000406499C2751B5000421302BF20AF1000C2C01D5E95E6A0D0410101000040E077525C1A37D3413E138B0043D9EDC1606C7F1DF340F0C10101000040568D1CFA25C0F341F89969B4795301C2D8BC1D4DDC34FF41010500004006000000010200004002000000E453205CC1170142B6064AC2F833F3415CE9E2402F32FC41306CBBE7804AFD41E4E8B4ECE3BCFF41C83AC728D68AE1410102000040030000007E10B469C91EF341A815A5CE4F20E5413200E4E39EDAE3C13030B30BDF4EC9C1501AEBC3AD88FD41A82E0224292F0142A84137E77EF9E4C14464AB20C6C2E3411C26F73E11C6E4C1010200004004000000FE37948BE8EEF441808DC4C9EC5DAF4105AF9FC2FA21F3C18E581C2B338EF94190D30BF17F06DD41B354ADB0AB3601C24838221C17FCED41880BD4E8BBCAEC4180AAC835471FDEC1C8BCD1F636CCD14176A9C1FE36BDF1410028EE61A19ABDC1010200004007000000AABAC29D8E23F6C19630F0630B5AE1C130A0C13B46FBDC41808776837A81C341B07B7EF1EE1DF5C1F0C1A03EDFF3C7C1D87208EBF67AD8417575D203715C02C2B468561E256DE141CAB838D79C240142D7003FD0C2EAF3C144304C76EFCCEB418C638986493AEC4180660DF4B409C741B8DF02B9D7F6E5415847F5C370CCD7C17E9DCBC9AC180042FCAF7E105A6EF84130F2E7EB1EE1D1C17A87E0684712F541E0C721A1384CBEC1010200004007000000D8B35AC95600D4C1064A3F83B0E3F1413760447257AFF3C13CC255AB6E8CF9C1915A873552D5F2C11AFD5DE3D779F1414493CD91F13BFE41E69A75B79C6AFA4157323EF1ED2A01C23AF83B8AA89500420814C4271BE4FC41D77CDF048692F2C1A8753B78EEA7004214770C7A9A05FD41800F42B514A9D7C1F08F308549F0F741B04DF949180CFE4155BF0FD6BB5B02C2E0A70AEEB6CBD6410813A3B25E86FF411EBC8B56297402420102000040070000000163114D9D08F1C18DA950B9604EF2C11C3508E84FB8FEC1271F5BB9385EFEC1931DE80C074900C204EE69C2BDE9FB41ACD7D5DC7038E9C1B55EFE4AF9F0F4C1C4AA2729192DFC4170F0E5706F95D24118E81A0E5210FBC1DA397D0A30940142766A972251CB014200E10456181FE3C182623924CCB5ECC154BD25131AEAFC41503C7E902B0DEC4174E9AFB97253F8C18B593AB7ADDAFDC128454B7FEF52FC41E035E78D188FB641010300004001000000050000008020BEA9B18AB64168D877EE22B3E2C1DA511CE32FAB014200726653BBEEEF41965F27454FE1F44110F9BA150D85F0C18C1A192CEBB5E94138DC63C2029BFA41702BA93B5F66C2419E681F60143E01C2D61643166D9700422C7EF8CB7F5FF0C18020BEA9B18AB64168D877EE22B3E2C1DA511CE32FAB01420103000040010000000D000000926E931D246BFDC1E608E13E0C76E1C16819ADD34DF8DC41287363D7B24BD1C1208338C32680C5C1F06E61D014B5D0C12ED41934DC82F2C14A942E6F6665F6C10847B5D46F4BF7C1924982CB1064F2C1DC9B9D938312F9C1772D61A2A79200C20063F30874A4DC41B476E5E01148E7C1287C2FC22188F8419C1B0F3E6378F441E0383D847EE3EFC1A641B8BCE78CF5410CC43387961EFF418E17BDA3B05FECC1506924F13B33FC41AECA99D9819AF441E0651619CF22FF418038752E86F19641B0DA2C6A4519D441CE540D31A250F6414068EA922B99D2C1101238484BD3CCC13848A694CFDFE241508A4237F4BECCC1CC64A04C7C04D7C1E065BDFA7448DB4177B7F74A3CD0F0C1BBAB1009637601C20801CB13B04AF941E444740D9DF6EB41926E931D246BFDC1E608E13E0C76E1C16819ADD34DF8DC410106000040020000000103000040010000000C0000005467B1082F41EDC1D66CAF91E81900C2B086D55E6D1EC7C1707BAF6BDCE7D0C1A93C8E69E4A300C2B3002716E0A500C28095E80280B3D541D16045513E45F7C1B41DFBEF8AADFAC19C4A503756EBE94105939F8538EA00C2E02F73566230FFC14C991651E5C3E84139E983E03C95FFC1803C01FC4E7CACC13070F0831B63D241087DAA4444D1F0C194C1BEC28ED5FF414C845ED4E1DEE941E82A424F03E8DD41206AA5696E33DC41B470A38DCA7DDDC1E20CED3B168CF94178D9304840C0FE4100EE18C7CADDD6C18C30E5741672ED413308148ED3A001C2C831BF9E8EE2DBC13C15DF1D4ECDE2C1E696167CA2A30142A80523664ED8D4C13CB624E6D3C9E5C1083B3842BDFFD8C15467B1082F41EDC1D66CAF91E81900C2B086D55E6D1EC7C101030000400100000009000000A9C2EAB7D637F1C1D75ADB25D72FF0C180C7A40B5328DB411094E46BA437D4C1661E1BED7097FFC1046FB2C4C35F0042D0F607FBF858C1415664D0CF46FEFAC130A38A509D77C5C110EE2EEF8ACCD24159A8AC80091FFBC180252F3B43D6A9C1908E4CE7B1A9D641F2127B4E270BE6C1B8F234994929E04102EA70AC4BC5F4410018EDF0F88265C14CC7B4DEFC1BFE41001003598BD0BD4140010350BA5AF741B0AC5D1DFB15E241B2E7990AA173F0C184E1BAEEDEA200425795A513DA8001C2A9C2EAB7D637F1C1D75ADB25D72FF0C180C7A40B5328DB410102000040030000009009AF364A84C5C14D57013D7D59F7C15CA0DB5F6813FDC114C50A43B86CF24160226A7EFBE0D0C1308CA278DB33C6C100E357964CCDAC41C8B3A0A893F1D941356D19D9FE8B02C20101000040F6740175493E0042C42C5765D13702428EEEE32BC37100C20103000040010000000C000000A2FC40148182EDC1001ED63C2159E54100AE151CE82FE24179198F92B3DBF1C1DC6F960D3EB2E3418C0C084D306AFF412E1CCBEC8B26FEC1803E18433C3497C1C0FAB1946567D2C1E0B498BB8B71DD4152C1F053A497FCC18036585FF618C84128F905CFC998EF418012981DDEE6D94130B4E723E3FBCF41E48F30E301B0ED4100F2AE16ED6BE641545F71085A8AF2C1A096124450E9B141B8AB2FFF24BAEA41C0FA4F8B9BA7B9C13039E23B7031E141906F9507A5DAFD41DC3491CA22B5EE414927FD65B8B1F3C1E43545D7B1D8FA41893ED74C69BA00C2D80CCFD4DFCD01C2AC1C22D853B601425A7740BE82AD00425BB23FF5AE59FBC1CC53E9873F5CEF41B445C19AF9E5E4C1A2FC40148182EDC1001ED63C2159E54100AE151CE82FE24101060000400600000001030000400100000005000000E03DC81E9B0400C206DCA4589549F7C16435F8F456B6EA418E30C6E03649F141BCE5D3B7F1D4E8C1A804A9E872ACEB415EF31B02799BF841AC572205CB85EE41D06B4472375DE241E8D6599D8E2CEBC14CD2F34B29CDE941FE8017C9021EF8C1E03DC81E9B0400C206DCA4589549F7C16435F8F456B6EA410103000040010000000800000039E476ACE37AFBC19897B208A0B3F6C1A47FA112FA21E9C1161C4150FB8FE6C1BE94EE150990EDC1A864A3C27446D4417CFC208D7FA7E541C7D06FEA1C4FF4C170EAD6867D05C5C1BE51C917ED67F74147A009D21EA4F2C100E43128740F8D41E0EE083424F5D141D0E409E6E91EC7C19CCA3CFBB883FDC18C951C8697A2E14184ABBED946CEF1410CD195394751F641FE41477F6BA1F6C11812D8569019F941880FC1C65380ED4139E476ACE37AFBC19897B208A0B3F6C1A47FA112FA21E9C10103000040010000000D000000E4240CDC041DF7C1A1EACC6C5CD0F4C1263E55985291F341CFF5FF4971F4FCC1969E0907251C02C2EC4981CAADB3FE41D83CC319AA4CD3C12888CDB470B0F6C122DD068ECF01F8C1E2C16DE24D26E3C16857749B767E01C200E5955DE081E34108BB6EAD0C7DEE41DF3D61952184FFC1F8A8F2C85FF6E141E09978B9A60BFC415512B7A3CBB300C2C60A3E3F920BF6413EEF74187D58F64156ED8EDA3694F7C17AF79F7096A2004298F20E344F74F74130F86DC2BCB8D641B042A7E329F2E34140A5E41FB9FBFA412A5528801509F04190CEFA491C7F00425A2BFA928C71FA41C89352B9C31A024280BC8713D39BDB41E849F9853DC8E34104A6B7EB08F2FB418C8078DCE6ACFE41C4E155331F9CEBC1D4316D45C1A7D5C1F82B772E785DE841E4240CDC041DF7C1A1EACC6C5CD0F4C1263E55985291F341010300004001000000090000001E38FEAA5C82F1C19E81606E7E0BE6C1EE54E872A079F441E098E2FB6EDCFA417D2E33A91F65F2C17A1EAACBF8DC01C258AFFE8CE62DF041285E0EAFBD2FE64178E3BE5DE370F2C1E4E14719FC890242DCD2B8ACE0D8FA41FB512AF3086DF0C1C0BAE17A7B7BE741084E834F6EA8F2410B91D25146E6FBC1FE911F9082DCFAC1F0293C80E43B0042769170A426CAF0412B43F890DB59FCC1D85AA8E5AF390042B0E9397B8C17D0C199E2E0025C6400C20E154A300942F141641506500BBCFEC11E38FEAA5C82F1C19E81606E7E0BE6C1EE54E872A079F44101030000400100000007000000F6F662815B1AF5C198C265C9252DDAC1C844A149F0190242DC69FB510AF9DEC1FA7412685F6EF5C1E8258932206B02C2E476AAA0DA28EB41F64AD2132B8AE8C1CAF8AF654E43E0C1D62AE7431435F9418687F065A1A0FAC100F77799570DFEC1223A3AB7ED7B00422E2232D2F95301426090877C19BFE241A4E7C5E1CE82FA410AB323715D0B00425A500FD91183EDC1F6F662815B1AF5C198C265C9252DDAC1C844A149F01902420103000040010000000B000000866EA536EEA2FAC1BA157FA2747AF3C1B0B67AC7F3C1D741A03BB768BFA8F3C116840970837701C28AA160C0C9B300C23031EA71A63FC5C10466DA499D1CF1C1C879E13DF525EE411C580FE0A9DBEA41C027BB4FD1AFC8C145DF0A9E3A9FF4C1E6368BFA3813F241006E8DE26013ABC1F0381E1058E2FC4146203F527513004208D66BC863DBD0415C78C5C90C52EAC1383A22EB052F0242E4D873E71D51F74120D293DDE3B9FBC1CAC0794D59BBF74186906530FFE70142A8B1CA60F504D441840D3FB7E7D4EA412A94873DD1BBFA41C0C66B069CDCED4103E0BA7D35CCFEC1500D0889EB9BEA415CBCD5FF8DB6FC41866EA536EEA2FAC1BA157FA2747AF3C1B0B67AC7F3C1D741':::GEOMETRY) 
    rsg_test.go:125: Last executed (24): SELECT st_angle((SELECT NULL)::geometry, '0103000080010000000600000012F04AFA3062E6C11CF7F4DD756CFAC1B6F7E10055BDF04180D6449A2FB9EB41A60EE2B97925FCC1C09B4C61C45002C2A0DD1EADF19C01427472CF149F9100C29E5EF1EDC035FA41989556C45DFFE84160AF06ED00D9BBC1E07C70906B41F3418991B18E4021F9C11E2E2A0FACB2FA4190EC2A26728CEF4112F04AFA3062E6C11CF7F4DD756CFAC1B6F7E10055BDF041':::GEOMETRY, '01050000C00900000001020000C00300000033ED90AFC78602C214A45B2103F2F9C12CF22EF7EA3501C2B4BB5CEF1C6F0142904A5DAB600BEA41FAC5F0059057E0C11C3528A9AD4700C298088C04EB8CFBC134371517070CE7410015CC8592ADB241365D9ED8248DFA41007E92AD718790C101020000C00300000074DC3D5F1024E341BCC502F56039D6C134693907ADACEF4168BFC4A3B351FE412A120258EB57E5C1A8E8E1120EB3FF41FABDE0DE237FE9C1C659F83865BDFCC17F1580153A87F0C1409D95FD953AB7C1608F380A7E61DBC1A2A0312DB530F24101020000C002000000C59AA9499FE1F6C1EF7A3311E91602C2F8A6AFCF07AEE3C13468D21143FA0142DE241A7AA7F9E7C15C60861E7743FFC1B0CEE745F11C00C2602C2A2AF377CD4101020000C003000000A6CEB17E5FBDEAC110509D260365FFC1AACEC301EAA401C2D85AAE1862CBD3C1925E578DBEC7F441E4FEFFB02CF9E4418A6CA9641E8A02C27584100C6F1600C2901C384EF50BD94146CDD96146D7F141508702362013EF4132F042F1285FF34101020000C006000000D0D8737D2299C841C016E48217EBAEC1506332D39774EBC1A0EFC0B73288D8C1F07B44E98653DE412A76CA490A11F041B49E1C98A5FDF74134F8E220534AFD4178CAD1683440D4C10AD87CF795B3F541F60721281899E4C1EC5E5362ED6D0142CB7EDE868C39F9C11AE7DDA1CEAA0142F217EE8C5DE3FFC13CA0A5904C6AED41ECCE70C20C5FF7C18E418E90A20DF4419B00159FE40101C2107EC38AD2BDF7C16586DF2A324700C2260104807508F941907D41A61EC6FBC1D4ED57BA1DF9FD4101020000C0050000000315843B139BF9C165F296EDBFF300C2C84123BBDFC9D74174AD7728740002C23C90DD0EFFA8014260FB28D6808AFCC16061F5B7E995FBC10012801BB4EDA04134367ED19E9CF84102A9D43A7CF0EBC1C0239EF33A85F3C1D7E6B937574AFCC19E7C727EBD20F54182A086EC69F5F1411C92F496C9A5F4413E46B4A2591502C28087A2E01DF0F94180EE5B61E6B7FC414A14BBB7B5B4EBC1366ED0AC285F00C201020000C007000000C238F55502BB01C2A4D0558700F1F2C1E0558FAE6A78F141FA1150D2EAD4EEC1E5A6ED55016E00C2E3EE10A20939FBC18066C56070D294C17C70EE782B40F041B1381DC35F37F1C1D7BDA8673BC2FEC100E86C5023D1C2C10C2E097F7759FB41A0DD788D2553C44194628C2E4FFAF0C1EF1EB553EB7B02C2489698589E0DF3C1CC4B6A497B5BE44178473371A126F7C1000A7691F7EEAC419AA59BD5F654EEC1B8628B4FF187EB41586CEA23A3ACF6C1E095755BBF87D941966628D0A21CFA41500F26205A1ED84108E573C9FBAED4C14CE245D58F1701C24C955B4B9A99014201020000C002000000F454406B438FF641C852D9F31711EB41DCFFE2D7C951D8C1ECDED9D1D300FA41907D605BF1E60142601E46B4ADAF0042563DFCCB5B3F0242E0C0D2FE2D1CDC4101020000C004000000064A93D8D968EDC1E8863903DF74F0C13877037A60E9DF4180F8DB1F04FB98C100A707F23A0993410CCC937B7162EFC100B41AA894A6CBC1167D94C66C7FF541344C84AF728DFC411825CB2C52BEE8412853C6A987FBD041F02364558C7FE541DA4D63DBC087F941AA861E4AEFB6F941806013CDD150EC410BEE1D12C7F401C2':::GEOMETRY, NULL::geometry) 
    rsg_test.go:125: Last executed (25): SELECT st_angle('010100004018417F221454FBC1F65945C0696DF041CE6573A216F8E6C1':::GEOMETRY, '01030000800100000008000000D60B2F749136FAC1AD4E1688E536FAC1D0783184E7A2F2416869AA65EAB5F44139DA28E4296901C21C2540747402FD410CFD78196D61F4413C8DD4EB5C35FAC16C0C0B6E344EE241A89F7BA845BDFA4120F0422D4583C4C1F05CAC07E55FCEC1285DC1E9E7A50142E05951584E26D6419A8BB13E1F710142E4C83FE9B5F6E8C11405725051EE00429A878714F6EB014239E654932E11FCC1081DB4CA7800DC4108178CC4ADB4D3C1D60B2F749136FAC1AD4E1688E536FAC1D0783184E7A2F241':::GEOMETRY) 
    rsg_test.go:125: Last executed (26): SELECT st_area('0104000020E610000003000000010100000090F582C018135940CC1E4DF088F8534001010000007866EB74B4AB4CC0D652D56D1A594040010100000040266BFDB3FE2A407CD5053D6DA85140':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (27): SELECT st_angle((SELECT NULL)::geometry, '01030000C0010000000B0000000B716B5C074F02C2F8809895A9C2DB41746F7072072EFFC1941B40BD1B55FFC1E2220F32BE2CF6C136ACE879527EE5C10251A01DE7B2F5C160736A14FFE1D941B81FE3C49335D5C100C8F9A71AB0C1C1C4621EBF1CD8F2C1B252ADCF558100421A972766AFB4F74118C0D1674020FDC1FA7AA1CD8372F341E58C1C684680F7C1E058428F1CBDE4410095A99A5058A541C646A3D7E50DFBC134C8833D327AD9C12A9A132EF9FE0142F463E05801D9FE418803762D6BDDF9417C3AE295D002EFC132D10B59F543F241204F3BBFE1D7FB41664512B4ADBAE8C180EAC426587F9CC1103F4A79A579C8C1AC4D0F1E9D5CED41DCB283B34FD4ED41FEAB474EE9850142B48445221F4702C21473004FC625FF412BE6D22E0189F6C1CFBF8EA0689201C282521057C33302C2D09CD37DF287DF41A8ADF63561D5DC41F07E74C88A3FF5410B716B5C074F02C2F8809895A9C2DB41746F7072072EFFC1941B40BD1B55FFC1':::GEOMETRY, '0102000000020000002810024B171D02421CAAFFFA14DD0042F08F3B42549FC441942F8440ABE4ED41':::GEOMETRY) 
    rsg_test.go:125: Last executed (28): SELECT st_area('01020000E0E61000000300000010EE1C40C90962C0541CB44DF94239C0013E49511B5C02C22C76980C24DDEA41B4B9320000E26240CF7AA444997156C0F865CE8B205EDFC1C4BFABA96CC2FA41A8D6E188024D65407C1431CB85C6544000824E7AEFB2C741DADA3EED37FAE0C1':::GEOGRAPHY, false) 
    rsg_test.go:125: Last executed (29): SELECT st_area('0105000080090000000102000080020000001079DCE97AFCC8C14EF66A49951DF4C1404B74AC2BF6E241E02012A85777BEC198BB161EF44102C240B5814B904BC241010200008004000000BC1A23632C9CE941502D786A2E910142B8BE84D1D232F64110FE4D22DD9ECFC16C1D701F0969FA41F49854F3C020E2C1B55CDDD6A9E8FEC18687D5B5D54BF44170179D299BD8E4419431F652A2C5F5C1EC746A930E98E841A60ECBFAF879004201020000800400000040346B3FFB39D5C1A02D66F84EDCC9C13B4F25EFED7BF8C140F1FBCF80D5C9C1C00F67DF93E1CEC1B0FA4D7B579EFA4100F88C32B7BF95414A619FA7002AF2C1A0C7C8DC54EFB5C11C1E4784F6F5FF4114F2B5AEA9C4ED4197B78DF4E8F5F7C10102000080020000007B6B97B2E47600C2D8D68A95DFC6FDC1806E02756213BDC1ECF245E0809EE541D0C32F8E8333C6C100460B89EC7C89C101020000800A000000A4E5A30299B3F74140C32C8620C100C250FE5CD813A6FBC1045FF3E92B1CE14146540E8A744CF0C1D6838DB1EFBBEBC19402090C8235EE41144A79A19309F5C196648A18740E0242381DECBB3C6DDF4100227F44828C7EC13C5880BEA4C8F441706A489871DDD24164E26FFB349CF44190FFA3EA01EED541B0948EC1013CC6C1A451DE92BF8DEB41480D81E2A19DDC419C81294C0FC1F5C152885D7B952FF541FC42E7AAC3F0F541676DF481C45BF3C1E8A1F1489C56E341F02548AFB603CAC170D4D68C20DCFCC15C7B2FDDD894EE410847B93C0E6B014268557583C751FBC16406261F30ADEA417A2C2E57892E0042010200008008000000629A70EC939AFBC159B6E9C6FCF5F0C171CD43484DAA01C243AF6CF7F62EF0C15CDCFCC8CA00D6C19013BC54C7C1D8C148F82DAF7CACD4C1B84838BBE819FAC1700C68DA0C82F341DC89CE7BDBA5014248EF89C480A4D741C807A1707298FF414871D348EE13DC41F879B8B277C5EB41AEE0A6F1B09EFA4120CFA6267373F241DC6C34090C01FD4112E0B5E91A5CECC1F490B1F190DBF141E4FAD31393020242A88673FFB750DDC1BFE48AE5D19DF3C17C54C5208205F24170EBEDC4876FCE410102000080020000000ADE6781E5930242A7D343D8CB22F3C16828CBF5D5A1DB4164FE4F36B265F1C161E7C1FCA3C3F2C1081BB9945F5ADAC1010200008003000000444EB9A78B82ED4185DD3196C6FE00C24C2104D718A101C2384243F9AC96F04110FC385130CAE6C180AB9434EC810042BAA1C28C66A4F841681D804F88E7D3C1E0EC139567F3CFC10102000080020000001876CBBEA52FF741E2A0DA360AE9F041DB9172F6419EF5C19252F74BF395F141F4860B3C0B35D7C1E638A00C0DA7F141':::GEOMETRY) 
    rsg_test.go:125: Last executed (30): SELECT st_area('P{*s[5)':::STRING) 
    rsg_test.go:125: Last executed (31): SELECT st_area2d('010400000004000000010100000000000000000024400000000000004440010100000000000000000044400000000000003E4001010000000000000000003440000000000000344001010000000000000000003E400000000000002440':::GEOMETRY) 
    rsg_test.go:125: Last executed (32): SELECT st_asbinary('010500000002000000010200000005000000E495036096E2E541CA94490952CDF0C1F4FF2E8B9AE5F2411260C636B738FDC1486FABFAD468FE41EEF5FD124B84F7C1BC2F415B9D44FB41AC528D157238E7410C5E3A3082190242547D0799720901420102000000090000009933A139EE7EFBC194AA296C8E5BF6C13A9EEAB98272F5C13CE08EE121C9FFC15D661FFB077AF8C11626403356C101C23CBA8409EABFED41A6082F98883900C22C9FC22A8D49E1410758D8B631D4F7C1A053FB516E42F541B6F7ED85DFE7FDC112CC8F515F13004282303249AF9D01C2ACD1602849BCE5415E56348FAD40E1C16EB070355F15F84150D7764F6141ED41':::GEOMETRY) 
    rsg_test.go:125: Last executed (33): SELECT st_asbinary('0103000020E610000000000000':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (34): SELECT st_asbinary('01060000C00900000001030000C0010000000B00000052B946BBF60CFFC110E8C7610121D0410ADB64E27244F2C1AB7AF4DDE80BF5C138B3E5A7B5C8FDC19AF61A0BB4D8FCC1268975DBD2C4F6C1B8EA920AFF44F2C1110DCE166BFDF1C1B4A5C998424BFCC164E048E2A83BFA414A0963F8F523E9C1E0F98D453C1FD6411C77596DDB98FAC1945CEC96114FFAC19A01D69DD4CDF0419888A3334FDAFC41C0D114CF7C52A9C14A2505D2CD90F841D8AC2ECCC242E841B6E1D4358562FA41E050BE4ADC32EB4192D3DEA60203EDC1B75D68B68EFDF3C1649582A1428A0042600EC336A2B50142C8EB89E3C40D01C240581104D85BFD41E047F0B09BC3E2C1BACC2563139D0142C09F92AADA46DBC1271D0B55908702C25C5105853B72E5C17A6777E8DF5AF241D8FCAA557693E9C138767AEFF484FBC13BA32A1EF5B6F3C1EC6BE8BA1C22F64199F0D5D5E14D00C236ABBE81E134E1C152B946BBF60CFFC110E8C7610121D0410ADB64E27244F2C1AB7AF4DDE80BF5C101030000C00100000009000000DABB779E9810E5C1A028F18A6E6DD5C1A641F3C45F0AF941E2E201964CE5F64168A2815C44BEF3C1B7DBC91EBB39FFC1A0A80FFB63CAD6C1F6379818AC3B0242342F7893B326FF4110636D12DB60C341BCE88477D30AFE41CCE72142F43CF0C17C40B6ACD33A0142F06E271A642DC64150C77F92A519DE41824ECCB043450042508051833555FF41AC3641C7483AE541664FC3839848FDC1D09B9BE145BCD1C14A6086B4DB38F4412611FA098CEF01424360F7454D02F5C17ED1CF61E268F94158A95ED1B6C7E9C1E0EA88FBB603DB41C26F845419680042F2691B800632F1C12C153EA57DA5F1C1382E021CDE89D641F83553B40426DA4140B809C89DCDE3C1DABB779E9810E5C1A028F18A6E6DD5C1A641F3C45F0AF941E2E201964CE5F64101030000C001000000080000001D23D6C65B4FFDC1943F48C724B7EFC1A9C710EB8582FDC13040E3D7C370DBC154812E6008FAE5C15A0E9EFD4FA6EBC16031012C4E23DE41002F85DB13E8C241B008C2B536A1D3C1B2DDCDB5D99AF5C1C0D5785ACDA9EEC168EAE7064F9AFA41E89C242ABEF901429ED8AA6CDFBFF5C14FF5614727CE01C2AC0DF66CF9A5F4416C55FBCEDAD0FB4170CA459054C7EEC16A1EB1BDF20BE6C1E8732808D7E9DA41DEDAC479D4E40042400268971409E64100B1693A68EDB1C1F88C4224E773D441F4C2ADE757A8E041609FB728B7A3D741A20F89B5B313EAC18884433644C3E2411D23D6C65B4FFDC1943F48C724B7EFC1A9C710EB8582FDC13040E3D7C370DBC101030000C00100000009000000DEC9CBBCB176FCC160754ADAEAC8B941895407DA89E5FBC1046D9015EBE8E74100BAAA2B8441F5C1EB7DDAB9623AF8C1363968905439ECC1708B7C42096DFA4182C3D7D95E14F7C114E6932DCAB1FDC1D6BC35BF25DAEEC144D649CAC6E8FD41FC75BA680F0FF041E0CCC3C3DCE5DEC140AAED4A03CDD141D0E4502C3CDDF34108A3C24C7D06F241B074DA9E4342E24154B2ED373C26E341502060276ADEC6416881D7CD1220FD418E43A762B112FA410034C86F44A6FDC1F5C91AB4587400C2E471FEA1F29EFB41F0CD98653B15FB419C3A5D0D2B0801426A5E747047B7FEC140D13A80789FCEC1C4AB1FE1F4E5E641CBECB011F5B700C23CFA8FB3F49CF141DEC9CBBCB176FCC160754ADAEAC8B941895407DA89E5FBC1046D9015EBE8E74101030000C00100000004000000A05259447807BAC1C02203D6FC1DA7C1A8B2AB8EF11CECC1C62A299D80910042580F64E3581BD94180DD643F2414DA416AE9C1DC02F60042A827F595CFED00420C4033B779DDDAC1B884E1B8B190E841CC82DA2D4924DDC116DB561698A4F641A05259447807BAC1C02203D6FC1DA7C1A8B2AB8EF11CECC1C62A299D8091004201030000C0010000000700000082B40AEF363802C260805697EA11C6C1C83D43931A42FE41808B1A65D1BAC2C180C157579CC8A0C196B6E51A58AC00C20C0E1D5825F1FC41EE34B50EB58900C2FE7FA8A9806CF041CC0DE56B804DF2C146B35CD86732F9415243E103756B0242F2815F5F34A1F441F0CD76A0D270E841BE4B08DD21D7F4415FF40ED9BEB200C2DC5A533EAAE9E6C1E481D030138FE441984AAC6ABD83E2410AB5443D8BB9F8410B168EBF01BEF1C164A1288749CCF041437699C6E88D02C2589570B55A5ED4C182B40AEF363802C260805697EA11C6C1C83D43931A42FE41808B1A65D1BAC2C101030000C0010000000B0000005F5177E36EADFCC1309E6A70ADEACBC1D6937F394D4FFBC10076BD82059BDD415A66CD821BE800C2742D4525EECEE5C1E00E7B1CF2ABCDC172C0CC72CDC9F141E41D40FC24A6EFC16E9017A04DE7E2C1C84058ADF72ED0C12CC0629AF7B1EFC1708668A4691CE1414AF3EDC27EDAF3C10ECF24D7BD1EF9C1F842388B0F83F141F0704F59B8EEFF410696763A2D63F5C1C96BD33FB80AF3C1EBD487AE3621F3C1509D2833D58401423A8114E892CAF6412CB4812A841CF6C1806571771250D4C198AB6C3C362CE8411A41B4B6813FF541280135735CBEEF41A82976A06851E641D0A2F6D6D4B9C8413036B5C318DCFD41004E2D8B427DD3C1E2A3FF3F9DC5EFC15E37C4CBB1C8E8C1FC88AAAB1DDAE241F6DF49FAFF0AEFC10038FE7AB5B8D74109B803D80C4D02C24621BD35F704F341865D4BDA0DE1F6C1E06B777F420AC3C15F5177E36EADFCC1309E6A70ADEACBC1D6937F394D4FFBC10076BD82059BDD4101030000C0010000000A000000C04E594D65DCA241F747BA7E424DF4C1863074617B04F1C126728437F85FF2C158102623BFD1D2414C8237D1DA6F01C2CAA3042DA500F3C1102DC3301682CDC1C08ED300C23BFE413AC9746B2D91FEC1DA52C3C9D557F8C17BB0657BFA54FBC1A89A1D11725BFE41960531575101EFC1D8CEED76ACEBFAC130D71DB6FBF2DD416C9CE4A71F950042D0710BE475F3EC41C8E5EBBBAB14D84110EB015E84A6CB4184EE3E7FA4E6FD41781EF0AFC577F941366E3C5B3A960142E04E0EE8508CFE415061912DF962CB41E883D3345902E241986F7C2DBA84F6C1167B72A3A11DEFC1504DF116F64AD641004B481687E3A2415603C2649C07F6C1B85DED488AAAEB4140A7FFD253E9CFC100D39F8ADFC2BCC1004138641FDAA74152E4D8727AC00142C04E594D65DCA241F747BA7E424DF4C1863074617B04F1C126728437F85FF2C101030000C0010000000900000040D588EAF029A2412C49C3643A60E5C1DECE13C33012014222742BB7A954ECC1DBF121DA124FF4C1E24A2C257D6FFDC152FE10E9F726FA4114C8EFFC559CFB41BA2EAE1A367DF241A2811F8D56E4E3C1149AB21D7B58FE41147D53143FEDF4C1D47A4EEA76B4FC411A8236A0E6F3F9C1186ECA62D549D6C10FAA1CED2F15F0C106D9B3ACA00802422ACF0BA0CBDCE6C110C654ADA995ED410AEBA061DC29024284AF6A17CC97FE4138F63763A5D9F24106AA8638EABCE8C1000E31823EB5A1C12C9F3D67D08DF441B0D19E671A85FC411EB3F2F9820DF241E8021AE3D514F141B80D6F56E269E34100A469E1108277C1FCDB15205AB1DDC1B0C176790022FB4140D588EAF029A2412C49C3643A60E5C1DECE13C33012014222742BB7A954ECC1':::GEOMETRY, (SELECT NULL)::string) 
    rsg_test.go:125: Last executed (35): SELECT st_asbinary(NULL::geography, 'bJ':::STRING) 
    rsg_test.go:125: Last executed (36): SELECT st_asencodedpolyline('010600000000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (37): SELECT st_asencodedpolyline('01030000800100000007000000003809E408AD734188AF6308F275F5C1D8A7020F029DDE41A02BFBF75103C1419BB08939B990F8C1ACD705730A2FF6C14EDCE73622A7F34140412E32439DB041242B5659E105F5C1388DD65E2D3DD6411A7F1EEAE283F341D2BE2874C5A0F6C10079B0D1E8CCC941F019E65F4917F8414458A244C32AEB415C8E62E721C8FEC11E30E1D2124CF441669F2EFF9114F8C1003809E408AD734188AF6308F275F5C1D8A7020F029DDE41':::GEOMETRY, 1844037865:::INT8) 
    rsg_test.go:125: Last executed (38): SELECT st_asewkb('0101000040B3BE940484BEF4C198D5197711A8F141A294132A726BE9C1':::GEOMETRY) 
    rsg_test.go:125: Last executed (39): SELECT st_asewkb('0101000060E6100000E01C168D2E6D644046B121A80FA55140C8976A2B1CC3D1C1':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (40): SELECT st_asewkt('010700004001000000010200004004000000B85C746862D7D24130784B9610B6C8C1A22F3366D80DF34140CF51A20E49C5C1003AE2DB985E914190577A6263B6EE4122891516B746FBC1922AB7D408A0E0C134926384A006EB41D04A76980ECA00C2DCBE8292C88AE0C1004A92DD61C8AEC1':::GEOMETRY) 
    rsg_test.go:125: Last executed (41): SELECT st_asewkt('01040000C00800000001010000C093F9075742B9FEC185C431FA7628FEC1506DB6865779CB41E63F83C2F040014201010000C06EB0EC780BF6F9C1AC2CC0B3FF74F5C1C1C47292E348F5C1307DC1DD325CFE4101010000C06F3E678ECCCCFCC114C6885C9B760142305DBBBEA5ADD0C1E283F7372001E4C101010000C0B66126C9B77AF6C11080BBB70DB0D1C154A53A9F36A3EA41740B1E59EFECE84101010000C02EB539F91492014206CE14BAB007F9C1C8510DFD8BC9EC41B87AC7C7A51BEC4101010000C06ED1343336770242CF732F178B61FCC1904AECBE3CFCFB41071FC1148E7700C201010000C014AFB182D048FA41676F3371527AF0C1D3108175005EF8C18A959BF2FC51FBC101010000C0FC97421AD107E041866624A2DC34F741CC5E04D8E756ED41E4151E8391E2EEC1':::GEOMETRY, 32767:::INT8) 
    rsg_test.go:125: Last executed (42): SELECT st_asewkt('0107000020E6100000080000000103000020E6100000010000000D000000FB0EC09B861D50C0DB64DA60522540C00054ECD153D01EC07295731CFF8341C02C8E328B4EB66140AB47CD773FFD53C0ECD309A9B5A96540FD774AEB9E5254C0C04151970A946340A76B8645D2774EC0A015B909F99D5A409C0592AF416E47409CC625C09BFB58404CD2780F84D94540A070989261EB3F407E321C01B52B544038A3F0FFC6C93D40185D52F956764640CF67DC68AD8B63C0EC4B8DED3D6A5240FC3923F3614358C070FCC04A46D148400701560FEC1A53C07440C0B088A73140FB0EC09B861D50C0DB64DA60522540C00103000020E6100000010000000C0000006CD768F7C38455C0E402D548AB2B34C0276200DBD56F52C01067E6ED0BCD52C070BFD846A54A25C0E761752E5A7053C050A3A7F211ED2C40DEBBB5DFE40550C018D8B3628A5052405BEEC8B36A404BC084879C69EC9063404050C9E1C0A355C0A65DB81FDD956240E872D4B0923522408C196377578355400E9D49DBAD23564040F21F2D43883940EEEB3DFAE2F2504048063F4DDF175DC0C0B68A35BE69364093B291553D4565C020BE08874E2E23406CD768F7C38455C0E402D548AB2B34C00106000020E6100000060000000103000000010000000D0000008EA3445B447C60C0A71D2559BB3341C093D8309366435BC08807610E14F644C0F2E88942D13763C033F41529AF674CC000F60FB5373A61C05DA1BE5D029C4EC052C96FCB61FF58C0087B2900771E4CC07E4737BA76366440A3C3F431D1CB55C08A7C3C2B20416140990E374E3D564CC0ECABCD4595D55D408CE96AD154C33940D002CC48F0CC5140D8FE954B53A42940CC5B2F44CE8562406267B87F9420524054BB389CA9C85440809D417FCC66564076E9FF0985C955C018708331419B45408EA3445B447C60C0A71D2559BB3341C0010300000001000000080000009FD5ADAB6E3057C0E0BD69BC50A239C0E657CD4302245AC0563C1D6A0E1E42C038A4B50DB8F430402287B17D4A0B47C088622F49148D5F40FCA9F0FD81C953C09880222E3DE54740E7A3CB3FA80942C04C3384CAE2C24640D83B892DD5DD2BC0588B803F01C8594032D7A150767255409FD5ADAB6E3057C0E0BD69BC50A239C00103000000010000000D00000099EB3B8B15F055C03495E0845F0132C05521468BB47466C0153CCCEEACA54FC0C471080BD1666540D55B0D8212C552C0EE884018043E6340461C97B27DE447C0E04B58FCBBF15D40F837EC7BB4162CC0B49A32F286B86040382347429DD92AC0525E4BC1095F6440607B7CB4FA222EC0CC1C13A924475B40DC87B1AA7D6D3C406CC2C0F9FF7B4140087ADBE8125A4E40E085C2C4C6D715C03EC7BE3151585540003A982C83F0D33FF83C1D67602D3E40757C5E62230763C0C021F67B5ECBFCBF99EB3B8B15F055C03495E0845F0132C00103000000010000000C00000044DDC6D68FE260C04B44918AC61750C057A9B0984ED264C09F708A0DE2CC54C0F0E2CA5AB6473DC038BFFBDC66E955C0F08558C325B9304008FA411394E83DC08AE577F980F75140774EAC8F46C146C012E5275D458B64408849842DA0A333C078315948B5A44B4004B11C062AF531C008E4FA77D7E35340E0A77ECC6C6F37400EA3DE0BD44361400240C53ECF96504098708DCEBAF545C0A05211612B9D4440852B218682E864C02052063ED021094044DDC6D68FE260C04B44918AC61750C00103000000010000000B000000BC8E0FCEADCD44C096E819C7558152C090564D3D6D563BC0007AA1A625D150C04E876E59C5B3604070A5B9284DC813C0CCCAB7A817F165406CDC3D4CA58A39409C408E4B3DF05640D01E30AB80D94240C71BDEE99F6559C0FEE6B33A6941554098920A16E99840C040B6E1FAEDFB2C40767039DC534F4BC0A0DF89763BA43140DC67D7E89EE564C0A0EA7A7A99E4344070230889502966C02C3F4F5A5FFF3240BC8E0FCEADCD44C096E819C7558152C00103000000010000000D0000007F29D392210E64C03E0A74833A1452C0EC8B39AA9FBB4CC0CD14BDF08B6B56C060470E50352521402D0F6EEB33C54CC020F1CE2C480E21402D0E8C5F5E7350C0E684EB10AB486540073345A0C47C55C044183439286C5140C075EA468F62F03F9CA8923AFFE7614050B0CED98189504038A9671868BB4E4074C5A9351B35474028B9FADF88BC3640ACCDE2C21F6F4B4060EA6B616B0819C0A03D77B482CF504011AC129C3A0950C09849FAFEA27B4F4089D3865F583B55C03091EDFCF3EC4D407F29D392210E64C03E0A74833A1452C00101000020E610000000B3843078E85B4028D8AC65E4AD3EC00106000020E61000000100000001030000000100000007000000A65BE14C1A5159C0329D05EC4C824FC082CD690CB3F851407AD71E8FA3EF41C04C5BF438F6285840008B46A6418B0A40A829122033814C4050B1EE374ADC1B402820B1B728A05740784A8EC68BC03B4005EC7EF667F256C000DE04CAEDB327C0A65BE14C1A5159C0329D05EC4C824FC00106000020E61000000500000001030000000100000004000000802728E744233D403DC2E7F9769C4FC0DB4ADF655B3E64C024B322137FD63740D0B3FFD2E7A255C0C0F0AF4A507227C0802728E744233D403DC2E7F9769C4FC00103000000010000000900000071A5E4CC781364C0AB54BC13D89447C040634E15ABC6074070D5DA27EABA54C0181AB95981654840A16552A351A94AC06C9BD69FA0575240605E6130E41A46C05C04C2E5271A64404A0116C8411448C08538F39A97E557C0F65E46C4B60441405F59DCBA840C56C0B00220A7AFD32640C325B27BF49B59C0A679E22280803BC071A5E4CC781364C0AB54BC13D89447C001030000000100000007000000A92A5B49237F65C00C04B0D652CF4EC0EE3190889D7052400463133336944BC09A7816986D88654030CED17C74C7144038C9E29D95DD5A40D08BEC7B004D1440A614869046B06440FC65C23152BD434090AE9F2B3D14254080BD04D4C570E7BFA92A5B49237F65C00C04B0D652CF4EC00103000000010000000800000073B436F62A5252C0B179C90A601A45C05817646D521E42C0B565381722194AC0A80ECEB7AD1B434014535ECCE1234BC06402EBF0392C4F4076B9F3203F0B52C0AAEFDA80724C5240A4DA915DB96030C0EA9BECCE830D62407846E687AC9835407CAF05B3F75F4EC0EA39544C3817534073B436F62A5252C0B179C90A601A45C0010300000001000000070000002232E5A8390264C0B233ED57232C40C080D559FEA0E943C03FAD2261ED0240C0D0FD3E159A5B2BC0DC250CC58A3F47C0D80DC08FCE906340B0BDA51BFFD13CC0F8F24816D37F3D407095B5509F2620C080C6EEFFE79755C07C53DD2141DD49402232E5A8390264C0B233ED57232C40C00102000020E6100000030000003CD35B154BB44440FE6FAE0063794CC080EED8FB927E1140AAEA8E13D9654140E87CE189A7E438C0F092A0E34B1C22C00106000020E6100000080000000103000000010000000C0000003CE2868EFBE45FC0006F1F1AB1BB0AC0A916D21CF97E5EC03059B9A311B325C0A8B8A73BA6A43DC0BB2653CD3BF552C0C0598995025010402311745A1F5241C01C1BFD0FB04A5D40C0929A0FEDFD03C0ACDD7005E56D4B40B0380CE6078B10401C252A09FF854E40182A0FB1C36823408689B137278C6040B43B5886F0A75340D4EFE76A24354CC08CC51DD576705340196AFB81B24857C08CE004BF4F963E409E0A089A6E7D4EC0705054E3027524403CE2868EFBE45FC0006F1F1AB1BB0AC001030000000100000009000000008E9E8DA55DF33FC674CD903FF83EC060ACB481A36538C0D32C3764ABC455C0C4775AF7C3915E407E9578A06B6D54C0D8B1FCE2F8D25740F0939A957B5814C0E257B353B38E6240E038E745ECA84B40D8666A3E64EF5540B84F56B7C6E64E4068D1F028A27F36409EFC07F114705240B858DD3745C43BC0281778EC95CA2440008E9E8DA55DF33FC674CD903FF83EC00103000000010000000C000000B9DEEE7323D060C004F3D3499FD455C0ACD2359B53EA50407E3B90B5B44555C02466BFD999995340EBF347E7C84454C05CF0D6F0A2FE5740DB35EF5C80B64CC0BABE004CE38752406035A2556C551C401429E50822735340808B4CAEC5A02A40F84273BEA52357400C3B9A6650DD4540946E29CF28E64A40124D7F42A52355407834260B228334406C5E42F853FF4740FC6FF241203E60C01ADE9997E43456402C132EEE16A852C0EC7A910214024840B9DEEE7323D060C004F3D3499FD455C001030000000100000009000000A0966E59D61B3440C9F5D9DDA4C353C0B0FF74FF85C025406E26BB6BEE3B42C03C4420E3A42F5740C0CE71934CFF3EC0186BE0D435A35640E4BFE9C139E339C01C4ABFB91F765E408CA24E958EF24440181A4B4B2FDF48C0C425C164BBBB4640301C58EEBF495BC03C8A984462F34840D11EA88A0C7B65C07C0A1C5B60144E40A0966E59D61B3440C9F5D9DDA4C353C00103000000010000000900000039FA8DFB8AF361C008C3D199B81638C0627252158EB14AC0B41E4F30B56C52C0C0E7BBE111D804C076D0EF38594253C0D29C2212935466407E0E2105238C4FC09225F201200C624024784CC1812F3C401053CA53321F5E40CE906A9B552241406C6332DABE17544090E42A0DB239374098DFB77AA6393CC04C1DCFED2C32454039FA8DFB8AF361C008C3D199B81638C00103000000010000000A0000005E0C9F1F40454CC0B0E1F3BA7CA329C008D2C19D3313544098237FC2815A51C088D8588B08A65940D9895F8BFA7853C0AC6BC1FE376556407C5824F7D57642C002D154E56DF86240488C6EDB334420C050019214E01D6540DCA616595EC45340A30231033FDF50C0E063FCC3A1D25040608320714BBF12C0382B82ACF528364026E541DE7A7A61C0DE2938407DAD50405E0C9F1F40454CC0B0E1F3BA7CA329C00103000000010000000B00000000E6183041CC1E400003049E014739C0986E11E94F6E4A40B5209543554D45C068E49B14EAD15A4052D47F4DD14D52C016CA9747B3A860406FCB7E811CE953C018DB6F00AD4A4D405C2D8E4A162231C0A490307953574E401803EE86969629C08CC73336E2AC5340A8882C83FEF54740F0D1FB3CAD9451403C854B71B0BC46401A078D0CE14D4FC090E2016D12CC21400080DA6EA02407C058B6C5721AB727C000E6183041CC1E400003049E014739C00103000000010000000A000000583C65BA52613CC0002C3023440E28C084162CA4733744C0617C2B1ABE914AC0E893DC14A7173AC0988F3DA975724FC0604658861A90544023A1E31CC3484FC03E447035EE7C604034AE3CCE228441C0F0F89427BB6D654064958EA0741A50405C29B017B9DE60405476D693DE634C40708F1B29972C54406E7652BEE7575240DCF6192FF02654403E56143B78E25340583C65BA52613CC0002C3023440E28C0':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (43): SELECT st_asewkt('``K~FR|T':::STRING, 3751314262887522984:::INT8) 
    rsg_test.go:125: Last executed (44): SELECT st_asgeojson((5e-324:::FLOAT8,), e'\x11.\x1bx\x1eI\x14``;':::STRING, 6078885587720026799:::INT8) 
    rsg_test.go:125: Last executed (45): SELECT st_asewkt('0102000020E6100000020000007B07D727DD0666C0D9AAF453374753C020A2EFA16A503340F968F716986348C0':::GEOGRAPHY, (-2745090967731568151):::INT8) 
    rsg_test.go:125: Last executed (46): SELECT st_asgeojson((SELECT NULL)::tuple) 
    rsg_test.go:125: Last executed (47): SELECT st_asgeojson((4173730971:::REGPROCEDURE,), '':::STRING) 
    rsg_test.go:125: Last executed (48): SELECT st_asewkt((SELECT NULL)::string) 
    rsg_test.go:125: Last executed (49): SELECT st_asgeojson(NULL, 'dU;kn2':::STRING, 4637139874805421579:::INT8, false) 
    rsg_test.go:128: Currently active statements: map[]
    rsg_test.go:515: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions1357095741
--- FAIL: TestRandomSyntaxFunctions (301.43s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestRandomSyntaxFunctions.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20684837?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20684837?buildTab=artifacts#/) on master @ [cc61b3c16062dfeef5ee82e6a80d868d499e047e](https://github.com/cockroachdb/cockroach/commits/cc61b3c16062dfeef5ee82e6a80d868d499e047e):


```
    rsg_test.go:125: Last executed (4): SELECT st_multipointfromtext('P?jxEm':::STRING, (-2420441671716909683):::INT8) 
    rsg_test.go:125: Last executed (5): SELECT st_multipointfromwkb('\x':::BYTES, 2779714051534119305:::INT8) 
    rsg_test.go:125: Last executed (6): SELECT st_multipointfromwkb('\x22':::BYTES) 
    rsg_test.go:125: Last executed (7): SELECT st_multipolyfromtext('+':::STRING) 
    rsg_test.go:125: Last executed (8): SELECT st_multipolyfromtext(e'E\x1dd0#\x1b':::STRING, 8399550561006828381:::INT8) 
    rsg_test.go:125: Last executed (9): SELECT st_multipolyfromwkb('\x6593d1282768f38f':::BYTES) 
    rsg_test.go:125: Last executed (10): SELECT st_multipolyfromwkb('\x206a2e7ad5':::BYTES, (-3735479932318851301):::INT8) 
    rsg_test.go:125: Last executed (11): SELECT st_multipolygonfromtext('b':::STRING) 
    rsg_test.go:125: Last executed (12): SELECT st_multipolygonfromtext('':::STRING, 6175235887885104043:::INT8) 
    rsg_test.go:125: Last executed (13): SELECT st_ndims(NULL) 
    rsg_test.go:125: Last executed (14): SELECT st_multipolygonfromwkb('\xf31da704e995a8648f':::BYTES) 
    rsg_test.go:125: Last executed (15): SELECT st_multipolygonfromwkb('\xad7ac08a':::BYTES, (-1460306126506653575):::INT8) 
    rsg_test.go:125: Last executed (16): SELECT st_node('0103000040010000000500000072F5AC2CADF2FAC1C8D599AAC4E9E0418231E2E0EC05E6C193FBD451FC8DFAC1169EF6B9DEC9F1C1E1419946771E01C24488FDF37AB4E641E8CDB9EDC53800423CEC6D8F4BFCF041A4607D41D081F1C1823DFAEF696600422ED9F0747549F44172F5AC2CADF2FAC1C8D599AAC4E9E0418231E2E0EC05E6C1':::GEOMETRY) 
    rsg_test.go:125: Last executed (17): SELECT st_normalize('010200008000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (18): SELECT st_npoints('01040000C000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (19): SELECT st_nrings('01010000C020C123E91277FE41CE29D3C5D7510042766EA6B55EC5FCC177764D9E694D01C2':::GEOMETRY) 
    rsg_test.go:125: Last executed (20): SELECT st_numgeometries('01060000C000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (21): SELECT st_numinteriorrings('010200008000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (22): SELECT st_numpoints(NULL) 
    rsg_test.go:125: Last executed (23): SELECT st_numinteriorring('0106000040040000000103000040010000000A0000005848614A55DDD7C10038717A0E48B7C1E076F8B19D07E8411CBAF1CB85A3D6C13C460FBE0AF3D6C170775850F0A3F741E1B226F8AA3EF1C14553F97882B8FEC1306EFC6E1327E2C17EAE8C7FE2D4F5410B348EA4405500C2585C8DAC4AA9F9416A0947C66619F441C8E0A213ECD2D64166D7AF046690F7C1C6DF8785E496F341E83189244059F841E004F729C315EC41782EAF16FE89E9415C2970EF37BAF44169EA0A15B63EFFC1B06D9BC5FFF9C341F84BA5612C24DD417812C880CB77D0C1CC14EC648CC4DBC13CA49D15709DE041080CD8E6B048FFC15848614A55DDD7C10038717A0E48B7C1E076F8B19D07E8410103000040010000000D000000B4B75BCC1E70F5C1D87EFAC7C8AFD1C150391D0BD38AD641D879CCBF733FEBC13030B6327F8CD0C182F1E182414CFBC116320A35FEBDFCC1F39EA9E5EF9500C29AD1C1FA2B6EF641A0124E45953ABAC1C55623C9DD2DF1C11C505941F9E5E141B05427444F78EA4134007723E80EFCC1749C1272B273F3C1EC61D96DB949FD4100A8967F3A57FFC14CE74A7EC742EB4124558F97FCDCFC41AAD4AE08A0CCE3C147F5598F8086F0C1566B41711BE90042A49481BF95E6E541B00B96E83BCDCE41441CD715B7B90142DE67CF30651FF741D0BEF99BAE77FF4148ED70AD1E77E2415CF097F41AB3ED4166FD51C0417A014262EBB855CBA8E4C1F64E98B458200142D850D35B636ED4C1E36251ABCF2FFFC10C324DFBA7ADF341B6F8B7B080550142B4B75BCC1E70F5C1D87EFAC7C8AFD1C150391D0BD38AD641010300004001000000090000006E33150B58F1F8C1406F2FC81C7BB1C1B0F35D11A548CCC190B7239E4260FCC1C625D4F8D20CF8C13029E70787A6FDC1AC37EE9BA596E34165BC951B576AFEC1FDFB17C85537FEC1B8DF01B9EE2A00426EF8F2DA0C65F44168CD6642A392D641DA8FBCD9FFD3F941D073A70139E5EF418CFA8849E7FFF0418CD387BA9B86FF41E0A858845124FB41B4A6A38EAA23F44142C7D014B4BCF0412E080083480FF1413C8E264BA874DDC134E67104ED14E64158667CF957D0DE41D4BE42316656FA416E33150B58F1F8C1406F2FC81C7BB1C1B0F35D11A548CCC10103000040010000000600000016C6728B8B13E0C161DB17DA8F4602C28CEE626E4205014210B2009D73CFC8C10FAA15C959C801C2A83DAF3F57E3FC4128FE53EDC9F0FB41653C8D78B21802C27621F0B28FAAEEC10E51421DE562F941B61EA9A7E66A00C2882C2C6EDFA3DA41149AEBD5B499F3C1D22ECABD8BA50042203C94496327C24116C6728B8B13E0C161DB17DA8F4602C28CEE626E42050142':::GEOMETRY) 
    rsg_test.go:125: Last executed (24): SELECT st_orderingequals('0107000040010000000103000040010000000D000000F07EB2A3CE5CD6419D26EF8B11E1F9C1C42AB7E5A03B02C240FDEB427371EF4172F620FF12FCFCC1C0CC0A474494CA41EED35B2E196CF14102BBC6F7D083FDC1F4BA4D2167E5F5414E457DCE93DAF0418CD6FF2FEC3DE8C16BD7C08BE959F0C1F2ED55FED4190042616363F1115DFBC1CB09B32E789BFEC1CEE6AE7DADD20042C62ED43B3340E9C1408C2F32F4AAFB41D01A6AF71D140042745661FA04650042D6EFE2BFD3A7FBC180CB8D2EFCD2C9C13CD703D42C64FB418E9CAFDFB7FAF6419690D38ACE36E8C1E820CC77DDCFFD417004B37E154CFFC1CC1584792945F4C1301CFA5187EBF7417EB301B8240BE7C192F4BAE53089F5C11ECB5D275995F141C0C993626D0F02425ACF281EF8A0EEC170291AD934F3E941A3219B8ED79402C2F07EB2A3CE5CD6419D26EF8B11E1F9C1C42AB7E5A03B02C2':::GEOMETRY, NULL) 
    rsg_test.go:125: Last executed (25): SELECT st_orientedenvelope('01010000C0000000000000F87F000000000000F87F000000000000F87F000000000000F87F':::GEOMETRY) 
    rsg_test.go:125: Last executed (26): SELECT st_overlaps('01050000C00800000001020000C00600000010B078D3E2BEECC19EA4C361A0CDE7C1D6E8A4A693B6E7C1D026CD218AB5DBC1B19876954125F1C108FADBD23727F2C1228D3CB99F55EEC10036FA1BBE5EB1C1704F77FDB608F9C1B4CA10277118FCC1C884FB5467C8E3C1B49A929B54FAE641608D0ADF3FB6FC41546ED7B91630FFC1DCAAB321A1EFF9C1B804DE44E871E5C114995930D347F04130762CCDCD01E5419A9AC331F9E2F641C4DF20E19DC0FD41CB8DC4EF9C8600C28456A7DD9AAB004218ED1E67932501C280A5603EA740E54101020000C00500000034F6D8BEAFB500C25CD061C0F744DEC17432D45AA1A1FD41FEF4D7BC6BB7F44193DE89FA888701C230212BE2D08FE6C1B8B309E69969F7C1467ACB3EFDA4FEC198032B847E7D0242467B4B011766F7C1E21080FAF440FCC1F0B091DF4031CBC15C877889D58CFC414822310BE939DAC152D5AC4AB61FF241A8FB9D486B340042443CF8C7CD56F541604F12FF9677D2C14C7CCB7C5EF300C2454DE112710E00C201020000C004000000D8713DF987A1E641124D3F6CEBADE6C1B051C37DD61FE5C1CCBC72736F89E44164B0C43984C6F241DACBCAF37252E0C1403B3DF3ECA4B8C15A736DC5E700FAC1AE792AF8C67B00428A358157D78B004262BA1DB5F9EAF441BCD8CF4E2FFFF841A8EFBD87CE40FA41F8962371681AFA41D22BC67080EF00C2C8F0487354BFE4C101020000C002000000CCDEB5DE6DD6E24162A69DFB96380042F4889BC78E6AFC411A3181E4EF79F041F060D4C0ADC5C64172D7516C544A01429EDF7A6E53C701427CAB4A5BC549014201020000C002000000E0DDEBA9F9DEFE41504CF55E2F16D6C1C71FB83E1284F4C194D2ED34B153D5C17CBDAF47094B0042CCBB7EE30D2AE34168355F4865DAE8C1BCCA6B90BACCFF4101020000C00400000080E04305538BD8C1B7EBDEF218B8F1C13C61E25C7BDCF9C134BBA62BC00EDFC1F48715C8578ADBC16C390127F26EF5C1FDA533262DE8FFC1CC69C155D2F9EC41324C13241792FA41E0F73A780AD9FDC17C01D932D5DEFEC1FC395FD85ED2FD41A26CB468734B0042FEFA6C284FF6F6C1EC042120B55FFB41CE3139725F7CFBC101020000C00400000070C1A94DA1C2FA41785980D85ED0ECC1EC7775DE66FDFB41FD17275F14A2FCC10E1930E4859A014290541F8DC7A5C941DAAE89D1FBA7FA41D0028D6EEA2AC24180161AEBA5D3AB4148174A2E03B1FE4130C927220EC8E741D8CFD9DACA50FD419AF3F36AD2F2FCC15A152D2BA36A0142F49F1D315F9DE141075FA4004C0CF9C101020000C005000000AC63B9860ADAEA418013A14EB3C1B5C146B7D28F767602C220641DC7AE0EFC4184C3BBB25722FE41BAED3DAE288AE3C1502DCE1E3945004223E8614F23ADF9C1D091E85CEED5C9410EA46CD2F8BAF941C0367E5FA69DB8417BDAD6DFF6A0FEC1C055DB89FA18C041D2A80703E82F014264FFC38BAA3AFD41D636231450EA00C2E0D88D6DB2F2C041EC813E86670DF5416BEC9A0D90C801C2BA9EA6D39C6100C2':::GEOMETRY, '010500004006000000010200004003000000780406183714FC4114257D63CA50F541CC6D9F08C4EAFB41C86CF683B2440142205BDB55E0B9FF41DF12C67DDA8000C21EF66441E98FE6C192241F9F352C02423C2124CB7BFDEEC101020000400400000040D9CECA3D80AAC1D8828B4B0175F14174530476B7A4FC41AD9B452EFE55F9C1A8AF5F35F0FEF441B67A0BDDD164F2419AF6E30FC40AEDC1284FA0A5A794DD414C25FC722375FE41C63F426FD8CFF1C100115A252A2590C1482899C72D96EC4101020000400200000040F62BA18A8CF9C1C700113594C9F9C17C55BBA50174E44180B04860E46AC9417B4E83E212E8F3C14AF1604D7A50FA41010200004002000000800831F9551FA7C1E0649974612CC7C1F44BDF17393A00C2A4D1DA0EF332E44106370461C2ECF2C11838557BD28EFF41010200004007000000AAE9A5E340BBF7C100A3306BB5D7B2C18430D4D83343F6C1C87E21AC745CF6C1C2A79969A34AF6C1342C5F15985EF94116447F2012230042B219C0B108ECF2C1D2EC71CD994EE9C1F01170C51A52FB4110E9004643AAD941947EA05E582DFC410875BE7343EFD141D0F365CE79C6D6414402404EF3D3EE41EF3589B34492F8C1A4CBBF227532FE415898032D47C3F3C17D670EC9E1F3FFC12C9661297883F941524B47262AFCEAC101020000400400000050F35EA2BB2ED0C14264B926CA95F2C1313137BDD6D3FDC1468D465AEE6AF241BAFBDF252358F54136EFB65B366BE9C1647A2C7F7792E74134EF98327B01F641525BCEA89CFEF8C1D87C39360945D3C168B1EB4363460142BE4F940F0FBCF2C1':::GEOMETRY) 
    rsg_test.go:125: Last executed (27): SELECT st_perimeter('01050000E0E61000000200000001020000C00300000030AAE261CFD34F40885F125BB3A44640788BCB20109AFDC1D8BA4CFFF55CD4417C7D8B7BB96C4340D8109ECC59604840E873F73A45E4E94110A5D021DEE80042EDE9148A6DA752C07EDEC7FDDBA85340107C70949944D3C17ED8F55895F3E2C101020000C0050000005D185F4286DB51C07271A733C7933CC06076ABAC4E3802C2803A754B21D9CDC140C82FE5316A1340C1A5B85F3C1C47C09C81C9FA332DE1C15A8277DCCD43FBC1607344618B734D4060BBBD8ACBE62BC0A205E14B89E2F7C1948B2705F0BEE0C1B04B77FABDC16540100B92FA9BC35140B6D2233C058CE3C13F2CDD5871AC01C216DE5A87D7C651408048D6AF38BB46403C367B449E5DFC415459BE281D11DDC1':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (28): SELECT st_perimeter('01070000E0E61000000700000001050000E0E61000000900000001020000C008000000E80C3CFE603A43C0D6990F64507445C005DB59D2EF0E00C264423D311642E5C110133B61003239407AAA4AFAED2343C064753FCCBB9FE04100BD3B192C16B3C1C03A0E359CBC5E4089C3D047489B44C09A454E90861F01C2C0197D3EE062B1C1FC4FA5869C286540E80EFA733E3C4B40F0479B134EEA01C2E88AB29BC9EDEEC1008B9C5CCE4F4240606B705E223A1140A47D52B98EE3E041ACFE9161B39EEC41E42576BF384B5340063D0A04CC814140C84E7BDC954AFE41702B123EACB701C20C15A8D4225D4640D0AD7EE2A99A2C40CC08E36A84C6FF411A4B9B6EBA18F7C1C0D438EB8B4C40C05017069A49243440380DA31EFA22F3413ECB6E80219E014201020000C00600000017317C5316FC65C083971B43E1EA48C0DC23950B1B71E341061BEC13ECD0FBC1400A4227578B5BC02F05CF3A620756C008290CC678FDDFC19CF4B9C788D3F54198D3BAB78CFE3A404E3ECA9F7FAE52C0789723DBF597ED4130989637A1ABC2C16C3607456D7664400020403E0D5DC03F04820174F78F02C2F04F4180036BFB41DC32535E91D761400CF7FE65972138402068991A05B2B8C1F4243C150F0A0042849D91F9027F5D40C0FFDE4F18824D4074D4BE02880AEA41B0AD5EEE8E52C44101020000C005000000AC6881D96FBF5C4000EBCACF8747F1BF0C900ED343A6E3415076ECE4E703C741ECFD2F2B183C6140A0DDA5BA3630154052A22F6FFE7DE7C1C09EC861A21DF4417C17A8B9DFEA4740F89EEEB8B2165640685601E056F5D14138D77BB92818FCC10C8C9604C3B760C02C0E068DA61F4940C56EC072DB96F5C1D6E4D8BD3E9800421518D2ECDE2B65C06C4E802CF8B24A40B2494BC5AFDDF641B9922183B0E1F2C101020000C0030000000589D7C2DF0453C0F248463D644051C00043342A211AF3C1A67BFAABE5AEEEC17C6980E3D1BE5A409F2388B09DE244C0E88B828BD8D6DF413814A8A3EADAD0C1C49F6640819659C0705580B7F6AB5440DEEB2DD8B58000C2CC4601A0A42AF84101020000C0020000003C0D98F3C6514240D8BA3316EA384B40C0D1FA62E5F8A7C1C6A8A3DCE72FF64174E6B53D9FB15DC0687B8DFC5B865240AE24DD00F797FAC102B0E1B0C01BFCC101020000C0020000000E88F1EEEE546440F8638A2A537135C0609E9263378EC44176F0A4225149F4419E8257AA322A66403A710AEBFE8D5540662A6BF71DF9FAC139D2AFC94AD7F3C101020000C0020000001877D7A17F6B66405B9D084AAA864BC0B4D27233DEA2DFC138063E4D7BF0E14178316D70C1455640000AC61DF4D9CC3F57F661692D6F00C236BBE422C689024201020000C004000000806C330E7ED02F40E09C7444950E0EC04E74F5A3B701004240E7CA9AD06DAAC15059A51C6A824D40B07440665B1810C0E91E7DE17B2902C25056290BF165EEC18A61E3DDE6D861408822143E0CC35440BE62B53CF844FAC15C33D3030AB5FE41EC6B90C8451943C0B6437A8574685140F9AE4E1B7981F6C1C0ED4B4A37A8A64101020000C00300000062F16A02E4F66340F75F934B9E9352C0B4E69119AE6BF541A701DC824C76F9C1D43DDE2E91D96240BA2B64828D2E50C0E05F738A7723D0C133D8211A970200C2D8D75BB5AF845C40A8107BC617442240B82DE4EBA82702C2811CBAF75BEDF7C101010000E0E61000009CDB74C335555A4020F6B7308AEF0A4032A30572AAD5F5C1F456ACC69948F6C101010000E0E6100000CC43DE145779574020106A7AA0540840ECA71A42A469E2C132CC5FEB31ADF04101030000E0E61000000100000007000000600ABCBFA72026401E284705B4C547C0EC13A0077B0FF0C12CFA4D5D2439EFC12438655D860C5D4058FE8A702D9A51C0AC6A50EB1E39F44180B3AE10272D00C23E12C4FE0C4962403D62D2EF7A2754C000B2976C4A6B85C1B62B591D6D16E5C154544E1B58DA5140C0E6741B57A823C09CAA1823D37CED41584D61775F32EAC1E06648C62E2F274070725F39953B50407CE7ACEB2B09D9C120B662FB3BEAC3C188243D1F4E6358C0F8D22D4E1FF72040E87E7916773CEC418CDEA07E7C8F0242600ABCBFA72026401E284705B4C547C0EC13A0077B0FF0C12CFA4D5D2439EFC101030000E0E6100000010000000600000014735756EA1441C040F7F59D29361EC060552CDF8232D84118C480DB4F5DFF4157FFF301B6F759C0843FDEE9123552C0F714A197898100C2488438F1358402423800D549D8E161405F547DCD4C2150C0B2D85C30E63BF441780EDC1F57D4FC413886503ED8A53EC03C380A569C2E55401CA9874AD1ADFF417890E6D2915BF14178F25567115140C0AEC40791CB1E53401CF5166A9BF501429EB75449E41FF7C114735756EA1441C040F7F59D29361EC060552CDF8232D84118C480DB4F5DFF4101010000E0E6100000D9243EDAA29F65C01221B62ECFBC52C0F82189DF5C92FB4174F8CFC920F501C201060000E0E61000000900000001030000C0010000000D000000E0EE775C55F74EC07CDAE05AFF273FC068127A952386DEC1C4752CEC7F04FE41604DE0DFD70923C0360EFAB2857756C0AD342052527FF1C1206D6830B861FCC1602E41CC38CE604007D3272EB9E044C05C20CB0A06AEF741445DC9723289F5C100E60A6F0F096140A825B1FC19FF3FC0F16518F3AC0CF2C1400E21E059BCEAC19438A7B2073C6540DC92D4F3D7F449400C574F2E802BE241D0E468BFEED1C5C1445D62440CE65C405007C636A3444740EA1A83553B50F9C19C6A3475A8C3E6410CD5A40604E75F407A6390A1DE1651406690B2BB01F1FAC1785BADC27814EDC15473723709D55440E486FDCFF8184640001023009FDBC1C19C471CCC63F3FBC1E8DB1BB334B6334080247BC5D46C544010FB30574BF6FDC1CC80F6EFD9B8E74190BB7161A4B73E409C798C0445583740E8BADFE48EA6E5C12074DDB85CA6F041C088D911B59F50C04035EB2EFFC44140B88ECFDFD146014232C660AB00270042D0F42CEE46FC3EC0C04130FC5B7E3240B059D874287ACDC1B6ED043CFFDDEDC1E0EE775C55F74EC07CDAE05AFF273FC068127A952386DEC1C4752CEC7F04FE4101030000C00100000007000000A0901972FF272CC02B7FAF3C96BC53C010FF4AFFFCBCD8418EBBBFAEAAFD004284C8A78D65A34240585D345E15AA4DC0F2F7A465CA87FEC1D8029D5D2FE2D1C1A42EED58C2E55740833A81276B9849C030AB4E7D6ECECBC1D02DE071C716FF410684F6DE71376240D9F9C39F01744BC05BFA2CCEED76FDC17C7B7D487A8BDEC180262F96CCE55040142C6550FCFB3540B80647160F29E741EEDF65BE22A2FBC16EE94504E0475DC0ECD39AA03AB34940CC4BCD8779E9E3419A91DF13B60801C2A0901972FF272CC02B7FAF3C96BC53C010FF4AFFFCBCD8418EBBBFAEAAFD004201030000C0010000000D000000521F32CA085D5FC06C30978B4ABB51C000CCBD4B42930042B429BE310BF7FBC1348167893F4152C0AA416E46F7BA4EC040B591D728B4CEC18B79B2FFC528F0C178FE275ED48140C062D6E60B845C56C060B11DC2D68FE2419460C7F9C759D7C1E8D4FB8E057B54401A9EA503375052C000AE8FEA737B71413CACEF43D2CEF34144992D108F345F40E8D16991EF8B39C0307CC0A19465D7C18C9445395A0FECC1D413676FCDC85B4000DE0D0985A123C05093AE33B92EFA41F4733A730D23F2C1ECE527C5AC804540E0FD2F3CB04E1CC0EE1C76F2EC9B02C20671291FB519E4C15AA76529F3246040C866C960BCC44D4086CD69B55AFFE4C178F012416EFFD94138B9E9F079083640F4713D6425B353406CEE3C6DB616FA415BF41DE1114EFAC1D04754E5214F29C000E320D70EA93A40681F91A1706DF2C14C9A84971849E641F2675BE5B9535EC0A67814CBF39C5540666A0716C3B0E5C1EEE2EF33272AF5C14264B86681A861C0F07EB6C15E202E40D453F25308C9E34104CBCB6DEEC0FB41521F32CA085D5FC06C30978B4ABB51C000CCBD4B42930042B429BE310BF7FBC101030000C0010000000B0000002A9A442CE0A14FC0D0B77FDE178D45C014DD6991083CF141EEE78685FFC90042D02CF6D324CC3EC076041C540C8B42C0DBD947D9F8C6F4C14AF2063028FAF5C1F0A9A857397D33C0C4F03225115351C030ED6229A16CDA417021DEC8765701C21C2EC2C7DD625E4075718267FDAC54C080FB9529E42C97C134E96BC9CF77014220983C32DB704640E894F32A136532C0546F7121B956EF41EC33808238D401C2C0EC1833C5B84C40206A7002299827C0638CC9F6BE45F8C1302C3C14F38ADF418CA9E91EAACF63403E1A7151BE3B5640DE057A026E72FCC1C494857080ACDCC10023EC2704FA11C084770AD3674E4F4070D3803FAFB3C6C17448F9D5B588EBC188A435A279BB4DC064EBF84A573935409587C7EAFE9EF9C1D1443402DF0A00C2F8356DF7D40552C0E0BC9F50408C08C0E85DCF7FFEB7DE414053632F1F23B4C12A9A442CE0A14FC0D0B77FDE178D45C014DD6991083CF141EEE78685FFC9004201030000C0010000000D00000037A5E10B356960C0E881993293BE20C0EE56BF25AD8602C21CDB965296FDE541EE30E5D5B2B465C0A6C446D6BDA151C09CD82ECBDEE9F841C089C7FD4B0CD64115505F84773E50C0F802060F2CCE32C0A2BC3C91380502C2B86E59250C47F341803B487A0EC111C0FBBB7489605E45C06412F769B872F641A28CFF546AE2FCC174C6CB00B5804140A0BF5F55F3D134C051B346B52D3501C200378E5BD197B5C1947191D4A1345C40FCB81BE4EC874940E0737D6B9DEEF6C110D6A6B14356F1C1505BA2078ABC394074371CB65E684A40A08A50486874E3C1600E06253E36F141643776E908315440B49553E7384B554000FC94169ACA00426A0AD9F67EFBF8C100619CB6E37826C0B6E0E87A84A955408201ADFF76AEE5C1BA5ED5D65B55E5C10CC279DDA28F48C042CFEB624A564240202B687056FA004268173EB9B3F1F8C1105ABF6CACBA60C0E4C928B6E35A4540DA5BC3B30E51FCC158178310F2D0F3C122BBB8814F2F62C00496E118C510434070F9F5C5B6AFDC4118104DC6F018EE4137A5E10B356960C0E881993293BE20C0EE56BF25AD8602C21CDB965296FDE54101030000C00100000008000000B8FBBC90C9FE43C00070A1BF5F2CB43F1E4877D72B46F2413EDC93D4DB02F241C80640D48401544050B04B6878432CC0EAEEB842716BECC10C11B4F40B63E341F0A2D5CB661B6240839304F7657941C060A041221E92D1415062DDEB3D05C7C160B3A65EA4685340E00A659F6D22434010CE66E8FDEBDFC10C485DBD4E0FE6C11CBBD7A6EAE55040F0EC9BBB2F1D48403726B1B74584FAC1A87074042385E141F82AE9C62C8549402A02E77B1AA651400CAFD10F39D8E6C140292C9DB68DF041F0B997308C482AC09499CCDBE0194140404F29F459EEB0C1DA428BF064D40142B8FBBC90C9FE43C00070A1BF5F2CB43F1E4877D72B46F2413EDC93D4DB02F24101030000C0010000000D00000046D20B13062063C00BF9E3139AE451C0B42BBE9342C1F141AC458C332137E041203813C0F26848C08C5884ABE50854C00C72AEBEB602F4414E687F4B1ED2E4C1E0236665DCE12E406C681DE997D143C0DC1DE416C571E0C108756C0A8188D7410E333B1B3AA76540E328585CCE6440C0405108AAC5E5BE41A80276872898F9C134B3850790D3654060B25D2230FD1040F857DC5752EFF441D3B911833BCEF2C1B2AB692E74EA60403CA9C338436C32402C97D017B4ECFE414CB298C83AE601421CE19DE431C35C40A4F8C5093DF6494090B332B4DCDDDBC1243DB7D73002E6C18059265FA6A84640788D6765843D35400022DD7FC6D588C11808E7358DC4F141C07F096CEE932740CC40B8832D7B4140F071BA991E8AE141B6F4BA5EBEF8EEC16262D625393A53C03E7020583F515440D825B11587DFD54114D3E267D375F8C14FBD978C012460C0AC4CBC10C45152402AD2AA2F4AA0F0C1C00136FFFB8CF141F4D57CB6245A41C068812FF8BC542440662017CA90FBF7418080899D93B99D4146D20B13062063C00BF9E3139AE451C0B42BBE9342C1F141AC458C332137E04101030000C00100000004000000142EB8091FA143C0ACCAF13DDE12424092B83465D8E8F14126C1F6DE1318F341B4434986C42B44C006C528814CE95140EE4F18F01010FDC1408BB1CB24C1D3C106263E24360160C0782CD5FFF3774E40C89FC1E020C5FB41488C3970B1B2E4C1142EB8091FA143C0ACCAF13DDE12424092B83465D8E8F14126C1F6DE1318F34101030000C001000000080000004CB46BAA703961C0E0D89F7EA69A20C05C299FBAB0CFFC4130F385F80EC1D3C167E16F7354B862C00E75F8B80C8152C0E6437B1DB94EF1C12A1003CE0EB3EFC1200E7E9891274940746E1712427B4BC0CFF51F5A6A6C01C2FCC38E042903FD41C48042B3BC6B5A40A0D2CF4DE5AC03C0881214246753D8C1738558FE325EF1C1E49E0E889AB25340800F1DD7B8DBE5BFE138AB5EEE95FAC12E6046983298F241D0EB6082FAB42AC058103984EB054340149CF6042567F9416449D67C1DD9EC413B90EBF3F4F95CC0C0D69F726B654D40B5C67DB8DB9CF2C108D892351319EE414CB46BAA703961C0E0D89F7EA69A20C05C299FBAB0CFFC4130F385F80EC1D3C1':::GEOGRAPHY, false) 
    rsg_test.go:125: Last executed (29): SELECT st_perimeter2d('01070000C000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (30): SELECT st_perimeter('01070000000900000001010000007603C4F7290DE2C1E8407E5205D7FF41010200000002000000BC495B3EED99F4C1B1828E89479F02C2C0D6BB27D158D5C19E8FCD640EE9014201040000000100000001010000008A0795629C8302428822CB6EE3DBE441010500000002000000010200000002000000787390A276F1D641F484028A9545F3C1A0665F93FC4FDE4180ABAC298D4AA641010200000004000000F38CD458971EF0C18D38D426DA66FBC1CE0049630D23FA411C7A90F0AE93E5C1020148E47FCCF541BE535B6BB7B1F541A303057E65D5FFC1600A52107420F741010500000003000000010200000006000000282A9533F2A0F8C1D93D8D1A5518FFC11081D481A408D1413C110728A128D7C1681D7769A761D341000A5E3190FD92414220496B46C7F44188D35F4BFC53E341AC0268DC8254FB41BEA67C7A2369F0411474522AAE61F5411CC2A6C23F2CF841010200000003000000900A786BF3C6F0C133B38372BF6000C2A8A5B51509B6E941C3FD4968F17C02C2FC55F554D26AE2419AC9DB08DE83F9C101020000000700000088B99235F48DDAC145D59EE063F001C25C327FFE81C4F74146701448E36CF6C1BE87A4D6A4DFF4417011E1830973D541E8B064A9C4D5004288808D37ABA1F341603762B2E68FCB417C6CA5229979ED41A8DFD1A4EC3ED1C16C98FA742B9BE0413F5A281AEA3CFFC1902BFCA6DAB001420104000000030000000101000000CEE34E8FAB54FA411C5D9A80A809F74101010000000599181A233AF3C180998A85CD56CD410101000000F0FD271A60F3F341C4175D1A8EE4ED410102000000040000007875536D782AF041DAC9C01A8FBF01C2B01EB2E0B51FCA41685267AAF924DCC11AA15ADAA9BDEAC126A09B52D5650242B2F3BE8A08F0FDC1BE2B049CB502F0410102000000030000003CF2773851DDEF4146E4E3A17CEFF041A89534D021E8E941708187E66BCDFB4140403A1E9F12F5C104E1A66CA5C5FD4101040000000900000001010000003828736354F7E5414FF7392F4FB5F2C10101000000ACF413119FB5FE413E7ED5383147FCC10101000000305ADD2E2991FFC1D29D7E420D4AECC101010000008C1B42BD97EEFA412EE0EA65EC71F44101010000009F92BCE6EFAB00C258A8B3E35C13E9C10101000000C4F1DC998B1EF7C1C0B73171C975F741010100000060E7A3A75208FD41A06E4FDEEE0AE441010100000090175479F28BDA4170C178C2D08EEBC10101000000B613D600D7080242748F1560B379FB41':::GEOMETRY) 
    rsg_test.go:125: Last executed (31): SELECT st_pointfromgeohash(e'6Dl\x7f':::STRING, (-7914129083167292870):::INT8) 
    rsg_test.go:125: Last executed (32): SELECT st_pointfromgeohash(e'#\'':::STRING) 
    rsg_test.go:125: Last executed (33): SELECT st_point((-0.8310817498370617):::FLOAT8, NULL::float) 
    rsg_test.go:125: Last executed (34): SELECT st_pointfromtext(e'T\x0e<\x15\x11':::STRING) 
    rsg_test.go:125: Last executed (35): SELECT st_pointfromtext(NULL, NULL) 
    rsg_test.go:125: Last executed (36): SELECT st_pointfromwkb(NULL::bytes) 
    rsg_test.go:125: Last executed (37): SELECT st_pointn('01020000000A0000002A0683942C6602C22C9FA3A7E47CF7C1F7AF4B068D80FBC1376305512786F7C1B24CD7DCEC45ECC1BEFE2DE4DE31FCC1703196BE63CBC1C18E56E4868389F6C18A0B7227BBE7F641F7A9B50CB31DFEC14069AD31242BBE415C3D1AAE5386EDC15CAA222B8691FC4198029188DAF8FE4110D637AE227FC6C1F04AE2CD4DCBE641579520CB7C26F6C148180F6E518EE841EA391E77D037FDC194A7B700EA9EED41':::GEOMETRY, 1049603355288940253:::INT8) 
    rsg_test.go:125: Last executed (38): SELECT st_pointinsidecircle('01070000400300000001060000400700000001030000400100000009000000B4CB996E019FFFC1708B92512692FBC1CA48926230D9F8C1CC01ED127149F641696BDE70367BF0C18EDB2144867CFFC1F6677703C1A2FA41E055D733C1E2BF4150CB210509E6DFC1C49E83FE3341E6410059D9FB3D87DF412A5CFD6EB22EF3411CCD9946C96CD6C1307065E816C4CC41F0B1C4BFDD9ACC41E60A746DCE70F8C12C204B5B8A08E5413F892A56FA51F9C19C0C243A1322F9C148F1BBA8127CD9413EC9C1325C4AF541CDBB76CB5DA3FEC160C4FB7F748CC841EEDD4529972F02C2B4CB996E019FFFC1708B92512692FBC1CA48926230D9F8C10103000040010000000A0000003E4F83ACB63AFAC15CA22DB44F4EFCC17A0A181FF0BF004238073EF5E19DFDC1734BEF7E7A2501C29007ACDEA314C0C1924DFF4D376DE7C175A39EC3A39602C2A63CF4382A49F641BC33EDB1C79FD8C178FAB5D87137FBC132615BCE00C9014222FC2839B98201425ABD1E5A2C18E0C1DEAF6250FDD8F141F648FF6F403F024220B38A4CAB5EE741CC20795D2A53D6C17258BFB750F0E7C182DE181FCAA20042EB72C2FEB1AFFDC10C0AD4C9BF31F9C100AA37637515AF418A92C60B3EEDEEC1F1E6F650340D02C2E88116E471E4E0C180218835CDFABDC13E4F83ACB63AFAC15CA22DB44F4EFCC17A0A181FF0BF0042010300004001000000050000000033F09097FFBBC1FE8C4D6364D4EEC1E0E14DA90489E5C1F4ABDEE27F84FD41B08F4640D7F7D3418E91ABAE2035FCC1CC9F8A2FADA5E341642C485F40F3E84176C995E60C8AF0C1D911FEFD84E7F4C156ED9C394DFFF341604DCE3A896ABAC10033F09097FFBBC1FE8C4D6364D4EEC1E0E14DA90489E5C101030000400100000004000000BB8597E7E017F1C144E517F16366DAC16CCEBAE858F1F5413CBEF1EA08DFD6C1D8C53C834411D3C1D23684F2A2870242FCBE26652244E6C1FC2C18031A9BE74104AEBEE32CE9F9C1BB8597E7E017F1C144E517F16366DAC16CCEBAE858F1F5410103000040010000000D000000A8B0B013FE7B00C26D70A6AB37C0F1C15EEDADEDBC76024270A975FEE80AF0C12AE8659B6509E0C14A59FEFB7CBAF441CC10F819D270FEC109E5589331C9FDC10013F1361FD3D8C1763F1E4C482AFA4169BA1846560FF4C15ABD403AB9F0E6C12C8904AE4147FB41ACEA818DC0D9DCC1C02F24C1A703D4419C9B4FF81D31FD41C8B0AFF035A3E441EE5A6BFE1479F6C1C02446EE5620F341E84D20CBDDF4EB419854A895BD81D6412E71849C010E0042A8D645A62F370142AA90188AF2F4F8C1A21892D7707BE2C158DDBA6DE908E341ECF82AAB0B12F7C17CE678FA62CF01C200C93AFE4B0AFC41804073788CD7A3C194CFE990C883F8C1921C710EE7F1F0412026FC629FCCFD41D67B13A65061F5C12CC7C142B1A8E8414098D0B490FD00C2A8B0B013FE7B00C26D70A6AB37C0F1C15EEDADEDBC7602420103000040010000000A000000C3596244D98301C20865B9D200E3FFC15CE507372837F4414C16AEA0FAB7F5C1899DD03AC5EFFCC15C8E2BF2E03EFEC1E2DA64BDCAD3F04166EBBD0E4B82EDC160F4E7BDCB0EFA4180C299335F4F0042A8CB03226E3CDCC110AE883C2DF8C441C0EE685379A30142C0191AB7A9D0DAC1EF4FCC10D06702C2E0C53CBD3DE1E3C14819B287B5F5F341383CE12A552AE641F5425CD25025F0C1E2A468AFB524F9415451A4B70A61E14158149AC66A8900C2604113FE667D00421447DABBBAD5D6C1EA0EB43CD4CCEBC190E1BF64A4C6D241F6F9605B756EFBC1C3596244D98301C20865B9D200E3FFC15CE507372837F4410103000040010000000D00000022DB9EC61B8EFFC184C107594A1CFCC192D367956EC2E3C1A82A727AD185D0C11A97ABC34C66F7C1C45F396AC489FDC1387BB6F92FA0EF41E476559BD59802C231EEB47B3C31F0C1066CA21476950042C7411264CD5701C288E185E60CA1E841EC0A7F77A5CCFF41E8AD3B1C98ADF6C10DE2B0C5C6ABF1C16CDD6C0D4B89F841086B3E6BC1ACF741103BFDBAAE2101C240FFE4CE4430FD41BEF7CE82B4D1014279998A120D74F0C1CA76167F33EAF94198D40A9140260242D821AEC5D43A0242B4C1FEBCA818E44180CF6AA4C51CF941ECCC6564CCDAF9C1401EFEEDBBC1D1418C3298DE383CED41C189F664E6C3F9C18AD8BF6531FFE8C148AB05A1C3FBF541848C2E47E9C0F541D37E2E83EA70F4C1D43D01C9B216E041EA2FF7639B02FA4122DB9EC61B8EFFC184C107594A1CFCC192D367956EC2E3C1010300004001000000040000008B441E2BB809FAC180DBD219044900C25078776E0178D2412FA84F9C32F4F5C1501073AC5EA7C4410EDFD2F66F3BFEC186DE461C2F13FFC1300FD42E32A801421690E3506E4DF3418B441E2BB809FAC180DBD219044900C25078776E0178D241010500004001000000010200004003000000B46ABAF23755DBC11847CE16602600C2A0E65B7B3111C7C170C80E9B59F1FF41826D40DE4FADF3C194D9D10A0480E941081D1CC2F2A8E1414870A22348DAF2418156CD60681701C2':::GEOMETRY, (SELECT NULL)::float, (-0.8222967098773971):::FLOAT8, 0.9384525037766186:::FLOAT8) 
    rsg_test.go:125: Last executed (39): SELECT st_pointonsurface('01060000000800000001030000000100000004000000A548CA37B9DBFEC1A68DAFF0596DEFC1C6DA8F25C0B200422E88CB3ECF17F54169C899D51F23FBC138FBED5C39F8D141A548CA37B9DBFEC1A68DAFF0596DEFC101030000000100000004000000C1DFD311E919F6C1CCC84B49F086E5415B6E36ADC338FDC1C043B0B57371BB41BAFB592BA7E1F94154DDF0882A32FF41C1DFD311E919F6C1CCC84B49F086E5410103000000010000000600000000DEAAE8CC5BBFC14CB3458CE87C02C204250B056F89E941D8F34315C0B4F3C1E0EFED8E49750242FF523E60C6ADFEC1D80F6F1D96FE0042A0BCE7D1CC18ECC1F78C55891DAAF3C11A793962498AF34100DEAAE8CC5BBFC14CB3458CE87C02C20103000000010000000B000000486818C0EA96DE411C940AE45D79EBC150538DD9E269DF41CF01CA025C9CF2C15C8340075720F941733BA4B7CE0302C2940FE4CEB478ED41389023AF6CC8D5C15A37E5CD4CF2F24160AF6095D764B44184DD88329871EE41A054B8027875DB419467698D6606FE418635FFAC61F10042C09F2FB607DDB4C19C67878311F1F141CAF972848BAEE4C10C50CDC78A90E74136CDC6D2432AF2C12829AB62C705D441486818C0EA96DE411C940AE45D79EBC101030000000100000007000000747E07FFD497F7C100780E8F912CCE41BA3F7523794DEAC18CA84AF9C00AFFC194525A592420E841C006369B30E0A441F415C77CCC23FF412CF8E5CD7E04E741B3A6A5BCABF6F1C1163A79EEA1870042070BA6C1BDDFF0C14C2FC8DE461CFC41747E07FFD497F7C100780E8F912CCE4101030000000100000008000000F2FC1DBEBBF5E1C140C6BF98A207EC41B2DCFC33E0CAEEC1602BD8370F4BD0C10841A5BF187AE6C140CFEBB7343EC7C1521679573AA8F4413C71D9ED27DAEE4148398B9FC0A3DE41F03860380775FE418018561C11A8B6C1701ECCEF57FB0142E0F9A7A5672B01C2A237EBB5399F0042F2FC1DBEBBF5E1C140C6BF98A207EC41010300000001000000040000002891FC634073DFC1F86B9B47AB41F8C1F0A854110FACCA4192E0D20E18D0EFC1588CF13D0BBDE041D0B227D61FA9F8412891FC634073DFC1F86B9B47AB41F8C1010300000001000000090000001584A2B0DABAF3C1A487722ACB86FBC10A74A57B6609F041145697B3A8CDFFC11853906814F0DA41F050A96DCE80D2C1BE1227FACE04FA410019457EF315A9419C95DD15992A02424625DE9ABC93FA41389097214F6EF6C1405809D77963FB41376C518D411BFDC174C3AFAEE421FA412B629E8A16B000C268AA7F024BD7E4411584A2B0DABAF3C1A487722ACB86FBC1':::GEOMETRY) 
    rsg_test.go:125: Last executed (40): SELECT st_pointfromwkb('\x':::BYTES, 6675786848432072721:::INT8) 
    rsg_test.go:125: Last executed (41): SELECT st_points('01020000800500000028F4A50CBE80EDC1B79F460C058801C2B887DF3B1C06E24146DF96F29106F141D8649735FBEDF2C1EC18ECE41FB6F041AEB22B6AC8FC0042981EA5FC1FC8F1C1C83D58511FB4DB41643592F8F345F941D0694673286CD8C1BC72B22208B4FF413C894117C9180042A8DCA62ECE68D641ECDE863BE83EFF41':::GEOMETRY) 
    rsg_test.go:125: Last executed (42): SELECT st_polyfromtext(e'\x10$':::STRING) 
    rsg_test.go:125: Last executed (43): SELECT st_polyfromtext(e'pI&\x13zAZ\x05':::STRING, (-532808187883101985):::INT8) 
    rsg_test.go:125: Last executed (44): SELECT st_polyfromwkb('\x':::BYTES) 
    rsg_test.go:125: Last executed (45): SELECT st_polyfromwkb('\x7f671931ef54':::BYTES, (-1):::INT8) 
    rsg_test.go:125: Last executed (46): SELECT st_polygon(NULL, (-2239317963377155878):::INT8) 
    rsg_test.go:125: Last executed (47): SELECT st_polygonfromtext(NULL) 
    rsg_test.go:125: Last executed (48): SELECT st_polygonfromtext(NULL, NULL::int) 
    rsg_test.go:125: Last executed (49): SELECT st_polygonfromwkb('\xc6c699':::BYTES) 
    rsg_test.go:128: Currently active statements: map[]
    rsg_test.go:515: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2238271334
--- FAIL: TestRandomSyntaxFunctions (301.37s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestRandomSyntaxFunctions.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20690579?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20690579?buildTab=artifacts#/) on master @ [8e56ad9c46e544c2ef368e02322d949a03723891](https://github.com/cockroachdb/cockroach/commits/8e56ad9c46e544c2ef368e02322d949a03723891):


```
    rsg_test.go:125: Last executed (4): SELECT lead('[[null, false], null, false, 0.22670088546451322, true]':::JSONB, 9121051895197901027:::INT8) 
    rsg_test.go:125: Last executed (5): SELECT lead(B'0110000001100011000001101010100110110110110110000001001010000010011111100010000111', 1354689008448637857:::INT8) 
    rsg_test.go:125: Last executed (6): SELECT lead((SELECT NULL)::ltree, (-7477847489472580824):::INT8) 
    rsg_test.go:125: Last executed (7): SELECT lead(true, (-8623167826485141587):::INT8, (SELECT NULL)::bool) 
    rsg_test.go:125: Last executed (8): SELECT lead((-663693619918113229):::INT8, (-7945986493568125580):::INT8, (-8473098095323771142):::INT8) 
    rsg_test.go:125: Last executed (9): SELECT lead(1.007196626985312:::FLOAT8, (-2057012519234466716):::INT8, 0.6027045175263215:::FLOAT8) 
    rsg_test.go:125: Last executed (10): SELECT lead(NULL::box2d, NULL, 'BOX(-1.7483540296081932 -1.761648327808457,-0.6464972151718014 -0.7615091365716019)':::BOX2D) 
    rsg_test.go:125: Last executed (11): SELECT lead((-57596627.95618160829):::DECIMAL, (-4160558218061858457):::INT8, 76.00660654360011119:::DECIMAL) 
    rsg_test.go:125: Last executed (12): SELECT lead('1990-05-25':::DATE, (-8324068945375596452):::INT8, '1989-07-26':::DATE) 
    rsg_test.go:125: Last executed (13): SELECT lead('2024-10-12 12:59:10.000525':::TIMESTAMP, NULL::int, '1993-10-05 05:30:11.000974':::TIMESTAMP) 
    rsg_test.go:125: Last executed (14): SELECT lead(NULL, (-3805974689824265846):::INT8, '-72 years -5 mons -610 days -03:37:57.781731':::INTERVAL) 
    rsg_test.go:125: Last executed (15): SELECT lead('010300000000000000':::GEOMETRY, 4845526943110568082:::INT8, '010300008001000000060000009D42F2E1AB0FFFC1F4F353DDB6DEF7C1343452178104E8C1AC1E279DBC25D9C106D570B45495ECC19D088495C2C1FEC1DCB0F9B5A56101422037E92578B7C2C1E80A4C1C5B3BFBC1C0F756CF6D09FE415051F99DF4B5C9C1DE5DDBFD5BCC01424090ADCED5F0EAC180026A5F0D9296C14844ECE9CCEAF4419D42F2E1AB0FFFC1F4F353DDB6DEF7C1343452178104E8C1':::GEOMETRY) 
    rsg_test.go:125: Last executed (16): SELECT lead('01010000E0E61000002806562C0438434070A5D3CB5B3E5040FEE154BC5148014232266627E38DF5C1':::GEOGRAPHY, (SELECT NULL)::int, '0101000020E6100000D675E238AFBC62C00491B1943F4030C0':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (17): SELECT lead(e'\x1c&;^W':::STRING, 127:::INT8, e'[ue.\x16Ze9':::STRING) 
    rsg_test.go:125: Last executed (18): SELECT lead('\xa87b28':::BYTES, 3109844040103802550:::INT8, '\x22':::BYTES) 
    rsg_test.go:125: Last executed (19): SELECT lead(NULL, 4451809512898895561:::INT8, '1970-07-08 03:12:51.000052+00':::TIMESTAMPTZ) 
    rsg_test.go:125: Last executed (20): SELECT lead(0:::OID, 5988944975724269451:::INT8, 3679993362:::OID) 
    rsg_test.go:125: Last executed (21): SELECT lead('f5d88ef0-9620-4ac2-9057-7b64006ca429':::UUID, 0:::INT8, 'ffffffff-ffff-ffff-ffff-ffffffffffff':::UUID) 
    rsg_test.go:125: Last executed (22): SELECT lead('192.217.196.41/30':::INET, NULL::int, '43.159.177.255/0':::INET) 
    rsg_test.go:125: Last executed (23): SELECT lead('FECC1EDE/FD99FF3B':::PG_LSN, (-1841059098659283058):::INT8, NULL) 
    rsg_test.go:125: Last executed (24): SELECT lead(e'$3\x11':::REFCURSOR, 3580479691039297139:::INT8, e',^d\x05\x15\x11\x7f':::REFCURSOR) 
    rsg_test.go:125: Last executed (25): SELECT lead('12:49:01.258281':::TIME, NULL::int, '00:35:48.706478':::TIME) 
    rsg_test.go:125: Last executed (26): SELECT lead('16:50:03.100498-10:42':::TIMETZ, 5466788207329515554:::INT8, '00:00:00+15:59':::TIMETZ) 
    rsg_test.go:125: Last executed (27): SELECT lead('{"AZzkXP": {"b": null}, "c": 1.4562388279675567, "cyZaR": null, "foo": {"KI": {"Y9JTQvL": 0.2646474758404974, "ekbl07u": 3.3596188266193305}, "a": "IT", "b": true, "cx": "Ad1I", "foobar": [null]}, "lZ7hkvK": {"U5": false, "foo": {"c": false}, "w23L": null}}':::JSONB, 7603313575603075910:::INT8, 'false':::JSONB) 
    rsg_test.go:125: Last executed (28): SELECT lead(B'01001001001110', (-3373041522216721529):::INT8, B'111100110110000100101110111101000011101101101101010110001111111011010110010') 
    rsg_test.go:125: Last executed (29): SELECT lead(NULL::ltree, (-5864905916823942440):::INT8, 'uCpN274.1g0.MXakPe.U.E2WQ-Gu.XgM7W':::LTREE) 
    rsg_test.go:125: Last executed (30): SELECT least(e'\x1dD"\x12R':::STRING) 
    rsg_test.go:125: Last executed (31): SELECT left('\x9a1b9618032e3845':::BYTES, (-4766086768169704446):::INT8) 
    rsg_test.go:125: Last executed (32): SELECT left(e'.b6F\\2@':::STRING, (-3811000271435813220):::INT8) 
    rsg_test.go:125: Last executed (33): SELECT length('Mv':::STRING) 
    rsg_test.go:125: Last executed (34): SELECT length('\xae':::BYTES) 
    rsg_test.go:125: Last executed (35): SELECT length(B'0000000000000000000000000000000000000000000000000000000000000000') 
    rsg_test.go:125: Last executed (36): SELECT levenshtein(NULL, '  ':::STRING) 
    rsg_test.go:125: Last executed (37): SELECT levenshtein_less_equal(e'Qri\x13=':::STRING, 'pK29UGG+':::STRING, (SELECT NULL)::int) 
    rsg_test.go:125: Last executed (38): SELECT levenshtein('<2;34NW':::STRING, '':::STRING, NULL, (-3860203373367021301):::INT8, 87820957085617184:::INT8) 
    rsg_test.go:125: Last executed (39): SELECT levenshtein_less_equal('':::STRING, e'\x14FCP':::STRING, (-6993308210119810263):::INT8, 2316459439438769680:::INT8, (-7285148300267153962):::INT8, (-6982483727317508186):::INT8) 
    rsg_test.go:125: Last executed (40): SELECT like_escape('N':::STRING, '-+':::STRING, '>AD.b@gsQ':::STRING) 
    rsg_test.go:125: Last executed (41): SELECT ln((-0.6643688365028491):::FLOAT8) 
    rsg_test.go:125: Last executed (42): SELECT ln(2.34116100964106861E+31:::DECIMAL) 
    rsg_test.go:125: Last executed (43): SELECT localtime() 
    rsg_test.go:125: Last executed (44): SELECT localtime() 
    rsg_test.go:125: Last executed (45): SELECT localtime(8693316915355648945:::INT8) 
    rsg_test.go:125: Last executed (46): SELECT localtime(2669130148754778656:::INT8) 
    rsg_test.go:125: Last executed (47): SELECT localtimestamp(NULL::int) 
    rsg_test.go:125: Last executed (48): SELECT localtimestamp(5761671587620575812:::INT8) 
    rsg_test.go:125: Last executed (49): SELECT localtimestamp((-5116318024747177618):::INT8) 
    rsg_test.go:128: Currently active statements: map[]
    rsg_test.go:515: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2372664486
--- FAIL: TestRandomSyntaxFunctions (300.68s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestRandomSyntaxFunctions.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20696420?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/20696420?buildTab=artifacts#/) on master @ [05c6218cc548c914bcf0925bde1d1a9000854056](https://github.com/cockroachdb/cockroach/commits/05c6218cc548c914bcf0925bde1d1a9000854056):


```
    rsg_test.go:125: Last executed (4): SELECT st_asbinary('01060000400400000001030000400100000004000000A0831A7B0F52004281E6C35315D5FAC19C1149425226F641B8E0D333AA97D9410068A1BDD53D9D4108E4BF641C3AD641E0588E17D270C8C1909D744C7576CE419AE82FCD2164E6C1A0831A7B0F52004281E6C35315D5FAC19C1149425226F6410103000040010000000B000000346A60EAA89AEDC12853D637D469E541E0416456E90AB841E18D44F18BBCF3C1F0011DF07FFF01C280E59C501054C0C1C0CF79885789F541B4D02AFCD34FE9C1E4BF57E8FED3F34160B63AF36E79E241B0DD2E29C029D741E4F725DFE335FD414A3F9587444FF2413E5F8B48FC6CF34180FB55A68B1BDB41B6CB0DB23274F64158D3E429195F0242DC393758FE8BEAC1782DDFBA31B2D54110E1A038DFEFF941AA803B3BF4AFF141308BF81AD3D1F7C190D7149FD2E2FB419C493A6F1643E0412A874AD4090EF4C13052A2179D98F2413EF00CC026BBFA41B75C509D627601C264857CE170ABE9419C79B171D2F3F541346A60EAA89AEDC12853D637D469E541E0416456E90AB8410103000040010000000A00000033FE03D7AC6F00C2DA17EBF15399FAC1B06195BB4559C641009A2D7DA9768E416AE48BC47E5A00C2DE4AC2D4BB10F8C108B50A06CE91E4414E6C11357B4CF7C15809EC6D33A6EA414806C15B04E3EF4196976CF87734F8C1FAA90EE29A24FDC1D89AB88A3C0601421271DACFC174EAC19CFF70CCBBC3E341B892DF81BBE4E541B830A17F34C7D8C170E248277F5FF241980D5EF29052F941E0931B5C7758B741CCDA50EF68DCF04190D19642372AEFC1D055544AD3C7F841922498B8E85BEAC1E97FAFCDB7CB01C21C2CC2C3E09AE841009DFF8C0B34C1C133FE03D7AC6F00C2DA17EBF15399FAC1B06195BB4559C6410103000040010000000C000000F11B85F3D657F5C1AA366D412F86EBC1E0084D13BC5EC341E0B2335FBFBFCA41EE4530C33B9FFBC1141E6F721302E041D4F3097B3880E541862E0E339390F9C1D036C8C3AE3AE84116DDD5C422E8F1418E60E977562DFBC15CA021774A88FB4130074F88E637FE41061F4A34DD22E1C1721351F721870242182115523FCFD341C04D4316BB65C5414077DA29BD7ABD41D2F5567CDB57F4417A327A4EE2DBF1418F74B08A5738F4C1C0CAE349001EFA41FA65043EA6D0F74148DB4016595802428E96CC882854E3C110409780A2A3EE41F0245C8E3790F3415170DA66C99201C2C842E5D2D8BC0142961C0726FC92F2C155C4B02BC274F8C122541C0F38B0F5414004EF876668AC41F11B85F3D657F5C1AA366D412F86EBC1E0084D13BC5EC341':::GEOMETRY) 
    rsg_test.go:125: Last executed (5): SELECT st_asbinary('0101000080B846E6DC00D6F441C0AC3517ECCEB7C115F6716F7D4700C2':::GEOMETRY, e'/\x0f\r':::STRING) 
    rsg_test.go:125: Last executed (6): SELECT st_asbinary('0106000020E61000000200000001030000000100000004000000000000000000444000000000000044400000000000003440000000000080464000000000008046400000000000003E4000000000000044400000000000004440010300000002000000060000000000000000003440000000000080414000000000000024400000000000003E40000000000000244000000000000024400000000000003E4000000000000014400000000000804640000000000000344000000000000034400000000000804140040000000000000000003E40000000000000344000000000000034400000000000002E40000000000000344000000000000039400000000000003E400000000000003440':::GEOGRAPHY, '``':::STRING) 
    rsg_test.go:125: Last executed (7): SELECT st_asbinary('0101000060E6100000E0B3517A692F3FC04F71B4FECABD54C01422058DBB20E141':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (8): SELECT st_asencodedpolyline('0102000000030000000022E1B735F301C2FAD60799BDB9EAC1A9AB604F3C2CF1C154A0E38D6184FEC1A0A268BB016BD54190CC167DE7D7C541':::GEOMETRY) 
    rsg_test.go:125: Last executed (9): SELECT st_asencodedpolyline('0103000000020000000500000000000000000000000000000000000000000000000000F03F0000000000000000000000000000F03F000000000000F03F0000000000000000000000000000F03F00000000000000000000000000000000050000009A9999999999C93F9A9999999999C93F9A9999999999C93F9A9999999999D93F9A9999999999D93F9A9999999999D93F9A9999999999D93F9A9999999999C93F9A9999999999C93F9A9999999999C93F':::GEOMETRY, 1275155233:::INT8) 
    rsg_test.go:125: Last executed (10): SELECT st_asewkb('01030000A0E61000000100000007000000EC47914E47484CC0ECE85A23D51851C0C0B48E2FD141C1C18C29A259351747C06E9ED2197A7056C030F1C83AACEAE641064CED6AC9325240705A577ACA1D31C0B026D62DB7BBE841D0C39ADF4E3B314020AEF75329570E40AD847154B4F7F9C1708D8C618E3942C0B8E8DF44968A5540908F409A05F9C441C947A76D080965C090CA61C835A51D408C5ADAC760CCF8C1EC47914E47484CC0ECE85A23D51851C0C0B48E2FD141C1C1':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (11): SELECT st_asewkb('010300008000000000':::GEOMETRY) 
    rsg_test.go:125: Last executed (12): SELECT st_asewkt('01030000C0010000000C0000005682C2C75AE8F5C174BD90554017EDC176C139A9030B02C21436F945A2E1E3C14E54FF013491F2C15ADBA0C99085F1C14AADC7907D0BFAC148EA81134ED9004280552A1736B6A141DFB9C5214296FCC1BECF626D52BCEAC192000093C507F2C1646B0159A1CEE54152AB4031B85502C2A09FC8202B8AE641B037C0C09139E741B0063895EB6BFC413FF6A4756B4401C2F847E8425DE7F4C1C223AEDCE4F5F8411C321D983C12E3419C7D54A461C3E6C134FA8E2130FEFE41E0071F3C9B63F541BCF6FF07EF26FF41EA6D9AE407BCF241A721669CE071F1C1327AD2D5F77702C218A56725324C0242CCDE886F11090042AEF388AE42BB00C29CB2EECB05BBE6C1D8F2D95EDD18F8418A42E20251FAF341179B9EC169E9F5C180A3FF032E27AAC12EC38D83410600C2A00B361034C5DD410C7B79E1279BEEC1D03B4F0FB8F501C2A4C937D78C1CF7C10086CD2438AA9DC11D4D75137A6BF9C1271AFA6222DCF1C15682C2C75AE8F5C174BD90554017EDC176C139A9030B02C21436F945A2E1E3C1':::GEOMETRY) 
    rsg_test.go:125: Last executed (13): SELECT st_asewkt('010500000000000000':::GEOMETRY, (-2128885290806827320):::INT8) 
    rsg_test.go:125: Last executed (14): SELECT st_asewkt('0104000060E610000004000000010100004004BA60A989594140FCC0D32771AE35C0BAB0BD82C0C1F3C1010100004084D6CE04232A644010CC29C7F7CF3740C41DFDA996FC004201010000409480E1D88D2766407C630510567554C0909D239BF8B0FFC10101000040485C37A703FC39C0B86F4667D72F4740F8AFFB68C5130242':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (15): SELECT st_asewkt('0105000020E610000002000000010200000003000000000000000000244000000000000024400000000000003440000000000000344000000000000024400000000000004440010200000004000000000000000000444000000000000044400000000000003E400000000000003E40000000000000444000000000000034400000000000003E400000000000002440':::GEOGRAPHY, NULL) 
    rsg_test.go:125: Last executed (16): SELECT st_asewkt(e'p\x14D8b\x15':::STRING, 2147483647:::INT8) 
    rsg_test.go:125: Last executed (17): SELECT st_asewkt('X':::STRING) 
    rsg_test.go:125: Last executed (18): SELECT st_asgeojson(('1998-11-13 05:41:19.000346+00':::TIMESTAMPTZ,)) 
    rsg_test.go:125: Last executed (19): SELECT st_asgeojson(('{"C09": 0.925152601960447, "Cz1E": {"YF82O": null, "dPS0k": null}, "IR5PEp": [false], "Woe": true}':::JSONB,), e'..[\x15ZL\x7f&)':::STRING) 
    rsg_test.go:125: Last executed (20): SELECT st_asgeojson(('{"4e": null, "SiWNkK": 2.90411564625525, "foobar": {"vY": "b", "wO": "2TIjw8tDB"}}':::JSONB,), 'r+':::STRING, 8884196558915717762:::INT8) 
    rsg_test.go:125: Last executed (21): SELECT st_asgeojson(('782FD4A8/C5849823':::PG_LSN,), e'\x12>\x06z2agl':::STRING, (-9046085317773296981):::INT8, false) 
    rsg_test.go:125: Last executed (22): SELECT st_asgeojson('01050000C00500000001020000C0050000001CE8D900D7E2F1C1007F57DAB660E4C1C07B1AA6233AE3C1A448BFB611E5FB41294127C6BE48FFC1FFB468276F5602C2402B09B5D127C94140316A0F18AEFE41B887530EC73BD0C190E927CC630BCCC1209926BE607BCC41BE6DC4EB1EBA004220E33A57D6FFC9C128EC02FE1EEFD3C130FCD48ECE43DD41ECBE8A7993A8E6C120B916D4C577FD41AE38FA87BB90FBC10A01B4CD84B9EAC10C397512F260FDC101020000C003000000B045FA14EECEC941CDE4456F0F4700C268668A82571DFA4110F067AD9116C841BC58B709C119FE4140306A121DB2CBC1F0A31D02E456CFC161E0A9D5CC5401C22C65075A57BEFA418059FB0064E9EA412401F54300A7E6C1D350D306A34FFAC101020000C0080000005ABA2D862EC100C27CA1437CBF33F0C10A8B9A7A5C7901429C5938168052EE41309CA10AD6A9D8C128E3DBC13D79EBC1E86BAADD0F4601C2D06765E9BB93E341A085E591090DD9C1D56DB0EA749EF2C168065AACBE00FCC120B15E0C849BEEC138FDD77B93B0D9C158E5B8B0658BFEC1C72D5F5DEDE0FBC162CEAA5F81BCFA412CF559852951E941F6485ABB4347F1C1DC6E327CC916F2C1E4ADF67F20BE004262257C5F636E0142A0D11CC4A8B8FB4166EC0C889A42F4411BB7CC471FC901C2F83F130EBE1ADF41E8851E385898DC4198D4AE099852EF41F4A2B791E868D7C11089F5C68C2DFB4172FF07ECE3D4F94118C8676268C9F4416E6AB6B32AB6F6C101020000C0020000008E2395DC4D2F0142A97E5A1AE050FEC1C099BB13D1FEC14198E81566C267F64180B550DD16B19741F6204B605312F441D62C31A5A72AEFC1CCBCD5F538EE004201020000C002000000601D71272236C0C120EB15301EEEB14148715F1FD999D141A1DC6460BE16FEC1D407D079B672EC41B0CEC4F90F2BF9416C5706A97C69E741B43237389CE0D6C1':::GEOMETRY) 
    rsg_test.go:125: Last executed (23): SELECT st_asgeojson('0103000020E6100000010000000500000000000000000000000000000000000000000000000000F03F0000000000000000000000000000F03F000000000000F03F0000000000000000000000000000F03F00000000000000000000000000000000':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (24): SELECT st_asgeojson('01070000C00600000001040000C00800000001010000C0A01898B46E61FF41C4895F9607C0FE4198559416BAE5E54184E4F73DEF6AFD4101010000C000DC76A093C489417244DEE0B29EE1C1707FE3DE291EF841AC1B371D4F4FF04101010000C09B03B6A507C200C2C620AA8DCD230142041D41C109B7F2418861990038F9D54101010000C09A3E02E7BA85F041A8631B23D1EFE54178BD40697DD0F2C13084DCCB8228D2C101010000C0CCB85F9CFEEDF1413030F68DA4FAF341C46DB81870D1FA4126C3DC3A07DE00C201010000C0B87A85575B32EAC108B0D25C58DBEE4121378DB2C6B1F7C1B0D401B4FB3CE24101010000C0D1C76F036432F7C17076FB1195B0C8C166E096C41EB4F94166769D9E704BE9C101010000C08CB1D2208FB7E74128A6CA5CDFEEF041784B1CD6A9E9FC412F8F4C082CBC01C201030000C001000000050000002E97234060DEE4C1ECC246A3C64CDAC19E57A47E3714EBC1D9AED5EDDFA1FBC1C03B5A8523E4BD4100FFAE8183AEE1C1E0046FA768B9FDC11872E549C363D74170EBC1F4187EFB414CCC94A4A63DE341AA44D6D94D3FFA4192B8C3CE3688004230EEC060467FFDC1C493E2718AD0F741B0D0C711B415D4C18C1AA6459689F1412E97234060DEE4C1ECC246A3C64CDAC19E57A47E3714EBC1D9AED5EDDFA1FBC101060000C00600000001030000C00100000006000000666A28B9053001C240EE421F9632A8C1954396BE12CCFDC16E8A42880133F84100B167A21DBCFAC180407D007C64C8413264C1830F11E0C1A49734FF198EF9412EC8BB9F1C2FF2C1B269CC4C7582F9C1784AEB926011E441C812F7AF502DEB414CECD5C1F6F7EA41BA77F596782F0242705FE965DB47F84100A3C8C9E72080410408E36E259DF6C1209481B70DD4FA410874CE22BFAED34190E343882124FBC1666A28B9053001C240EE421F9632A8C1954396BE12CCFDC16E8A42880133F84101030000C00100000005000000422A06A3C61A02C238BFEE95844FEBC17B726755BCD301C220F11C73EDCDDA41B8F2904EF83DFF418293FB31BFD4FFC1843794D1E6B5E7C1441BAC1B5F60ED4198976590AD44D1419EE38E65F9D1F941DD99064EFC9201C2440F11B8F97F01422C97BE12E460F3C15CF7F65F6B0BFE4130ED7D4117F1D8C10A12993C120AE2C1422A06A3C61A02C238BFEE95844FEBC17B726755BCD301C220F11C73EDCDDA4101030000C0010000000A00000021C5FCA526E2F6C1E4814F1FFB4402C27C74D94DD72BE3C14846C2B76E36E9419BD0B1646E22F6C1336DB57B3CF901C2405AFEAFB5C9FDC1D46478E778C5ED4100CC8958B446A24174C561A69F30EFC1F563A8F578B5FFC1B2D10FC84E9802421422538C6D2FFF41C045B2231F78C441E20B25C78029F7414C312CAE49BFE041385940A3F772EB41C0AD6B26AB0802424EDA87FC0472EFC184CF68F13A38E84160D7C86D36F9B9C1C0D60B18A181DC417ABBD9EC104FFBC184393D837AB6E641C0729669F3DAAEC1D060E66A7689F4411CE4A08EDE6702426096B2C31B80FE4100B065B197885041929F673B65D3014238A4C25A00D6D741402D3259FBBFD34196FE0CA2EDAAFFC1D82B352B1A68D241C091B7815EB8AF418E1002302D8BE3C121C5FCA526E2F6C1E4814F1FFB4402C27C74D94DD72BE3C14846C2B76E36E94101030000C00100000004000000093E185FED99FCC1D4D7E3109DF0E5C1325AC0F60C6C01C2EF4B4E66BB35F1C102307B6141060042BAD35BBED29CF9418CED687BA513FC4100E6E7D54734934148ABF3AB6A73D541A819A4BE1415FE4128F1EB01416CE6413A33E49E0EC2F941093E185FED99FCC1D4D7E3109DF0E5C1325AC0F60C6C01C2EF4B4E66BB35F1C101030000C0010000000B000000161DFECDF482EBC100E0AAB6028D304169521D8AD45FFCC100C6FAE857560042D8A748C9C22EF9C1989E9AB42520E3C19246A9DB926FF6C1D2C4730F78B5EDC1A425397C482FE1C1F3B6F3213DF8F9C1BE443755A34101C228CE00B78470EBC1C80D3E632AD3E64188D2FD36A0F5D0C1C04D68DDC770F9C1A06CD0D9B3DCD4C1A4F703AD4AC0FB41D0902F3BBD31D941202311852792B8C180D512023F8CFA41780EE013BD77FF41385D1C90B93BE841306F671EE06EFE41A40CD211A250FDC1D0D20D72512BFD41A2027DC2B0F00142E44795B3ACC7FF41CD1B8043B28BF2C150EC9F0972A4C9C1000B221CF1D7EA41C094B6A83E6CD1414ABB39A0DA1AF9C13865D6E8E621FBC1B87AE9410B12F641361EED0AAB5FF0C170BAA104E54BD541172F454C240500C2508CD2520F28EE41322DBDC03233F441BFDAE4B4D89CF3C1161DFECDF482EBC100E0AAB6028D304169521D8AD45FFCC100C6FAE85756004201030000C001000000070000000A734E3E178DEAC132EF0487FD5DE6C17CC3915CC648EC41E6F2B0CFA7F8E3C17606C87CEBE3F0417C19273A1920EC41A4B5289E8500E7418B845B4B117600C2DC29328367EE01426ACF5883BF11F8410265FF29F540F3C1CC36C1BB9458EF4106A1B9E2103BF341E809A573F074F741072C63E2727F00C2280227633DAFF5413091FA9C3ACDD6419470ED7CD71DF441C67764A84ECAF141D21CF2CA816BEFC138D866D90833FEC1E6DF9E8F10680242E27CCC41E724FDC1ACBA31EF5CFAF3410A734E3E178DEAC132EF0487FD5DE6C17CC3915CC648EC41E6F2B0CFA7F8E3C101040000C00500000001010000C0388794F94D90D4C19A7C32885412F841888B4A2BF556F141760FB53771E9F6C101010000C0100F778A1DF0C5C19420244A5AC2E1C156CFAA087F5EF5C15C9AD765E11BE34101010000C0D2E70D702F19EEC134708F7B268FF5412E8BD281AF31E1C1721E38D95296F7C101010000C0FA6F6A62BEC9F8C1FFD99A27E2C3F2C1C63EECF61398F141C0E8849A9DF6DB4101010000C0B61CA97D130A0042466ED254E6E4F7C1CAED6DCCD3C2F841C8781C2116B0FA4101030000C0010000000B0000003B50C3D6421DF4C128F320CBA8E8E0C10D3A818DC006F4C181D548AB01D300C28C3AF6B3B616F7C10A27A1B7E41E01C2F8A4D569EB84D541E067713FA559B1C1209E646539D7BCC10A125DC92040FCC12201B095EC44F041604A39E637DAF6C10C00214C5E50FE416EA2D6A2E889E9C146D3186AB37F0242B49A98AD64EBFB41963814C64390F84120D68FC6321AB7412EA3CDD45E06F1419492389866E6F3417AF2263EACE500420CBCB7F5BC19E9412253BFED0178FA41D4151812DDB2E4C128544E03DF47FE412C817CD3B62AF341B39E1F1FC1CF00C2D34082C6D6ADFDC1E0944FDB9577F6C18E1D95A67AE701424034AB883342A7C1A837F323E6A9DE41BCABEAD5ECACF3C1002F58D089ABEA4154D1053BED5AF7C1C25E10621561F4410CC4962D28F200C240A2FD299F8FDB41E2C802E61A36FCC1906924A8EDA0F5C13B50C3D6421DF4C128F320CBA8E8E0C10D3A818DC006F4C181D548AB01D300C201020000C00700000028A80472A28E02C29057C37B1E56C4C1DDE6BAA28B1D00C2102E2D51AE2AD841EB0710C730D9FCC1EE51489CA49DF8C18033D5F0F6DB9FC140F0C96F3BD9C2414020692D4EF2A841A057816A21A3DC410435A96FE55AE3C19455112FA264FC41348531536A12014298F3F2B3F770FF41C0E2C8FA199101C22C6F7D54DD30E541E02529208087CD41CE56AB6A053FF441BADA0B1A12ECFFC14EB9935A095302C280E7B10E095AB6C11E9908271AB0F04134BB73D9741AE641EE5B6AA342F4FFC1CA3E2D8AE6AE01C2802E1D937113FA4170E35AF842D8F4418893779E704C0242':::GEOMETRY, 3801592121822060974:::INT8, (-4302296564961979119):::INT8) 
    rsg_test.go:125: Last executed (25): SELECT st_asgeojson('01040000400800000001010000409086DCB13A9DED41C82A5563E904D54108091AB1DEC3FA4101010000403C80AD0BEE45ED417B27DC7AD4A7F7C18E02059B0E63F7C1010100004080F0151F2FFEBF413A03E4A3D855F8C1855A472DDF87FEC1010100004066AB2420486CEFC11D8939BCC945F2C12CD6DB9B0721EBC1010100004088150F4C33BEE5C1962DC99C7FEDF04101D6F8D7FC6001C2010100004068CB4644165EE44138A00E724231ECC1604C96509989FA4101010000409C0E794B097EFD41C85D62FB7295DB4154BA32DF99ADE0410101000040F0711728E59FE9C1B6C42012E34A024260FBD2E0A634C841':::GEOMETRY, 0:::INT8) 
    rsg_test.go:125: Last executed (26): SELECT st_asgeojson('01070000E0E61000000800000001030000E0E6100000010000000B0000008F4236BEA7A664C0C07D4878F08338C0DE9047B275D3FCC194430EA8A54AE341F3DBD35BF82F63C000FEE71B43A046C01E2C875043690042F896574F9D02E941603F087D518341C0C3AA728A7CB852C0C031A27A0364C641F0B2AF751B24C4C1C458246DDE835A408474739A76B335C06A7BEC522664FAC12091484F39BFBC414830D4D62E1260407040C2A0213236C0C28D857D67DBE2C114E5821DCCD4E7410C6090AB350D4B40181CF7D6E209284020F2412C8757C641C06D5584259BD841A0D268AF2BC733C008FD2F765D573440085B2650D0C101C2CC429D8742EBFE41A4D1BC4B378741C07850CB67882C2240AE663E954F6EF04158EFB70E5E08FE414F36ECA5516858C00817F44B6BB92240812BDEA60A7102C260C90A0C4C4CFC415040E57AA9A165C0503FF63FA2B03F40C26C5825F1CCF5C1DAA1F0261F42F8418F4236BEA7A664C0C07D4878F08338C0DE9047B275D3FCC194430EA8A54AE34101060000E0E61000000700000001030000C0010000000A000000E4BEA992167A5FC0DCBD3764160A31C0B0AE487CAD0DC641F690C54D4676F6411043C70A8FE442C0B43A9D5DC18147C0766ECBAE3394FFC18BC3BA29556800C280B30F9E2667614045CE741C77E347C07C93165F9FA1F741207D2492E016DC41B094716E2C135D4078D06B684B4825C06083C2C72563B641784472D88638F441FE2C2D7617AE6540D0EACD4B23E44540C8E8001F64ACF241B73CF5E9F69A02C2D6FE56D3549B5040BC7CF38AD0304C4000B4141D4EF2B141527FCCFB22B000C28083E271558A07408C577FE9785832407C014F390BDFF941E2AE8229825AFFC1A4638F7B51024DC0D0761443004F41404C20DCA4B39CEE413454B61BDA9CE04153DB9ECE0CF960C00C700CACC3C64540623B58DC23CFF54110368BBD1D73F941E4BEA992167A5FC0DCBD3764160A31C0B0AE487CAD0DC641F690C54D4676F64101030000C001000000040000007831668A6EB642C02A9AF796DD1450C010322A4ABDA1C4C188701AB49633DDC1B6270221DB546540C0B20EBAC8E421C02E9B94D9010901C296C391658CDBF14130EB33A91ADF48C0A03F68C2DB1C32C0E8B7D66161BFF1C13EA5779EB810EEC17831668A6EB642C02A9AF796DD1450C010322A4ABDA1C4C188701AB49633DDC101030000C00100000005000000D776B41D0C0A60C0B6246992A58D4EC02C4A1B9BDBBCF241C1468FFB5754F2C168D872CCD7676540A461EDEEB5DF484010CB17C0F6EDE2410859856B735F0142304E39D60B7932C0749A65C484324D40F8FF7A8F2BF7F241222979996416FCC1806A1E476C6145C0F2DD93BB56B755400AE0C8405E11FA41CCE07884F5C5E741D776B41D0C0A60C0B6246992A58D4EC02C4A1B9BDBBCF241C1468FFB5754F2C101030000C0010000000C0000007091CA6523AC64C019425ABAEAC540C0E0626703576BC8C170FF8ED6A738CEC1A7D5A1AB93DD53C0BE9F635C68CE53C0E68DA779894AE5C115072B0B6BA5F6C1E8DEFA8CF5D030C0209324DC7DDE43C0349E7CDB977B0142E09D4F308757F0C1003D98C33C491E4031E144DD34F052C050962C868DBEF6417285FA4E0E7B00C2E08F0A42ECE131409F8065E870E650C08E3D0E1C4AF3E5C12F9B39931C68FEC188543130B8C240409A420C76D2493DC08009D34AF70B954112B6D77E006CF341786D0BDED221604090B6D1DF565C334030FF4BE57B6CE4C130E84CB7B3D0C64160684BBCA15A6340B03388E97B7B4540367FD8C63658E2C16066BA7BA3BFF24198DFD955E24C374070D08A5D213A5340C0618EF91105F741A4301296B242F0C158C9AA2957F658C040F7AC2F2C3AFF3F7CE925C6E262F141681718BAF671F64156D9AACD4CD661C0C089EB52B3C81BC021D0F9227FEDF0C1D8F3886A4253F8C17091CA6523AC64C019425ABAEAC540C0E0626703576BC8C170FF8ED6A738CEC101030000C001000000040000009861143AC82C33C0D3CFD6168D634FC0662E081F10ADF4C1AC9942E8588EFE41343C8E982E1843402246E4D4CD2C534095452FF2783BFFC1DCD7F300E41501423BF2568B210865C0C2D1A43413D75140DC1275BF5A1AF3C114B4447DF941E2C19861143AC82C33C0D3CFD6168D634FC0662E081F10ADF4C1AC9942E8588EFE4101030000C001000000050000001E8B024F9DCB6440401DC44D866023C06065494F840BEA4190C839153DDBC3412C05FF27CB9165404AFBF36A10C151404041760B5EE5B8C1119D5C3DD36B01C2DC497A477DBA604090211E7600794E4014A2294B11B8E4418EE89E3823DBF14101DC0D90054862C06415576995E24940607B1A11AA62E94150F3F10C2AAECFC11E8B024F9DCB6440401DC44D866023C06065494F840BEA4190C839153DDBC34101030000C0010000000A00000018F918A48FAC31C0C885D582ED9B2AC08489B7AED2D6FAC1778048D85AEAF9C1F0290954792054C0EEEF27E8B29D41C08A3C9705467C0242488B54FB11B7F54188FFB9D260F843404524052B384147C0C07AFADC7CBAFAC19B1BE9FE2E6301C238950478764C57406791440FFC8F52C06C6497B407C5FB4160716B4CBC76BC410492DFE603225340F89DD07DBA8B21C0A670B5D01B47E5C1568562857793E8C17CA5126BA2985540BC44670DB9F75140C7B30CC80BEBF4C1E46A5D38ECF0FD4100CD041A1E6B13404CFE4B91962D3A40087310058721DEC1CDAF06D84A53FDC1660C9716D5BE52C0EC7756D646AD31409ACD3069A57DFEC1001882A519D7A741AFC3F91DEC3954C0F06E593FC09E1DC01021DB9E07DECFC148B5F0BCFA2DD34118F918A48FAC31C0C885D582ED9B2AC08489B7AED2D6FAC1778048D85AEAF9C101060000E0E61000000900000001030000C0010000000D000000C5EB7FF320A764C0901F6C4D25E14EC06E73994F0DD2F5C1B0F56F713908D64100372916CF8508C062DB138A536445C01AB98C4F4421E2C195059CD4FACD01C264544CD39F7B5E40EB3593F76D8845C0C51836A49D10FAC16688D0288DDAF54168788B36D2CD5B40E881BAF37ED520C00ABDAED61DF701C20A4C7D218FBA0042304BF34E24216240001B4F030249D1BF080F99AC284ADD417064473C041AC941F8BAFA0926574C40404B9DCC29225540380E06A601C0E3415C9E046D3EF8F141687E61482DF14540761477D455385640A6C6198E8B6BF8C148B9A67800F8DAC128CFAF8D614045C05A8058725E085140F241DFBFF3FBF84180108198647AB541FFEADFC4BAED57C0765924A7AD025640A0A667895577F1412C28E35558BB0142FDADB27EA2735EC0067B360A140A5640409DF9E35E56CA416DF01B8C82C6FEC14714D47DDEA05CC07A74B8FC8D725240BE805D1EE225FFC18001BDE08737D9C15241838CEA0E62C0005BBFC2235A5240D02735B0653BD2C10AA06036540DE1C1C5EB7FF320A764C0901F6C4D25E14EC06E73994F0DD2F5C1B0F56F713908D64101030000C0010000000D00000020893524BE0231408850EBB3B91823C0247BCF32A971F441BFB3596109F3F5C1B8E4D8EC91474A40B43EC7DE1E804BC076E4C5983CFFF7C1C062F57A16EFFC41049526EC6271494040712031AB1C2440C0286BCEED35FC41D2B6702F2BDAEBC1A23BF50195456340B80D60795D7444C0EEC15C29B34DF841E818D321C4A6FAC1A0007937B72F614040ABA536D2A81540004FF4DCBC259041D64B3E32789AEBC19A5EAE148A3164405836E36A62D72C40001AEC8162D2BE41CC0BA7154BCFFD4158516488162B5740DC7C5320619E484042A2C19AC02EFA41105BA2B18658F9C1F04F88C453645D40F02B7758E27052405BBFC06EA618F8C1FAC5615E4981E7C1881A91D85AD130C0100AA8A84BDB3D409C564B501FECD6C15A02BD0AB3D0F3412DF0FD24849C54C04C9FF04B83404640969BF8D7FA50014278F080DF4D9500C2C316FB69A38355C028868727538F37400ED24FD99AB900C2CC04E2546E470142F4DDD321F80255C034469F5E4DE3364085F67D246028F5C158D4C52AE7AED1C120893524BE0231408850EBB3B91823C0247BCF32A971F441BFB3596109F3F5C101030000C0010000000A000000D424E306B97660C04A60A26DB91D3AC0E08F6929D095FAC1A2787F28DC94EEC128477946F73C66C0FC0813578A8750C0C0D8189EE6C5C9C1446E8B245FE8EC41DB12868E634F60C0D5F7191AC8FE50C00E7E396E0484F9C1E4C424D0B785FD41284A8BCE3E0D3C40173CAB63145B4FC0688558DE53BCF6C12C0820E4E7E6EDC1E8BFD64647195540A890E1A9FACA3C404895C5D1270BD9415852AEA5F3E1EA4188ACF98565065640CC0525B458A24640E0518D9D6A07D041FEE0A13E4685F341E0DC6707DF0F1A408026340A4B183940CC47044F553EFF41F89AB59052E2E741D48EE86239294040842E83B0FD13534080336511D467FFC160988A1B6ACCD2C170AF09649CCD4FC020D71C85913200C098693122C4CDFE4166F639288EB0E0C1D424E306B97660C04A60A26DB91D3AC0E08F6929D095FAC1A2787F28DC94EEC101030000C00100000004000000AF4DF6190E1861C0A92B5806016853C05CD5EF3FB3FB00C22C2DC3AC2B6C00425D755F9EBD8053C00876BCFC154E2EC022389E0A2BDAF54130F6453B359DCEC1E9CBEF538A2260C0808F26A698F02D40DC34BC91AEF3ED41ACF03F04415FF941AF4DF6190E1861C0A92B5806016853C05CD5EF3FB3FB00C22C2DC3AC2B6C004201030000C0010000000D0000006A216691C48C64C0C07B9E9A76B7FFBFE073023807F0FDC1D0449C8D5CCDCCC1139C8912A9C55CC0A8E7D172627028C0CC9EF499F8D4D6C104E23BDD43E2E64130261184E0D853C05BFAA2B544064BC0E549028EA9A9F0C114EC3D3DA3B5F74108F92AE3B08D5B40E9FEF9B3B6E355C019DD0574E0B7F3C1B0B3E1BC4FCC00C224D6BACD4AD45240998401CE9B794BC06EB83EB9A89CF9C1D0168854342BFAC1B47B1A6FFDD35F40829E75E8425C48C0BF23075F9D1302C21D2A73B61D9200C2DC0FC96D39C35040A0881B31EEC828C0C00AC7242A10D7C18005437E552D96C1F0C1D86138F4414000DC5D781AD61140C045CEBC6758FC41804D53C8B6E6DC415A612DF8C66D514088FCFC4C9A8D4540F6714DC36A8DFBC1C09437B627C1DEC1A0480D71AC18564068698292348951409F3FD220E560F0C1E896651FAE82DDC1BCCE7541B2134D4048D07B006A305140B44DD09D29C8F5C10B63B8480A7702C2B0CA1165964352C0203240654CDE5040C8BCD355DAC3EF41BAE157B34308F4416A216691C48C64C0C07B9E9A76B7FFBFE073023807F0FDC1D0449C8D5CCDCCC101030000C0010000000800000098F6D7380F713740FCD615F56A8039C038EEB4F9E029EA4100D8796E8324EF4170C62FDE73975A40BB716B55B65C55C0CC6565633F0EF541902C999264BAD84114D0573B57955D40174F12E2A8DD51C037EA9D3C5225F7C19A41D7F9478DF541504C7EC5F0A5554090B214E62F7D12C0F0420AB4DF93C941FA86F6CCC95BF5C1AC547A0217F9534020631E1DD2581340601FD015BC62B3C1C02B4E6C30B7AF417039265E43E5274068E129D2CC852340503F48D1A526EA418CDC427AE86CE9C1A62F6162062765C08E23185754C95040864751BE63B1F5412ED80B52A3B7E2C198F6D7380F713740FCD615F56A8039C038EEB4F9E029EA4100D8796E8324EF4101030000C0010000000C0000000845A4489DFC5CC0F0EC4E329E3E4CC03917B838D5E7F7C15C379223BDF600C2827B7CB273DB4BC052A1DDB60AB347C0C0BB50AD029DEEC142846EEC84C4F54174F3439751FA4C4058A1ED03165853C0AC507522EA87FF41105AE61E4185EC41126BA8CF909850402ED024A8A6AD53C09845BD859A30D1C178C8C5261CD8F84100892BDD704E5640F7950E72DFBC50C038BAD42700DFE141E051DDA8B5D4D3C18C8A9867A48C6540A8593DCCCC5C45C04037BF3AC152C6C158A4E9AC2CCDF441683BA8E76F8B43405620FE8C46F63EC018A0A6A57426E841627140223BF6014234FC38288D375240786D3B3D11F836C0302499D93B70C041A097E0CE47CAC5410C0630C16D725F4000B47FD640A6F4BF785DE27CF2680242BCE0EB252178FE41683E4E8B4FA540C0883D29CA40C12440101C89C29EEAE841AE7A7441791CEFC1162E3C2F0FBC63C0B4A51DBE39684E40E0A172E541B9C1C12F3A11F1B787F5C10845A4489DFC5CC0F0EC4E329E3E4CC03917B838D5E7F7C15C379223BDF600C201030000C0010000000500000030550A82668938407CCA7653E7BD3DC088709BB5FAB0F8413075602601D5ED411E70B01CCDA7514018F24088EDE440C0001CE81FE4B496C1382F390A43E9EB41CCB2C60757B55B4000AAFB5DE489E4BFF0CF5C41B105E6413AD469B1B495E8C1CC8BC1CEE48F64408AD08475F9A84040000ECA71204CE241229EAAFEBB05EDC130550A82668938407CCA7653E7BD3DC088709BB5FAB0F8413075602601D5ED4101030000C0010000000B000000D06832A9068232C0E37A0BCB2A0B51C0228CEF5F4D43FBC13395E800870DF1C1006AEC9FAC56DCBF700F5D466ACE15C09CDA549A1610EA41D419599AA2E5DFC1A06F336CE86324401679D0002F9D4EC02AEAF6955984FA41287F15A938750242305D2E0666DD43404CFB16D6422B3A401EDAF93C9E2AECC16C00420BDB70D5C1EA926432807066405A2540D041E150404A728251A41802424E6F893CD2530142307D0DCFC8445940E68DD09863D25140422D36BBEE58F1C1E010D39E16C8D1C1EA533E3FD4464CC01488D8AA09EC5340A81C2A7FBFC2EA41E8F0603CE5A4FC41B646BE8B89D953C0308EC9E52D924940E4B200398AF4EA41D0C6A211F07BD1C12D6DF13AAB5D5CC0D081CD1356E7484026126F20C5C900C2565B08F00CAFF24128AAF7B51BF84DC07085ABEE7EEE424058F9306CD6E3DA41847E8BC153930042D06832A9068232C0E37A0BCB2A0B51C0228CEF5F4D43FBC13395E800870DF1C101010000E0E610000078503929520234C019A33A5DC1CE51C0825C4EB686C2FEC1A0F1B8E0219DC14101040000E0E61000000900000001010000C06C8EEFA06B5366C0B42660FECB6F4BC0E859AA5C0B3FFD4100F4B7966870DBC101010000C0E03806B596185140148C8FE6D5E24440C89F87C925EFF4C1A92716834D23F5C101010000C0DC6C8C5AF0CD4940B0AAD08F02A52140006DFFE6B0B7FB419FAD92E71B02F4C101010000C0D0267F4888C32240D420F47417994FC02E1AB81FC31EF2412405E4CAB52CF44101010000C0AACD596FF7C254C08EE75B98BA4754C0A07871492B90D841AC8969FC9C23E64101010000C0D4483D7FC1C555401A7292D34E823DC03A28CEEE2536F941DCE5679A2128FB4101010000C0A0D0F01274306140D0D80B9EE78C4640382AE6066866FAC1423B9C2680D9FFC101010000C0C6619E51A1E36140CCCBC7B1FDAA494080BC1C70D5D9D0C1E0DB456D51C3B44101010000C028B70CB97B8D54406C82EC789C474B40E89BE255256DED41AA07ECCB4DB9F74101010000E0E6100000C2AFC7C6AB794EC0D0E22CA328222A40106BCA0E2D5BE941D4DF56D9582901C201020000E0E61000000300000078C05BAAC8E95EC0D02260EC746D45C01C88B117FF650142BCB08BE27F1CE3C1D649BFBBABE06340739686252B2D45C0629CAE35BD29F841CCAA4B93DF17EE415A7F1360786E52C0007B4E04CD2414C060E1254F2CB4C9C1C6CD3A5C65F5F44101030000E0E6100000010000000900000030781B82714F60C041906C273EEE54C00AEEA0AF0CA3F6419D1BFB52EA6F02C22D15688488A656C01A08C080B6BD55C03E68F4F75A0FEBC1607D201083BAF1411434838A740A6440670603AC1DE34EC0C87850207334FFC1A05BDFB8D6A0EE4198C4D489783C60402763221954B140C04C61BC7FE971F8C170CC3F4B4730D3C160267B247B2E2740A66DAA8DE03A55406C941B5FA917E2C140030743AC57CCC1D06D5C986E2543C0E09D92AFD83706C0E536CAB7939CF0C1F0BF0AA0B865DBC10755A253B90E57C06446637A95433340FD7D47B3B76CF1C1064EC046949BE7C12B5C565762B863C0D4BEBF0C8B5F4040408D8283D76BB3C108477B328F5AFF4130781B82714F60C041906C273EEE54C00AEEA0AF0CA3F6419D1BFB52EA6F02C2':::GEOGRAPHY, (-764953914336802097):::INT8) 
    rsg_test.go:125: Last executed (27): SELECT st_asgeojson(e'\but':::STRING) 
    rsg_test.go:125: Last executed (28): SELECT st_asgeojson('01010000E0E610000084C0930D90735A4049FC4F00366F43C0CA1D295BEDA3F141F0A2E34921C1C241':::GEOGRAPHY, 5154678297101344162:::INT8, 5677653487630790023:::INT8) 
    rsg_test.go:125: Last executed (29): SELECT st_asgeojson(e'S\x1f':::STRING, 6711111277046309878:::INT8) 
    rsg_test.go:125: Last executed (30): SELECT st_ashexewkb('01060000A0E6100000040000000103000080010000000A0000001AF2316BFA364EC060B85F3F6DC31D40727DDACD8570FFC17064312B972F2BC0C732B869398451C05A7120BB839BF3C1F899583F163A32401A62E49A1A704DC040CCA6CA9A0EF041C0C4FBE062BB4740F9E67E7EED4E42C0D062AF46655ED0C1A07E7AC7540717403866E61092F04340E004BF9D8D8EF6416055DE09323B3AC024EF901654894F40600B5E76796EF6411E520E591C765FC0A094730114824A400EE192783592F8415872108E69F364C044344F988DED4B40785990F01215FD419F31214F8DA560C0689FECBB685C31406084FCEBFB76CAC11AF2316BFA364EC060B85F3F6DC31D40727DDACD8570FFC10103000080010000000400000038AE9D6F52AF46C01672CD803F8840400077D130C7E2944138E7806BC74C4B40986D4A446D7A2F40004C087DC254FD412ED22B81D7365240608F8619A3B75540EC8083A10733EF4138AE9D6F52AF46C01672CD803F8840400077D130C7E294410103000080010000000B000000881692FF2E1763C0C5A3E61C74E94AC01E135CEBC060F2C1BDE7423857C152C0D851443A2CD733C078CC9A9B72DAF8C16C9FE50CAED951C0362821E8161F45C0DC0B76DFA727F4C17014A9EF8B17424085D1C61AF9694AC0802164807854E9411C5BCAF17A386540C8F06CD6BB2E24C0787C287AE0E1DEC1863E59A8343D614076E1AB1DF7155540C0C42FABE898E041E00308E7B9951740F0810DBA11DF3F406A5CFEC5E83BF5414066D1DA982040C0E0DD2386D36655408A4901CF5846F2C140295E5AB5EF4CC0CCDA107089B141404A76095FD4C6F4412C10FAED23BC52C00CD3FC256895424027D160D11348F9C1881692FF2E1763C0C5A3E61C74E94AC01E135CEBC060F2C10103000080010000000B00000069CA45E39EAE5CC014E4B26143E332C07C34CDB4AF7AFF41B4A77CCE124642C03DA6557828EE41C02A5D901A64C801C208689377928843C0A1A7B705DFEA55C08CA54D83D713E441C0DF161799A61C4073BC38D5CD8348C0C03DBCC78E49BDC198221A5974BB3D4034D4295C3A4A36C05ADBC69C5DC9F3C15435A54A1428594060F89FAB6C9A3E40C0463D8795D8FC41D8ED4D15079361402C0875BCAAA04740D099A8459288DCC19C5B0131968D4340BCD8ACDA99A843407483F0AF2A2BF641552B2D0876E060C0E0E0586849DA554006DC087939CC00C272FCB8DD2A215AC0F085212A02B52A40880BBAA293B8E04169CA45E39EAE5CC014E4B26143E332C07C34CDB4AF7AFF41':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (31): SELECT st_asgeojson(e'h%9\x1aC6u\x05y':::STRING, 127:::INT8, 5945733380189237955:::INT8) 
    rsg_test.go:125: Last executed (32): SELECT st_ashexewkb('01070000C00100000001030000C00100000007000000E7E5D55D924AFAC100CF42FEC3B9A0410A2CBB00B54AECC1F09B2FB97396E341F6616872FA4601C2F8988E6C4143E1C15A9AA1183E6BF041E00944D7A007D84180D648601BB2F5C15085E3A95CBEF4C1E2F8D11BBC000142FB6B61FBCF62F0C114EAD1426721EEC1A02FDFF5AF01C1C18ADD294D3914E5C1A4821B923F17FB41BCE539F03C9EF241C0ACB7C3A66EA3C1D6532186D80BF6417E824C14CF86F041D57765A4B14D02C2F639CFFD3C1E0242DC5C8072EB2F00C2E6AE0DCD611E0042E7E5D55D924AFAC100CF42FEC3B9A0410A2CBB00B54AECC1F09B2FB97396E341':::GEOMETRY) 
    rsg_test.go:125: Last executed (33): SELECT st_ashexewkb(NULL::geometry, 'e':::STRING) 
    rsg_test.go:125: Last executed (34): SELECT st_ashexewkb('01070000E0E61000000400000001030000E0E6100000010000000B000000C385226E44B562C0F0749C7E8EAF19C000BCA8079B5BD0416C8D4A2E749DEEC1E092451F6A241FC0780CB3E40B8636C0980BAB1E9065EA418A8F41BF5028F94180378AF1BBE6FBBF1CB264621DEE39C0DC17245852A1E341844EB62828E7FE4180EA5AAAEAE82740E88413A908E544C050E78391BF02DF4180BAF5939381A641B0F0BADCC849594072E556E1773A51C0ACE534E1567FFC41C80A7D182A02FD41E4729D45D25A5540740383A2DBF533C0928130D8D593EFC1F9CC730FC368F6C15EF6B43932B16340C42BCB2CA4844F40BC7A96AD4AF20142889CE640A203FD41D8EA5ABF9052604070199E4C7BA75340E04D1DC301D8E4C125281FFCE59D02C27407BB1F8AAC4E40F42F65FEE50B4D40A0DC4C66FF48C4C1188682D32A8EFCC180DAEDEA55D32440BC2FD6D59B823040F819FB40991DD341C444F12266A400C2C385226E44B562C0F0749C7E8EAF19C000BCA8079B5BD0416C8D4A2E749DEEC101020000E0E610000007000000006A0B0C1DF065C00003348BFD982BC0E46FB58BA60B0242F03FCC8AA68B01422C8DCE3A1E5C41409C9C1A3B197236C088CA0B685A02E141BE48A1C2F892F7C128313C0FB6DA3840505BB245B5BD21C08D2386F671A0F9C1600D26B77344CF41B84AD13F77696140289112CE8A1F20C006574386932202C21823AF41448AF941FCFDD9FD235E4240A02EF8F1DAEC2F40E0DB54B012E5C4C16856E6CB14C3F6415414E6D2351A4F407016926D059B3C40204BCE3C245ACD41EAD9CCCA61A50142A44C0EE9D20041C02C025FA341024E408A7B49F2386EF7C1F4A184307A33F44101030000E0E6100000010000000600000046CCD91D764E6440EC6674E5D0DC3FC06933898CA7DDF7C180EA3AAE74DFF7C19426AF7D57F36440C0B59747890B02C0E8620B499E7F004291208F442B4EFBC1FC0D501406005940186D987A8B2C4640C898F46E65DDF341F4A0D7A99D1DE041402A6B8DD3D80DC0A03E4726C7F81740D03A9A87EC42CB41781792C13468D94148B534E37BAF63C060A181C8D2961B40407CAFF9774AC941BA5015CA4EDCF7C146CCD91D764E6440EC6674E5D0DC3FC06933898CA7DDF7C180EA3AAE74DFF7C101040000E0E61000000300000001010000C0945BCCA16B3546402050FB8E78A84BC0BEC9BDB93611F6413E302486F310F74101010000C0F444C4D965CE64408F1E03819FF94EC002DCB7B1E89AE2C114F925CAF102F14101010000C0E8C1BC6989023240A8BFFD55B4AB2A4016A97944D385E6C1B4D13AF7C996F841':::GEOGRAPHY, NULL) 
    rsg_test.go:125: Last executed (35): SELECT st_ashexwkb('010500000002000000010200000003000000000000000000244000000000000024400000000000003440000000000000344000000000000024400000000000004440010200000004000000000000000000444000000000000044400000000000003E400000000000003E40000000000000444000000000000034400000000000003E400000000000002440':::GEOMETRY) 
    rsg_test.go:125: Last executed (36): SELECT st_ashexwkb('0107000020E61000000300000001010000000000000000004440000000000000244001020000000300000000000000000024400000000000002440000000000000344000000000000034400000000000002440000000000000444001030000000100000004000000000000000000444000000000000044400000000000003440000000000080464000000000008046400000000000003E4000000000000044400000000000004440':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (37): SELECT st_askml('01050000C00900000001020000C0040000007465013DFA1DFFC14A0807D83F65F9C19F3B2F47985EF4C1E1AADBFC42D7FFC1F12C3F19ACE9F6C15E3166B0ADDDE0C196BAD776FAC9F8C124B6D6E88633FF419D96A9D55B13F2C1521DD9E11684E6C1CCF1BB28C59BDDC110BEB934883ED6C17E6EB2195B5F02C25E0B51C8B93F0242E33715F8D6F6F7C1DED9EE01D460024201020000C002000000D0450FE34D3F004214CC064C9733DCC100FD5C4FD1DB9FC1F97DDB1AF3E401C21C38215FA40B004238D27525E7DFD1C100923C595F6AF2C1D889A3413B3AE34101020000C002000000A8ECEECCE23EEDC140D8F6DBF6F6C9C1805B25E02E96C2C1DC24EE9330D0FEC178A3EE73C8E9D7C19CE8FCACDE4BEEC1B4F8C6D41B10F541A4172AA8212AEE4101020000C004000000209BE77D10D5B04148A8404319BCF7C140B60BC36E1FEA4124416CBAAF7EFB4100F5BC6160FBDB4120B8C5B4BD4AD2C1B0A740BE9753E14112A12C38A7050242F23107A45E61004270CDCA4EC2C0E341A4305688A3D5EE41F64B7D61B2B70042239108F01CBDF8C1F0FE2D51D107F741983302FA4CDFEF4124540998E304EAC101020000C004000000B016BB8AF80EFC416AF54B375C98F1C100D38B7C4CA88D414486A65E07ECF8C154C3F8FE170EE0417089A77F100AC4415E3BD11ABE3802C2C07AC34DF02ECFC15A02963DEA48F941FAAF329733CE0142D0460E27E72DCF41C44451081FFCF2C1AA7C8F2A2F5CE9C138D97DA844C80042B21F667C3FEDE0C1F429B55A5C8EF0C101020000C0020000007CA21156E806D5C1A8FAE0F27DACF8C19873A1CFA1C9E8418F6293A54F57FDC1603966FDC99DDC414ED3D562963DE0C178A040E7F57BF7C1E0A0FF482C8BB9C101020000C00200000024EDE3A833FBF4416020A82E8578FCC1AC7DF3366C20F3C1D83E31161D48F6C12AA0BAF8A999FFC14C110FF8D460FD4100EC60D2747CCBC146C22B678518004201020000C002000000001ECAF9F20FFA41C067191DEA1ECB41DDA3E1617B77F2C1C22A07268370E3C160AD9FB53A15F841747BB13FDAF6F041C4F5323E9EF3F7418075948840A2A7C101020000C0030000002A0B5A4EC581E3C14C4B59164440EBC13424CC4B91B2D9C1129CE1856D1002426CC6079FEE2FDFC11BD761B33CE3FAC149924B413B83FEC1C49592B14664FF4112BD9894885DF941302370FF18A8E941BA04273A9DF2F2C118956A2423C9D6C1':::GEOMETRY) 
    rsg_test.go:125: Last executed (38): SELECT st_askml('0107000060E6100000090000000101000060E61000005D6ADD5107E965C04C3FA4C7E8CE4340000D57D9435DDAC10105000060E610000006000000010200004003000000B93701B3E7D063C068226C1CF68A2F40E648A1DEA1A9FBC15CD70855A5D742C0C0E4EDCD492B14C032D1B21031CDF6416CB32483A8B64340369ADE900C7E4AC0B0EB548497D2D441010200004002000000E41284879CBA6340101E3BB72DE929406C9A29D3E4FFF1C1489435CAEB8B31C09AA8FDC4BBC654403072522E908CE441010200004006000000309580B5CBF95140947CFE2CF69A54C0D0463454F410DAC1C0078A4839A1584080DCFDF136A20AC0E8FCAB9EA25DF1419086C129EE6B5D40B08E786E0BD1384038FD81D05814EC4134879C6205C05940847C7FA5E8C85240BC12AAE257DBE0C110DD594566B935C0D426FA7CD2B63140403E3E6290D1E441D46BE66AC95C43C0380B23CE28B92D4028CA97D017F201C2010200004003000000FA1263D5637C4CC0ADD722B67DA646C08CDB6E06842500C291C7F13DA2FD5CC0442C4C66D64950C0D02173F57E5BD6C10434C41436CF46C039F663798B574CC0B0D0B9899AA0C0C1010200004004000000D43580CC679163C04883B8AF47C34CC022BE4FBBB544FEC1188BE065D14436C0BC4E5FFE6AAC55C022089C75246C01C280881617D46716406C610EBA569C54C0A314A08EBCEDF2C188DDF993D7C740402C6AD9D6793E52C0A973DDF4E19900C2010200004003000000B0AAC62DB7D63E40BC2178F7022551C0A068ED04269CE041C05A909987035140CA00E4B87CD551C07C390D0CC4F5ED41701ABEE48B7855409ADB66545DE23DC0FAE37D3FD77EF3410102000060E610000005000000A4FBFEA010125C4014E49A1CF99033C040F38C654521DDC1B44E8BADF2095940B09E222B382A41402B9EA3E65972FFC11C05BD263B8C6440EA0E56C403105540D4EA0BE82C9B0042D4F10D2EAD095C407C882B7B529A5240A1D98F8AB19001C245775FF5CE3066C016C2D0FFC0065040E6A898BACD4900420105000060E6100000040000000102000040040000005C04270195264C409CFD49D1F84139C06E5C73974D0EEDC1F43E30FF5356414080CB76EB6053FA3F42B8642BE874E5C140892CA2441663400CC4C078B34933407C25BA96D914FB4170CA2FA6D07454C00A11519D32B9544054262F1F694F0042010200004002000000B11BAD75F0DD61C01697DA8B80F53BC0BAAC188CAACDEDC112640A339657664037B9F0BC5EC848C02C4C242EDB2DFF410102000040020000005E71AEFE1C7960403C07C528C1C2454023D542C06D9EF2C17E40862454706540308C132998084F4040320F993CF3AB410102000040060000003C6018B3195C50408CB6EA10D28A54C061B29A73C068F1C13C9676219BE0594078A39BDC3CDD2340188F84FEA998D2C18EFB4986491D52403C2CC11471824C40E85AC21D908FF741A699AF4EAB6C62C05CDBFB6AD36950400AD98BC5216BF741C93938118E2C5AC0783DECA0CB3D2A40606E2E810F0BD84160B8CFCB22E164C020824E7817B401402493693421ACF5410101000060E6100000D6D26E60034861C0985D6E2D16404C4078B1EC581428D7410103000060E6100000010000000A000000E4018C173AAA65C0808C4480D1BB53C0D71E222AA51BF9C1F0305E723A5242C09BD5FBE6820344C04F6053F9DAD7F9C1500D979BFA632F4026D380904C1D56C0214E2DC19A03F5C170668853F9D92140D238C73B718641C0DA8223D819DBF041D8DA5E0452FD3A4000067ED8C53B0CC0941BFB73E0FCFF4104C66D0828C64240909A607D0DEC18409A92329D33FEF6414079B8C4FFD026C08EC73193580E53408873D35EE76EF9C1F6A95C52B15D5FC050A4B85E30005540DA0D7C55CEFCECC16DF805554B6F56C0E01C170A2AFF0440284A4570A7E6F741E4018C173AAA65C0808C4480D1BB53C0D71E222AA51BF9C10101000060E6100000AA133FA264C258C03CD4FD0FA65F554090C3F4284CC2D2410105000060E610000007000000010200004002000000F89AC7C8E90C4F406E3BB9930DCD514000A583AA808CD841A57FAB50186065C0C8EF5C90C1513540D4EFCEE5E066F94101020000400500000040F3BA2CEA2C3D40A0A7C6AE6D7F00C020F5E9F486EBB8C16486519B68C05840F4A19CD8095431C0386F72C89AEDDCC1600B560548145E40F8B695D0454F32C05C56FB3E6874FFC1CCA6344C3C4F4740C240ECF3071F4140A8D8EC5C002C02425446328850364A400CE68016A97E554025B8585AE9ABFAC1010200004004000000FE63FA5B053553C06C244F784BD0514046F4E745781AF54100540BBEF9AD32C0506E626408A03B405C0BBFF5E4F5F2413268989613B650C0ECBCE866C7AD4640EABE95F942B4FAC1FA869DAA29595EC020E5DBDEA28D4C4060472823DB68D3C101020000400300000032484EDF483351409ED61EBD04174DC0FECD06EF317CF3410889193F5438544078B54C4A9FC637C00AB21B6C0E84F041F66EBED0D8966440F0EC9BA179233C40A899CCA35A85EDC10102000040060000006A82CE7374775140809025870F1606C0141E6A6C21EEFF41409DA352C7F73FC0B08AD1542BC25540A8E1A9F69355D4C1C8D2BA3842E447C05C75BA6439454D40F81C66312AA801C297B7381AD0455FC09024380E279F53405E5CA74E98A2F84137635F1E010C5EC0E8900551721B4A4068B28AAE162DFA41BF91BEB8492765C0BC9AE97094384940E041F736B8E4B7C1010200004002000000A04A840DEDCF3E40500074873FD612C0041B0D905B12F541442A639F63B15A40800FAEFE2F26E0BF50BA96050135E2410102000040020000000883D34D51594E407493A75E920D51403CA4B34F596AFC41F4AA77478FA34BC0CCF86C144E014740946E9FDDEE9D01420103000060E6100000010000000B0000006E6C8A5B6E9359C0A0B848A179F61F40E80A2603CE4DF7C173EBDD657B895CC00077E01BD2EBEDBFEA4A507EF9C4E7C1C03BA8E5518920C0188CE515422025C0CAFA0467F320EAC11CA859F8EA5A66406D081DEAAC7D51C0D81CBDB81810F6410011F86F629064400C5D8BDA500538400CE52EF52E6BD7C11E76328A58F86440F8120FBA62035340D06DC9D447F7F941F0E12B76A77E4240E84CB623EC465340169935964F840042C01C1D9CE0512DC0EA33E910BC0750403E78E7EF9DBD01C26414AE742EBA5BC07CB68AC65C663D404D434749D8AAFAC1802C51D17BF91DC0F435140A9F3D3740745032E6A0CBDBC16E6C8A5B6E9359C0A0B848A179F61F40E80A2603CE4DF7C1':::GEOGRAPHY) 
    rsg_test.go:125: Last executed (39): SELECT st_asmvtgeom('01060000000800000001030000000100000009000000DE80212BCF6DE1C1103174632047F2C1F6CFCB13CEF7F8410686381A30AAF6C1E02222EEB0FEF841C2122FCF79FBEBC150C128D3F05F024278C8F236E058E8C1FC4382B1B5B6FE4180AF4E5648999C4138EDB416D900D14140364C500D11F74110256B1F690CD6C1BCA4736B59E8F641A1BA73E66DEEF4C17CD902E4313DFF41DE80212BCF6DE1C1103174632047F2C10103000000010000000600000037716FF8A31FF4C100D4779D112CC9C100BA7D8677A28AC1833EAA19615AF5C1E88A10BB9B07014212DC3A5ED754F141882AEE9446F4E5417E4158B7950E024266B506EA3318EBC1749FEBF5F289FC4137716FF8A31FF4C100D4779D112CC9C101030000000100000008000000FC53EF0BADE0FBC198C8937813FEFFC1C4C1886D8AB9DCC1381AF238156BF9C1907343E3ECB9C2C118F6E72EECE6FCC1980DE97ABF9BF84170E6035FC9F1F0417895DFA25C3CFE4154E63F4AAF8D004258FDC2755A37ED41249D4165426DFB419069DC254C9CCD41603C148100F7ED41FC53EF0BADE0FBC198C8937813FEFFC101030000000100000005000000C8D8507CFCC4F6C11C97E60650CFD5C110D8298845D7F2C12CA8816D2CB1DEC1D220DC1AAA3BFDC1F0BD166E49C5014267B8F9159FAC01C2788741012C84DE41C8D8507CFCC4F6C11C97E60650CFD5C10103000000010000000D0000004E002ECFF255F9C1E01DDC135969BC411EA06BFBF072FEC14C47E4107AF1E6C198B46D816CD3DF41C4A006E5AA0FE0C1FAED95BCBCE3F541CF143395FAB8FAC1BA8F721B3EA8F041DC7CC0144B27DEC1E8F200A5734A00425DF92D835116F7C1C24E1F37A7040142483B3B8C174CE24110D30018CC09FE41787B9F26B3B3EE41D0961AB5C79BF84184D1B791405AFE41FC36E123481EF24180E4F306F286024257FC47B39B23F2C1502F2345BCF0FD4198ED3A8DC2FEF9C12C22EAECE07200424E002ECFF255F9C1E01DDC135969BC41010300000001000000080000004A942F90AB7BE1C1F48DA68E9EB4E1C10075DD993339D4410993A84AACB4F3C188E3556D1577D041897230AA95A4F9C1B07AAB7C1B6EEA411DC010D2481B01C286ABEE6DFDF2F9412029C6852D33CE41C297F241FE41F141A0D81EB176F7FD4100BA8CE213A6874190F930A0506FD4C14A942F90AB7BE1C1F48DA68E9EB4E1C1010300000001000000060000004494EF079908E8C1C8A10CC85C87E7415C2FC6646E07E8410BC3AEEF64DBF7C14A7DBBD7ED7602429AE0DBE7B9ECF241EA5343AA86E60042086E535B8DB5FC418C68AA4578CFE141D84ACCDEE6F9F7414494EF079908E8C1C8A10CC85C87E74101030000000100000005000000D839F23D3DD8D141B4F097DBD704E14140457775EE35FB41B6AC13B951C200420E9289EEF9F2FBC154A62FA7FB77FE4108801D6886EDEFC1E0FEF85C7A41FB41D839F23D3DD8D141B4F097DBD704E141':::GEOMETRY, 'BOX(-10 -10,10 10)':::BOX2D, 7230689981943994004:::INT8) 
    rsg_test.go:125: Last executed (40): SELECT st_askml(e'O:\x11Jy\x0e\f\tI':::STRING) 
    rsg_test.go:125: Last executed (41): SELECT st_asmvtgeom('01040000400100000001010000404AFA61A71B12EFC1544292AEA303E04180378F0E0E5AAF41':::GEOMETRY, 'BOX(-0.46088264583143007 -1.062670098531982,0.03644732137953666 -0.39548338617298195)':::BOX2D) 
    rsg_test.go:125: Last executed (42): SELECT st_asmvtgeom('01060000C00400000001030000C0010000000700000077EAFB0F3D3A02C2B9D6D00392BFF1C1283704250A03DC41626B4C2FB136F9C134EEC538B6C7F7C11E3F38653AE2F9C190AAB6B36670D7C1DD33851B6C2AF4C13484C9498330DFC1447449A7F0CEF2C1F08D0E08AA8AF1C108CB3B679B9FD2410054827760F562C160DA1D037066E3C1C02FD0746CDFB241C7A56A2E4C2B01C2ECA60ED90B86FF4198127D3E3CD2D841646FB5700311E4C12E2FBCAC30D6FDC1C069C00447A8C9419888DD30DA8AF9417036AC9BA182C4C1B412C557ADEEF7C177EAFB0F3D3A02C2B9D6D00392BFF1C1283704250A03DC41626B4C2FB136F9C101030000C00100000005000000F07848146A9FC041E2C1410A09A5F14102EEEBC1F0ABE8C164DF2BD862B5F2416AF7B83544B60042D421702FF4D1E141C033F4A04933C141309681F8BDA7F2C1DCBB91001005FE418C261BD78BDFED41ACEE3AB5252AFF4177C7ACF6700EFDC18E4FF6B0345D02C248797F977970FE41502223BE654DD2C1BED79DFF038DFA41F07848146A9FC041E2C1410A09A5F14102EEEBC1F0ABE8C164DF2BD862B5F24101030000C00100000009000000DB3D73E03CE200C2BC68141DD04BDBC168A320DAD66402C26E23D0EF90CDE7C18C2950625910F8C124BD35A998F5D9C1B85AD04B8A74ECC1842D7BCBB42EF941004C91717B70914120B1AB85EE3FFFC1B6C3E4DBC563F241FE1AAC6EE55BFAC1A81ECABDD9A8E341201A3FD3D817F04136DA61219178E7C1C12811BFAA7500C2148E0E4D6E56F54156BBA82C7A7C01428808CC44D11CDD4190757F6C34DDE6C19BB8804CC802FEC1E8838CDCA32AFB4130FEDD364533E041A78B12E64BA4F4C19B207C40B844FFC142869E684232F841E07FF594D779E641F0EC00CB72E8E5C1997B97AF7D35F6C10C9166778BB9E54148F35F0FF2E4D3C1A0BEFD31BB3ABA41DB3D73E03CE200C2BC68141DD04BDBC168A320DAD66402C26E23D0EF90CDE7C101030000C00100000009000000E83D2A7084ADF9C138283FEC040EF3C15D0F3891474E01C23A607FAB1B8AF4C160D183E04447EF4140773F146B69E1C1C0CB56C0C940CFC168427149DED9D1C134F29836DB5CFF4152AE76501F63FEC102B55095F00DFDC124012926202BE8413630D491C292014298F807C4C31DE741BC794F81A2A8F2C1305D13102E7EE44158A40C92A9A4FF41B8A63E013C3FF24102CD7505F307F8419434459847BDE5415E79F42C3E50FA41EA1EA6DE301F02421C6D5C4199B4F9C19B66957DE08401C22ABAE2B367EAE2C1B41D7441D793FF4100C2F37E5156F3C1FF0020D2E613F0C1AE66BBE76DB8FCC18446022AABC2E241F401318929C7FA413C072839419DFDC1E83D2A7084ADF9C138283FEC040EF3C15D0F3891474E01C23A607FAB1B8AF4C1':::GEOMETRY, 'BOX(-0.8590355028813085 -1.5503250398447748,0.6136755188496659 0.21860247103960534)':::BOX2D, 4278319356988574388:::INT8, 8746015892663629040:::INT8) 
    rsg_test.go:125: Last executed (43): SELECT st_asmvtgeom('010200004004000000C9C3AA177DADFBC1EC131AD17440F9C14E538813834EF141DC2FC09B2F98E5C1DE5557F9826801C2E4F9EBB3DC27D7C1A49BDEAEFDFAE3C1A3E9D8E673E4FAC1B88C1D8D2C5400422AAE808F876502428D9CBAC5EAFEF1C1DC63725E4B60F141':::GEOMETRY, 'BOX(-0.6858710642288747 -1.4907075938488434,-0.2889075394594619 0.9952441014102877)':::BOX2D, (-1):::INT8, (-1447381679079061241):::INT8, true) 
    rsg_test.go:125: Last executed (44): SELECT st_astext('01010000C0F2E8729BEB0DF24138D11FDAF482E8410003509225B7C7C1D2C0E0131FFD0142':::GEOMETRY) 
    rsg_test.go:125: Last executed (45): SELECT st_astext('0101000000C4FED0AFF3D0EA4110697D987727EE41':::GEOMETRY, (-3363413745854648633):::INT8) 
    rsg_test.go:125: Last executed (46): SELECT st_astext(NULL::geography) 
    rsg_test.go:125: Last executed (47): SELECT st_astext('0107000020E6100000090000000105000020E6100000080000000102000000020000007A728934D6EB4DC02056778AC42724C07E7A94ECE9A963403DC0C793F4F853C00102000000030000004AF1CEED476C5140B8212806051E4340B825759506C165408C2751579F123040843B228212884540960FF794AD8550400102000000020000006AAA9B18DD5E66C09F54825ACB1E55C08032C1F0D3045340ABD59666994F53C0010200000008000000A474E9BA54C153409DA19C99900C55C040FA960E532D6140FFA5B385269655C0681F42C5FFC8564002CD9B23BF4046C07C1558AE93C25B40DE27DD5BFE943CC00033AEFA228E47404C608A0C4616394098D1A453E84A3D40F832B45AC0783640B0921FB11AA24340E24C82B0103554409B3259B6F58463C0DCB3405CA3A65540010200000008000000A099C7CA5E7359C053E9CC291EED52C006D4954A907350C0DCFB4DB0E84B3CC01A7BA47595B94BC0CA4401B04DB242C08C2F47440F305D40EF0786F8702F47C040A7BD15D71732409CBDC38AB44C31C094BB0AF3D03C5DC06022ECA882774C400FFF2818D00762C0A4F1738902243B40D4AAABEB922861C000F84FD4D80ED53F0102000000020000001E485A2BDE804DC03464E5BBDC6A354082DDDD939D2C57C089B5121EBFAD51C0010200000004000000A003AFC41E1F124010204346309451C02069F80409F529C0C2124F1ED35E3CC0F89EAD9509C14040FC1AF2EC6F4231C060950C27CAED4F407E620EC11BC85240010200000005000000CCAD340DD75052407A33CF5C8ADF50C0EC6560D77FD85040A4BEDBF4AFC936C0E8CCFB67FBEF6440E6A0B3A0CCE250408C671D4218A862406E986BAB2EAA5040DCBF8E4C602F60C0E8CB3FE98FB143400102000020E610000003000000782231233D5166C024710CD872C33240B0F890C104F63840A3E018E4102946C0F628176C56B36240B69A94705D6051400104000020E61000000500000001010000006CE38C3F7E6C52400CE86A2ADA674740010100000098D896E42D6C4740B07A7B4D3E892DC00101000000A8679E33FEE23F4074822E7496A948400101000000D87E61E7CC7C61C0F00FA1B7FDDB54C0010100000054FDE6312A3764404C3C291AD1C33D400106000020E61000000400000001030000000100000008000000B82094AD392352C075A1F0FA67CB4FC040570C23679C32C00F506880BA0251C090D6172BE80E5D400DE457036EDE50C0B83475563FA8564042F78A7EE18448C0F04C20432C186440509FE8E52C5146409CD574E1785159409C133C1E44543D40C0D134C1E37D02C0C0A79EA4D874F9BFB82094AD392352C075A1F0FA67CB4FC001030000000100000007000000B02187BD7C2751C0202F5A82282039C0B08C73BC69EF2040C96566B3E48247C044074561D7815640006BBB81FC9226C02816428F30CF5B40BC89C3BAF62F3840740F6F94142359408CFF8A14F264384034C8A1CEB3DD5EC0DE184EE4054D4240B02187BD7C2751C0202F5A82282039C00103000000010000000600000013DF1B065EB65CC07F895ABB164343C0D05752525D6523C0C5029EAD169541C01063D9A842692DC0BC4E5E2D57DA32C0D44786CE05FA44405004703490A64A4018622F9A1D8E42C0E452EB024AF33A4013DF1B065EB65CC07F895ABB164343C001030000000100000008000000BE11B0C0400453C0A01854AA686646C0207F7C1346BC43C092E8B632E68C44C020F57DB42191474040E38BE0ABB032C048E6F5D220835D40F4FC43549A9431C02C823E31CD7865404CF3FD7F9BFD3340C0E777AF7C152540E60DB52874BF534000D7026C62831040E4DBA4D09F883740BE11B0C0400453C0A01854AA686646C00101000020E6100000F6FA1B5046F860406F0765EAE13148C00105000020E6100000010000000102000000030000002E38D144F89365401C80209770D25540DCD61EC6FFC5584038DF28934E074640C03984C230295940526BA2ECE30B55400104000020E6100000050000000101000000373D2E0B2F455AC010C3AE92C8D613C0010100000021FAB30EC30F61C070924AE2A91C394001010000005B520430E3B160C0024A0AECC4964AC00101000000E0B9BDC3195D4CC04CBF2495328949400101000000E4B90F937DA251409EF17663D2E441400103000020E6100000010000000600000028F0857042EB4F4000A615D0057049C0B2EC110786816440BD2D297E70134EC0909399196EBD4F400CA748DC20E44E40203EA585D31C3D40208B996A18EC5540004B22D17947F33F089531EBE1D14F4028F0857042EB4F4000A615D0057049C00104000020E6100000020000000101000000C00203C1455447C0E86544E263075340010100000000B8FD542A4C57C0A7F40BE178A240C0':::GEOGRAPHY, NULL::int) 
    rsg_test.go:125: Last executed (48): SELECT st_astext('^':::STRING) 
    rsg_test.go:125: Last executed (49): SELECT st_astext(e'd-\'V_\x0f':::STRING, 7308227490222744111:::INT8) 
    rsg_test.go:128: Currently active statements: map[]
    rsg_test.go:515: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions735983095
--- FAIL: TestRandomSyntaxFunctions (301.38s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/logictest/testdata/logic_test/json_builtins
+++ b/pkg/sql/logictest/testdata/logic_test/json_builtins
@@ -216,6 +216,17 @@ SELECT to_json((1, 2, 'hello', NULL, NULL))
 {"f1": 1, "f2": 2, "f3": "hello", "f4": null, "f5": null}
 
 query T
+SELECT to_json(('$'::JSONPATH, '$.foo'::JSONPATH))
+----
+{"f1": "$", "f2": "$.\"foo\""}
+
+skipif config local-mixed-25.2 local-mixed-25.3
+query T
+SELECT to_json(('foo'::LTREE, 'bar'::LTREE))
+----
+{"f1": "foo", "f2": "bar"}
+
+query T
 SELECT to_jsonb(123::INT)
 ----
 123
@@ -612,6 +623,17 @@ query T
 SELECT json_build_object('a'::tsvector, 1, 'b'::tsquery, 2)
 ----
 {"'a'": 1, "'b'": 2}
+
+query T
+SELECT json_build_object('$'::JSONPATH, 1)
+----
+{"$": 1}
+
+skipif config local-mixed-25.2 local-mixed-25.3
+query T
+SELECT json_build_object('foo'::LTREE, 2)
+----
+{"foo": 2}
 
 query T
 SELECT json_extract_path('{"a": 1}', 'a')

--- a/pkg/sql/sem/builtins/builtins.go
+++ b/pkg/sql/sem/builtins/builtins.go
@@ -12154,8 +12154,9 @@ func asJSONBuildObjectKey(
 		), nil
 	case *tree.DBitArray, *tree.DBool, *tree.DBox2D, *tree.DBytes, *tree.DDate,
 		*tree.DDecimal, *tree.DEnum, *tree.DFloat, *tree.DGeography,
-		*tree.DGeometry, *tree.DIPAddr, *tree.DInt, *tree.DInterval, *tree.DOid,
-		*tree.DOidWrapper, *tree.DPGLSN, *tree.DPGVector, *tree.DTime, *tree.DTimeTZ,
+		*tree.DGeometry, *tree.DIPAddr, *tree.DInt, *tree.DInterval,
+		*tree.DJsonpath, *tree.DLTree, *tree.DOid, *tree.DOidWrapper,
+		*tree.DPGLSN, *tree.DPGVector, *tree.DTime, *tree.DTimeTZ,
 		*tree.DTimestamp, *tree.DTSQuery, *tree.DTSVector, *tree.DUuid, *tree.DVoid:
 		return tree.AsStringWithFlags(d, tree.FmtBareStrings), nil
 	default:

--- a/pkg/sql/sem/tree/datum.go
+++ b/pkg/sql/sem/tree/datum.go
@@ -3877,7 +3877,7 @@ func AsJSON(
 		// This is RFC3339Nano, but without the TZ fields.
 		return json.FromString(formatTime(t.UTC(), "2006-01-02T15:04:05.999999999")), nil
 	case *DDate, *DUuid, *DOid, *DInterval, *DBytes, *DIPAddr, *DTime, *DTimeTZ, *DBitArray, *DBox2D,
-		*DTSVector, *DTSQuery, *DPGLSN, *DPGVector, *DLTree:
+		*DTSVector, *DTSQuery, *DPGLSN, *DPGVector, *DJsonpath, *DLTree:
 		return json.FromString(
 			AsStringWithFlags(t, FmtBareStrings, FmtDataConversionConfig(dcc), FmtLocation(loc)),
 		), nil

